<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>177 nips-2004-Supervised Graph Inference</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2004" href="../home/nips2004_home.html">nips2004</a> <a title="nips-2004-177" href="../nips2004/nips-2004-Supervised_Graph_Inference.html">nips2004-177</a> <a title="nips-2004-177-reference" href="#">nips2004-177-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>177 nips-2004-Supervised Graph Inference</h1>
<br/><p>Source: <a title="nips-2004-177-pdf" href="http://papers.nips.cc/paper/2738-supervised-graph-inference.pdf">pdf</a></p><p>Author: Jean-philippe Vert, Yoshihiro Yamanishi</p><p>Abstract: We formulate the problem of graph inference where part of the graph is known as a supervised learning problem, and propose an algorithm to solve it. The method involves the learning of a mapping of the vertices to a Euclidean space where the graph is easy to infer, and can be formulated as an optimization problem in a reproducing kernel Hilbert space. We report encouraging results on the problem of metabolic network reconstruction from genomic data. 1</p><br/>
<h2>reference text</h2><p>[1] R. Jansen, H. Yu, D. Greenbaum, Y. Kluger, N.J. Krogan, S. Chung, A. Emili, M. Snyder, J.F. Greenblatt, and M. Gerstein. A bayesian networks approach for predicting protein-protein  0.9 ROC index  1  0.9 ROC index  1  0.8 0.7 0.6  0.8 0.7 0.6  0.5  −4  −2 0 2 4 6 Regularization parameter (log2)  0.5  8  −4  (a) Expression kernel  8  (b) Localization kernel) 1  0.9  0.9 ROC index  1  ROC index  −2 0 2 4 6 Regularization parameter (log2)  0.8 0.7 0.6 0.5  0.8 0.7 0.6  −4  −2 0 2 4 6 Regularization parameter (log2)  0.5  8  −4  (c) Phylogenetic kernel  −2 0 2 4 6 Regularization parameter (log2)  8  (d) Integrated kernel  Figure 2: ROC scores for different regularization parameters when 20 features are selected. Different pictures represent different kernels. In each picture, the dashed blue line, dashdot red line and continuous black line correspond respectively to the ROC index on the training vs training set, the test vs (training + test) set, and the test vs test set.  100  80  80  True Positives (%)  True Positives (%)  100  60 40 20 0 0  20  40 60 False positives (%)  Kexp Kloc Kphy Kint Krand 80  (a) Test vs. (train+test)  100  60 40  Kexp Kloc Kphy Kint Krand  20 0 0  20  40 60 80 False positives (%)  (b) Test vs. test)  Figure 3: ROC with 20 features selected and λ = 2 for the various kernels.  100  interactions from genomic data. Science, 302(5644), 2003.</p>
<p>[2] N. Friedman, M. Linial, I. Nachman, and D. Pe’er. Using bayesian networks to analyze expression data. Journal of Computational Biology, 7:601–620, 2000.</p>
<p>[3] M. Kanehisa. Prediction of higher order functional networks from genomic data. Pharmacogenomics, 2(4):373–385, 2001.</p>
<p>[4] A. Doi, H. Matsuno, M. Nagasaki, and S. Miyano. Hybrid petri net representation of gene regulatory network. In Proceedings of PSB 5, pages 341–352, 2000.</p>
<p>[5] E.M. Marcotte, M. Pellegrini, H.-L. Ng, D.W. Rice, T.O. Yeates, and D. Eisenberg. Detecting protein function and protein-protein interactions from genome sequences. Science, 285(5428):751–753, 1999.</p>
<p>[6] F. Pazos and A. Valencia. Similarity of phylogenetic trees as indicator of protein?protein interaction. Protein Engineering, 9(14):609–614, 2001.</p>
<p>[7] J. R. Bock and D. A. Gough. Predicting protein-protein interactions from primary structure. Bioinformatics, 17:455–460, 2001.</p>
<p>[8] B. Schr¨ lkopf, K. Tsuda, and J.-P. Vert. Kernel methods in computational biology. MIT Press, o 2004.</p>
<p>[9] E.P. Xing, A.Y. Ng, M.I. Jordan, and S. Russell. Distance metric learning with application to clustering with side-information. In NIPS 15, pages 505–512. MIT Press, 2003.</p>
<p>[10] G. Wahba. Splines Models for Observational Data. Series in Applied Mathematics, Vol. 59, SIAM, Philadelphia, 1990.</p>
<p>[11] F. Girosi, M. Jones, and T. Poggio. Regularization theory and neural networks architectures. Neural Computation, 7(2):219–269, 1995.</p>
<p>[12] M. Belkin, P. Niyogi, and V. Sindhwani. Manifold regularization: A geometric framework for learning from examples. Technical Report TR-2004-06, University of Chicago, 2004.</p>
<p>[13] B. Sch¨ lkopf, A. J. Smola, and K.-R. M¨ ller. Kernel principal component analysis. In o u B. Sch¨ lkopf, C. Burges, and A. Smola, editors, Advances in Kernel Methods - Support Vector o Learning, pages 327–352. MIT Press, 1999.</p>
<p>[14] Y. Weiss. Segmentation using eigenvectors: a unifying view. In Proceedings of the IEEE International Conference on Computer Vision, pages 975–982. IEEE Computer Society, 1999.</p>
<p>[15] A. Y. Ng, M. I. Jordan, and Y. Weiss. On spectral clustering: Analysis and an algorithm. In NIPS 14, pages 849–856, MIT Press, 2002.</p>
<p>[16] F. R. Bach and M. I. Jordan. Kernel independent component analysis. Journal of Machine Learning Research, 3:1–48, 2002.</p>
<p>[17] J.-P. Vert and M. Kanehisa. Graph-driven features extraction from microarray data using diffusion kernels and kernel CCA. In NIPS 15. MIT Press, 2003.</p>
<p>[18] M. Kanehisa, S. Goto, S. Kawashima, and A. Nakaya. The KEGG databases at genomenet. Nucleic Acids Research, 30:42–46, 2002.</p>
<p>[19] P. T. Spellman, G. Sherlock, M. Q. Zhang, K. Anders, M. B. Eisen, P. O. Brown, D. Botstein, and B. Futcher. Comprehensive identiﬁcation of cell cycle-regulated genes of the yeast Saccharomyces cerevisiae by microarray hybridization. Mol. Biol. Cell, 9:3273–3297, 1998.</p>
<p>[20] M. Eisen, P. Spellman, P. O. Brown, and D. Botstein. Cluster analysis and display of genomewide expression patterns. PNAS, 95:14863–14868, 1998.</p>
<p>[21] M. Pellegrini, E. M. Marcotte, M. J. Thompson, D. Eisenberg, and T. O. Yeates. Assigning protein functions by comparative genome analysis: protein phylogenetic proﬁles. PNAS, 96(8):4285–4288, 1999.</p>
<p>[22] W.K. Huh, J.V. Falco, C. Gerke, A.S. Carroll, R.W. Howson, J.S. Weissman, and E.K. O’Shea. Global analysis of protein localization in budding yeast. Nature, 425:686–691, 2003.</p>
<p>[23] Y. Yamanishi, J.-P. Vert, A. Nakaya, and M. Kanehisa. Extraction of correlated gene clusters from multiple genomic data by generalized kernel canonical correlation analysis. Bioinformatics, 19:i323–i330, 2003.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
