<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>90 nips-2004-Joint Probabilistic Curve Clustering and Alignment</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2004" href="../home/nips2004_home.html">nips2004</a> <a title="nips-2004-90" href="../nips2004/nips-2004-Joint_Probabilistic_Curve_Clustering_and_Alignment.html">nips2004-90</a> <a title="nips-2004-90-reference" href="#">nips2004-90-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>90 nips-2004-Joint Probabilistic Curve Clustering and Alignment</h1>
<br/><p>Source: <a title="nips-2004-90-pdf" href="http://papers.nips.cc/paper/2627-joint-probabilistic-curve-clustering-and-alignment.pdf">pdf</a></p><p>Author: Scott J. Gaffney, Padhraic Smyth</p><p>Abstract: Clustering and prediction of sets of curves is an important problem in many areas of science and engineering. It is often the case that curves tend to be misaligned from each other in a continuous manner, either in space (across the measurements) or in time. We develop a probabilistic framework that allows for joint clustering and continuous alignment of sets of curves in curve space (as opposed to a ﬁxed-dimensional featurevector space). The proposed methodology integrates new probabilistic alignment models with model-based curve clustering algorithms. The probabilistic approach allows for the derivation of consistent EM learning algorithms for the joint clustering-alignment problem. Experimental results are shown for alignment of human growth data, and joint clustering and alignment of gene expression time-course data.</p><br/>
<h2>reference text</h2><p>[1] J.O. Ramsay and B. W. Silverman. Functional Data Analysis. Springer-Verlag, New York, NY, 1997.</p>
<p>[2] Scott J. Gaffney. Probabilistic Curve-Aligned Clustering and Prediction with Regression Mixture Models. Ph.D. Dissertation, University of California, Irvine, 2004.</p>
<p>[3] Z. Bar-Joseph et al. A new approach to analyzing gene expression time series data. Journal of Computational Biology, 10(3):341–356, 2003.</p>
<p>[4] B. J. Frey and N. Jojic. Transformation-invariant clustering using the EM algorithm. IEEE Trans. PAMI, 25(1):1–17, January 2003.</p>
<p>[5] H. Chui, J. Zhang, and A. Rangarajan. Unsupervised learning of an atlas from unlabeled pointsets. IEEE Trans. PAMI, 26(2):160–172, February 2004.</p>
<p>[6] A. D. J. Cross and E. R. Hancock. Graph matching with a dual-step EM algorithm. IEEE Trans. PAMI, 20(11):1236–1253, November 1998.</p>
<p>[7] S. J. Gaffney and P. Smyth. Curve clustering with random effects regression mixtures. In C. M. Bishop and B. J. Frey, editors, Proc. Ninth Inter. Workshop on Artiﬁcial Intelligence and Stats, Key West, FL, January 3–6 2003.</p>
<p>[8] D. Chudova, S. J. Gaffney, and P. J. Smyth. Probabilistic models for joint clustering and timewarping of multi-dimensional curves. In Proc. of the Nineteenth Conference on Uncertainty in Artiﬁcial Intelligence (UAI-2003), Acapulco, Mexico, August 7–10, 2003.</p>
<p>[9] D. Chudova, S. J. Gaffney, E. Mjolsness, and P. J. Smyth. Translation-invariant mixture models for curve clustering. In Proc. Ninth ACM SIGKDD Inter. Conf. on Knowledge Discovery and Data Mining, Washington D.C., August 24–27, New York, 2003. ACM Press.</p>
<p>[10] S. Gaffney and P. Smyth. Trajectory clustering with mixtures of regression models. In Surajit Chaudhuri and David Madigan, editors, Proc. Fifth ACM SIGKDD Inter. Conf. on Knowledge Discovery and Data Mining, August 15–18, pages 63–72, N.Y., 1999. ACM Press.</p>
<p>[11] P. H. C. Eilers. and B. D. Marx. Flexible smoothing with B-splines and penalties. Statistical Science, 11(2):89–121, 1996.</p>
<p>[12] A. Gelman, J. B. Carlin, H. S. Stern, and D. B. Rubin. Bayesian Data Analysis. Chapman & Hall, New York, NY, 1995.</p>
<p>[13] P. T. Spellman et al. Comprehensive identiﬁcation of cell cycle-regulated genes of the yeast Saccharomyces cerevisiae by microarray hybridization. Molec. Bio. Cell, 9(12):3273–3297, December 1998.</p>
<p>[14] J. Aach and G. M. Church. Aligning gene expression time series with time warping algorithms. Bioinformatics, 17(6):495–508, 2001.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
