<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>6 nips-2004-A Hidden Markov Model for de Novo Peptide Sequencing</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2004" href="../home/nips2004_home.html">nips2004</a> <a title="nips-2004-6" href="../nips2004/nips-2004-A_Hidden_Markov_Model_for_de_Novo_Peptide_Sequencing.html">nips2004-6</a> <a title="nips-2004-6-reference" href="#">nips2004-6-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>6 nips-2004-A Hidden Markov Model for de Novo Peptide Sequencing</h1>
<br/><p>Source: <a title="nips-2004-6-pdf" href="http://papers.nips.cc/paper/2668-a-hidden-markov-model-for-de-novo-peptide-sequencing.pdf">pdf</a></p><p>Author: Bernd Fischer, Volker Roth, Jonas Grossmann, Sacha Baginsky, Wilhelm Gruissem, Franz Roos, Peter Widmayer, Joachim M. Buhmann</p><p>Abstract: De novo Sequencing of peptides is a challenging task in proteome research. While there exist reliable DNA-sequencing methods, the highthroughput de novo sequencing of proteins by mass spectrometry is still an open problem. Current approaches suffer from a lack in precision to detect mass peaks in the spectrograms. In this paper we present a novel method for de novo peptide sequencing based on a hidden Markov model. Experiments effectively demonstrate that this new method significantly outperforms standard approaches in matching quality. 1</p><br/>
<h2>reference text</h2><p>[1] Sacha Baginsky, Mark Cieliebak, Wilhelm Gruissem, Torsten Kleffmann, Zsuzsanna Lipt´ k, Matthias M¨ ller, and Paolo Penna. Audens: A tool for automatic de novo a u peptide sequencing. Technical Report 383, ETH Zurich, Dept. of Computer Science, 2002.</p>
<p>[2] Ting Chen, Ming-Yang Kao, Matthew Tepel, John Rush, and George M. Church. A dynamic programming approach to de novo peptide sequencing via tandem mass spectrometry. Journal of Computational Biology, 8(3):325–337, 2001.</p>
<p>[3] Jimmy K. Eng, Ashley L. McCormack, and John R. Yates. An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database. American Society for Mass Spectrometry, 5(11):976–989, 1994.</p>
<p>[4] Andrew Keller, Alexey I. Nesvizhskii, Eugene Kolker, and Ruedi Aebersold. Empirical statistical model to estimate the accuracy of peptide identiﬁcations made by MS/MS and database search. Analytical Chemistry, 2002.</p>
<p>[5] Bin Ma, Kaizhong Zhang, Christopher Hendrie, Chengzhi Liang, Ming Li, Amanda Doherty-Kirby, and Gilles Lajoie. Peaks: Powerful software for peptide de novo sequencing by tandem mass spectrometry. Rapid Communication in Mass Spectrometry, 17(20):2337–2342, 2003.</p>
<p>[6] J. Alex Taylor and Richard S. Johnson. Implementation and uses of automated de novo peptide sequencing by tandem mass spectrometry. Analytical Chemistry, 73:2594– 2604, 2001.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
