<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>108 nips-2004-Markov Networks for Detecting Overalpping Elements in Sequence Data</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2004" href="../home/nips2004_home.html">nips2004</a> <a title="nips-2004-108" href="../nips2004/nips-2004-Markov_Networks_for_Detecting_Overalpping_Elements_in_Sequence_Data.html">nips2004-108</a> <a title="nips-2004-108-reference" href="#">nips2004-108-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>108 nips-2004-Markov Networks for Detecting Overalpping Elements in Sequence Data</h1>
<br/><p>Source: <a title="nips-2004-108-pdf" href="http://papers.nips.cc/paper/2546-markov-networks-for-detecting-overalpping-elements-in-sequence-data.pdf">pdf</a></p><p>Author: Mark Craven, Joseph Bockhorst</p><p>Abstract: Many sequential prediction tasks involve locating instances of patterns in sequences. Generative probabilistic language models, such as hidden Markov models (HMMs), have been successfully applied to many of these tasks. A limitation of these models however, is that they cannot naturally handle cases in which pattern instances overlap in arbitrary ways. We present an alternative approach, based on conditional Markov networks, that can naturally represent arbitrarily overlapping elements. We show how to eﬃciently train and perform inference with these models. Experimental results from a genomics domain show that our models are more accurate at locating instances of overlapping patterns than are baseline models based on HMMs. 1</p><br/>
<h2>reference text</h2><p>[1] J. Bockhorst, Y. Qiu, J. Glasner, M. Liu, F. Blattner, and M. Craven. Predicting bacterial transcription units using sequence and expression data. Bioinformatics, 19(Suppl. 1):i34–i43, 2003.</p>
<p>[2] M. Ermolaeva, H. Khalak, O. White, H. Smith, and S. Salzberg. Prediction of transcription terminators in bacterial genomes. J. of Molecular Biology, 301:27–33, 2000.</p>
<p>[3] P. Felzenszwalb and D. Huttenlocher. Eﬃcient matching of pictorial structures. In Proc. of the 2000 IEEE Conf. on Computer Vision and Pattern Recognition, 66–75.</p>
<p>[4] Z. Ghahramani and M. I. Jordan. Factorial hidden markov models. Machine Learning, 29:245–273, 1997.</p>
<p>[5] J. Laﬀerty, A. McCallum, and F. Pereira. Conditional random ﬁelds: Probabilistic models for segmenting and labeling sequence data. In Proc. of the 18th Internat. Conf. on Machine Learning, pages 282–289, Williamstown, MA, 2001. Morgan Kaufmann.</p>
<p>[6] R. Malouf. A comparison of algorithms for maximum entropy parameter estimation. Sixth workshop on computational language learning (CoNLL), 2002.</p>
<p>[7] National Institute of Standards and Technology. TREC video retrieval evaluation (TRECVID), 2004. http://www-nlpir.nist.gov/projects/t01v/.</p>
<p>[8] J. Pearl. Probabalistic Reasoning in Intelligent Systems: Networks of Plausible Inference. Morgan Kaufmann, San Mateo, CA, 1988.</p>
<p>[9] A. Pedersen, P. Baldi, S. Brunak, and Y. Chauvin. Characterization of prokaryotic and eukaryotic promoters using hidden Markov models. In Proc. of the 4th International Conf. on Intelligent Systems for Molecular Biology, pages 182–191, St. Louis, MO, 1996. AAAI Press.</p>
<p>[10] B. Taskar, P. Abbeel, and D. Koller. Discriminative probabilistic models for relational data. In Proc. of the 18th International Conf. on Uncertainty in Artiﬁcial Intelligence, Edmonton, Alberta, 2002. Morgan Kaufmann.</p>
<p>[11] T. Yada, Y. Totoki, T. Takagi, and K. Nakai. A novel bacterial gene-ﬁnding system with improved accuracy in locating start codons. DNA Research, 8(3):97–106, 2001.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
