<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>95 nips-2004-Large-Scale Prediction of Disulphide Bond Connectivity</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2004" href="../home/nips2004_home.html">nips2004</a> <a title="nips-2004-95" href="../nips2004/nips-2004-Large-Scale_Prediction_of_Disulphide_Bond_Connectivity.html">nips2004-95</a> <a title="nips-2004-95-reference" href="#">nips2004-95-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>95 nips-2004-Large-Scale Prediction of Disulphide Bond Connectivity</h1>
<br/><p>Source: <a title="nips-2004-95-pdf" href="http://papers.nips.cc/paper/2607-large-scale-prediction-of-disulphide-bond-connectivity.pdf">pdf</a></p><p>Author: Jianlin Cheng, Alessandro Vullo, Pierre F. Baldi</p><p>Abstract: The formation of disulphide bridges among cysteines is an important feature of protein structures. Here we develop new methods for the prediction of disulphide bond connectivity. We ﬁrst build a large curated data set of proteins containing disulphide bridges and then use 2-Dimensional Recursive Neural Networks to predict bonding probabilities between cysteine pairs. These probabilities in turn lead to a weighted graph matching problem that can be addressed efﬁciently. We show how the method consistently achieves better results than previous approaches on the same validation data. In addition, the method can easily cope with chains with arbitrary numbers of bonded cysteines. Therefore, it overcomes one of the major limitations of previous approaches restricting predictions to chains containing no more than 10 oxidized cysteines. The method can be applied both to situations where the bonded state of each cysteine is known or unknown, in which case bonded state can be predicted with 85% precision and 90% recall. The method also yields an estimate for the total number of disulphide bridges in each chain. 1</p><br/>
<h2>reference text</h2><p>[1] V.I. Abkevich and E.I. Shankhnovich. What can disulﬁde bonds tell us about protein energetics, function and folding: simulations and bioinformatics analysis. J. Math. Biol., 300:975–985, 2000.</p>
<p>[2] A. Bairoch and R. Apweiler. The SWISS-PROT protein sequence database and its supplement TrEMBL. Nucleic Acids Res., 28:45–48, 2000.</p>
<p>[3] P. Baldi and G. Pollastri. Machine learning structural and functional proteomics. IEEE Intelligent Systems. Special Issue on Intelligent Systems in Biology, 17(2), 2002.</p>
<p>[4] P. Baldi and G. Pollastri. The principled design of large-scale recursive neural network architectures–dag-rnns and the protein structure prediction problem. Journal of Machine Learning Research, 4:575–602, 2003.</p>
<p>[5] P. Baldi and M. Rosen-Zvi. On the relationship between deterministic and probabilistic directed graphical models. 2004. Submitted.</p>
<p>[6] H. M. Berman, J. Westbrook, Z. Feng, G. Gilliland, T. N. Bhat, H. Weissig, I. N. Shindyalov, and P. E. Bourne. The Protein Data Bank. Nucl. Acids Res., 28:235–242, 2000.</p>
<p>[7] S. Betz. Disulﬁde bonds and the stability of globular proteins. Proteins, Struct., Function Genet., 21:167–195, 1993.</p>
<p>[8] J. Clarke and A.R. Fersht. Engineered disulﬁde bonds as probes of the folding pathway of barnase - increasing stability of proteins against the rate of denaturation. Biochemistry, 32:4322– 4329, 1993.</p>
<p>[9] L. Demetrius. Thermodynamics and kinetics of protein folding: an evolutionary perpective. J. Theor. Biol., 217:397–411, 2000.</p>
<p>[10] P. Fariselli and R. Casadio. Prediction of disulﬁde connectivity in proteins. Bioinformatics, 17:957–964, 2001.</p>
<p>[11] P. Fariselli, P. L. Martelli, and R. Casadio. A neural network-based method for predicting the disulﬁde connectivity in proteins. In E. Damiani et al., editors, Knowledge based intelligent information engineering systems and allied technologies (KES 2002), volume 1, pages 464– 468. IOS Press, 2002.</p>
<p>[12] H.N. Gabow. An efﬁcient implementation of Edmond’s algorithm for maximum weight matching on graphs. Journal of the ACM, 23(2):221–234, 1976.</p>
<p>[13] J.L. Klepeis and C.A. Floudas. Prediction of β-sheet topology and disulﬁde bridges in polypeptides. J. Comput. Chem., 24:191–208, 2003.</p>
<p>[14] T.A. Klink, K.J. Woycechosky, K.M. Taylor, and R.T. Raines. Contribution of disulﬁde bonds to the conformational stability and catalytic activity of ribonuclease A. Eur. J. Biochem., 267:566– 572, 2000.</p>
<p>[15] M. Matsumura et al. Substantial increase of protein stability by multiple disulﬁde bonds. Nature, 342:291–293, 1989.</p>
<p>[16] G. Pollastri, D. Przybylski, B. Rost, and P. Baldi. Improving the prediction of protein secondary structure in three and eight classes using recurrent neural networks and proﬁles. Proteins, 47:228–235, 2002.</p>
<p>[17] A. Vullo and P. Frasconi. Disulﬁde connectivity prediction using recursive neural networks and evolutionary information. Bioinformatics, 20:653–659, 2004.</p>
<p>[18] W.J. Wedemeyer, E. Welkler, M. Narayan, and H.A. Scheraga. Disulﬁde bonds and proteinfolding. Biochemistry, 39:4207–4216, 2000.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
