<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>184 cvpr-2013-Gauging Association Patterns of Chromosome Territories via Chromatic Median</title>
</head>

<body>
<p><a title="cvpr" href="../cvpr_home.html">cvpr</a> <a title="cvpr-2013" href="../home/cvpr2013_home.html">cvpr2013</a> <a title="cvpr-2013-184" href="../cvpr2013/cvpr-2013-Gauging_Association_Patterns_of_Chromosome_Territories_via_Chromatic_Median.html">cvpr2013-184</a> <a title="cvpr-2013-184-reference" href="#">cvpr2013-184-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>184 cvpr-2013-Gauging Association Patterns of Chromosome Territories via Chromatic Median</h1>
<br/><p>Source: <a title="cvpr-2013-184-pdf" href="http://www.cv-foundation.org/openaccess/content_cvpr_2013/papers/Ding_Gauging_Association_Patterns_2013_CVPR_paper.pdf">pdf</a></p><p>Author: Hu Ding, Branislav Stojkovic, Ronald Berezney, Jinhui Xu</p><p>Abstract: Computing accurate and robust organizational patterns of chromosome territories inside the cell nucleus is critical for understanding several fundamental genomic processes, such as co-regulation of gene activation, gene silencing, X chromosome inactivation, and abnormal chromosome rearrangement in cancer cells. The usage of advanced fluorescence labeling and image processing techniques has enabled researchers to investigate interactions of chromosome territories at large spatial resolution. The resulting high volume of generated data demands for high-throughput and automated image analysis methods. In this paper, we introduce a novel algorithmic tool for investigating association patterns of chromosome territories in a population of cells. Our method takes as input a set of graphs, one for each cell, containing information about spatial interaction of chromosome territories, and yields a single graph that contains essential information for the whole population and stands as its structural representative. We formulate this combinato- rial problem as a semi-definite programming and present novel techniques to efficiently solve it. We validate our approach on both artificial and real biological data; the experimental results suggest that our approach yields a nearoptimal solution, and can handle large-size datasets, which are significant improvements over existing techniques.</p><br/>
<h2>reference text</h2><p>[1] Sanjeev Arora: Semidefinite Programming and Approximation Algorithms: A Survey. ISAAC, 6-9, 2011.</p>
<p>[2] H. A. Almohamad and S. O. Duffuaa. A linear programming approach for the weighted graph matching problem. IEEE PAMI, 15(5):522-525, 1993.</p>
<p>[3] R. Berezney. Regulating the mammalian genome: the role of nuclear architecture. Advances in Enzyme Regulation, 42:3952, 2002.</p>
<p>[4] N. Bansal, A. Blum and S. Chawla: Correlation Clustering. FOCS, 238, 2002.</p>
<p>[5] S. Boyle, S. Gilchrist, J. Bridger, N. Mahy, J. Ellis, and W. Bickmore. The spatial organization of human chromosomes within the nuclei of normal and emerin-mutant cells. Hum. Mol. Genet., 10(3):21 1-219, 2001.</p>
<p>[6] A. Bolzer, G. Kreth, I. Solovei, D. Koehler, K. Saracoglu, C. Fauth, S. Mller, R. Eils, C. Cremer, M. Speicher, and T. Cremer. Three-dimensional maps of all chromosomes in human male fibroblast nuclei and prometaphase rosettes. PLoS. Biol., 3(5):826-842, 2005.</p>
<p>[7] K. Clarkson, Nearest-Neighbor Searching and Metric Space Dimensions. survey, 2005</p>
<p>[8] J. Croft, J. Bridger, S. Boyle, P. Perry, P. Teague, W. Bickmore. Differences in the localization and morphology of chromosomes in the human nucleus. J. Cell. Biol., 145(6): 11191131, 1999.</p>
<p>[9] T. Cremer and M. Cremer. Chromosome territories, nuclear architecture and gene regulation in mamalian cells. Nature Reviews Genetics, 2:292-301, 2001 .</p>
<p>[10] T. Cremer, M. Cremer, S. Dietzel, S. Mller, I. Solovei, and S. Fakan. Chromosome territoriesâ€“a functional nuclear land-  scape. Curr. Opin. Cell. Biol., 18(3):307-316, 2006.</p>
<p>[11] M. Ferrer, D. Karatzas, E. Valveny, I. Bardaj, and H. Bunke. A generic framework for median graph computation based on a recursive embedding approach. CVIU, 115(7):919-928, 2011</p>
<p>[12] M. Ferrer, E. Valveny, F. Serratosa, K. Riesen, and H. Bunke. Generalized median graph computation by means of graph embedding in vector spaces. Pattern Recognition, 43(4): 16421655, 2010.</p>
<p>[13] M. X. Goemans and D. P. Williamson: Improved Approximation Algorithms for Maximum Cut and Satisfiability Problems Using Semidefinite Programming. J. ACM 42(6): 11151145, 1995.</p>
<p>[14] A. Hlaoui and S. Wang. Median graph computation for graph clustering. Soft. Comp., 10(1):47-53, 2006.</p>
<p>[15] D. Justice and A. Hero. A binary linear programming formulation of the graph edit distance. IEEE PAMI, 28(8): 12001214, 2006.</p>
<p>[16] X. Jiang, A. Munger, and H. Bunke. On Median Graphs: Properties, Algorithms, and Applications. IEEE Trans. Pattern Anal. Mach. Intell., 23(10): 1144-1 15 1, 2001 .</p>
<p>[17] S. Khot, and R. Saket. Hardness of Embedding Metric Spaces of Equal Size. APPROX-RANDOM, 218-227, 2007.</p>
<p>[18] L. Mukherjee, V. Singh, J. Peng, J. Xu, M. J. Zeitz, and R. Berezney. Generalized Median Graphs: Theory and Applications. ICCV, 1-8, 2007.</p>
<p>[19] S. Sahni. Computationally Related Problems. SIAM J. Comput., 3(4):262-279 , 1974.</p>
<p>[20] B. Stojkovic, Y. Zhu, J. Xu, A. Fritz, M. J. Zeitz, J. Vecerova, and R. Berezney. Computing Maximum Association Graph in Microscopic Nucleus Images. MICCAI, (2):530-537, 2010</p>
<p>[21] E. Weiszfeld. On the point for which the sum of the distances to n given points is minimum. Tohoku. Math. Journal., 43:355-386, 1937.</p>
<p>[22] M. J. Zeitz, L. Mukherjee, S. Bhattacharya, J. Xu, and R. Berezney. A probabilistic model for the arrangement of a subset of human chromosome territories in wi38 human fibroblasts. Journal of Cellular Physiology, 221(1): 120-129, 2009 11111333330000013311</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
