<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>225 nips-2008-Supervised Bipartite Graph Inference</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2008" href="../home/nips2008_home.html">nips2008</a> <a title="nips-2008-225" href="../nips2008/nips-2008-Supervised_Bipartite_Graph_Inference.html">nips2008-225</a> <a title="nips-2008-225-reference" href="#">nips2008-225-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>225 nips-2008-Supervised Bipartite Graph Inference</h1>
<br/><p>Source: <a title="nips-2008-225-pdf" href="http://papers.nips.cc/paper/3428-supervised-bipartite-graph-inference.pdf">pdf</a></p><p>Author: Yoshihiro Yamanishi</p><p>Abstract: We formulate the problem of bipartite graph inference as a supervised learning problem, and propose a new method to solve it from the viewpoint of distance metric learning. The method involves the learning of two mappings of the heterogeneous objects to a uniﬁed Euclidean space representing the network topology of the bipartite graph, where the graph is easy to infer. The algorithm can be formulated as an optimization problem in a reproducing kernel Hilbert space. We report encouraging results on the problem of compound-protein interaction network reconstruction from chemical structure data and genomic sequence data. 1</p><br/>
<h2>reference text</h2><p>[1] C.M. Dobson. Chemical space and biology. Nature, 432:824–828, 2004.</p>
<p>[2] M. Rarey, B. Kramer, T. Lengauer, and G. Klebe. A fast ﬂexible docking method using an incremental construction algorithm. J Mol Biol, 261:470–489, 1996.</p>
<p>[3] Y. Yamanishi, J.P. Vert, and M. Kanehisa. Protein network inference from multiple genomic data: a supervised approach. Bioinformatics, 20 Suppl 1:i363–370, 2004.</p>
<p>[4] J.-P. Vert and Y. Yamanishi. Supervised graph inference. Advances in Neural Information and Processing System, pages 1433–1440, 2005.</p>
<p>[5] T. Kato, K. Tsuda, and K. Asai. Selective integration of multiple biological data for supervised network inference. Bioinformatics, 21:2488–2495, 2005.</p>
<p>[6] B. Sch¨ lkopf, K. Tsuda, and J.P. Vert. Kernel Methods in Computational Biology. MIT Press, 2004. o</p>
<p>[7] G. Wahba. Splines Models for Observational Data: Series in Applied Mathematics. SIAM, Philadelphia, 1990.</p>
<p>[8] F. Girosi, M. Jones, and T. Poggio. Regularization theory and neural networks architectures. Neural Computation, 7:219–269, 1995.</p>
<p>[9] J. Shawe-Taylor and N. Cristianini. Kernel Methods for Pattern Analysis. Camb. Univ. Press, 2004.</p>
<p>[10] M.J. Greenacre. Theory and applications of correspondence analysis. Academic Press, 1984.</p>
<p>[11] A. Globerson, G. Chechik, F. Pereira, and N. Tishby. Euclidean embedding of co-occurrence data. Advances in Neural Information and Processing System, pages 497–504, 2005.</p>
<p>[12] S. Akaho. A kernel method for canonical correlation analysis. International. Meeting on Psychometric Society (IMPS2001), 2001.</p>
<p>[13] F.R. Bach and M.I. Jordan. Kernel independent component analysis. Journal of Machine Learning Research, 3:1–48, 2002.</p>
<p>[14] M. Kanehisa, S. Goto, M. Hattori, K.F. Aoki-Kinoshita, M. Itoh, S. Kawashima, T. Katayama, M. Araki, and M. Hirakawa. From genomics to chemical genomics: new developments in kegg. Nucleic Acids Res., 34(Database issue):D354–357, Jan 2006.</p>
<p>[15] S. Gunther, S. Guenther, M. Kuhn, M. Dunkel, and et al. Supertarget and matador: resources for exploring drug-target relationships. Nucleic Acids Res, 2007.</p>
<p>[16] D.S. Wishart, C. Knox, A.C. Guo, D. Cheng, S. Shrivastava, D. Tzur, B. Gautam, and M. Hassanali. Drugbank: a knowledgebase for drugs, drug actions and drug targets. Nucleic Acids Res, 2007.</p>
<p>[17] M. Hattori, Y. Okuno, S. Goto, and M. Kanehisa. Development of a chemical structure comparison method for integrated analysis of chemical and genomic information in the metabolic pathways. J. Am. Chem. Soc., 125:11853–11865, 2003.</p>
<p>[18] T.F. Smith and M.S. Waterman. Identiﬁcation of common molecular subsequences. J Mol Biol, 147:195– 197, 1981.</p>
<p>[19] H. Saigo, J.P. Vert, N. Ueda, and T. Akutsu. Protein homology detection using string alignment kernels. Bioinformatics, 20:1682–1689, 2004.  8</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
