<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>152 nips-2008-Non-stationary dynamic Bayesian networks</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2008" href="../home/nips2008_home.html">nips2008</a> <a title="nips-2008-152" href="../nips2008/nips-2008-Non-stationary_dynamic_Bayesian_networks.html">nips2008-152</a> <a title="nips-2008-152-reference" href="#">nips2008-152-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>152 nips-2008-Non-stationary dynamic Bayesian networks</h1>
<br/><p>Source: <a title="nips-2008-152-pdf" href="http://papers.nips.cc/paper/3571-non-stationary-dynamic-bayesian-networks.pdf">pdf</a></p><p>Author: Joshua W. Robinson, Alexander J. Hartemink</p><p>Abstract: A principled mechanism for identifying conditional dependencies in time-series data is provided through structure learning of dynamic Bayesian networks (DBNs). An important assumption of DBN structure learning is that the data are generated by a stationary process—an assumption that is not true in many important settings. In this paper, we introduce a new class of graphical models called non-stationary dynamic Bayesian networks, in which the conditional dependence structure of the underlying data-generation process is permitted to change over time. Non-stationary dynamic Bayesian networks represent a new framework for studying problems in which the structure of a network is evolving over time. We deﬁne the non-stationary DBN model, present an MCMC sampling algorithm for learning the structure of the model from time-series data under different assumptions, and demonstrate the effectiveness of the algorithm on both simulated and biological data. 1</p><br/>
<h2>reference text</h2><p>[1] Nir Friedman, Michal Linial, Iftach Nachman, and Dana Pe’er. Using Bayesian networks to analyze expression data. In RECOMB 4, pages 127–135. ACM Press, 2000.</p>
<p>[2] V. Anne Smith, Jing Yu, Tom V. Smulders, Alexander J. Hartemink, and Erich D. Jarvis. Computational inference of neural information ﬂow networks. PLoS Computational Biology, 2(11):1436–1449, 2006.</p>
<p>[3] Steve Hanneke and Eric P. Xing. Discrete temporal models of social networks. In Workshop on Statistical Network Analysis, ICML 23, 2006.</p>
<p>[4] Fan Guo, Steve Hanneke, Wenjie Fu, and Eric P. Xing. Recovering temporally rewiring networks: A model-based approach. In ICML 24, 2007.</p>
<p>[5] Makram Talih and Nicolas Hengartner. Structural learning with time-varying components: Tracking the cross-section of ﬁnancial time series. Journal of the Royal Statistical Society B, 67(3):321–341, 2005.</p>
<p>[6] Xiang Xuan and Kevin Murphy. Modeling changing dependency structure in multivariate time series. In ICML 24, 2007.</p>
<p>[7] David Heckerman, Dan Geiger, and David Maxwell Chickering. Learning Bayesian networks: The combination of knowledge and statistical data. Machine Learning, 20(3):197–243, 1995.</p>
<p>[8] Claudia Tarantola. MCMC model determination for discrete graphical models. Statistical Modelling, 4(1):39–61, 2004.</p>
<p>[9] P Krause. Learning probabilistic networks. The Knowledge Engineering Review, 13(4):321–351, 1998.</p>
<p>[10] Kevin Murphy. Learning Bayesian network structure from sparse data sets. U.C. Berkeley Technical Report, Computer Science Department 990, University of California at Berkeley, 2001.</p>
<p>[11] Peter J. Green. Reversible jump Markov chain Monte Carlo computation and Bayesian model determination. Biometrika, 82(4):711–732, 1995.</p>
<p>[12] M Arbeitman, E Furlong, F Imam, E Johnson, B Null, B Baker, M Krasnow, M Scott, R Davis, and K White. Gene expression during the life cycle of Drosophila melanogaster. Science, 5590(297):2270– 2275, 2002.</p>
<p>[13] Wentao Zhao, Erchin Serpedin, and Edward R. Dougherty. Inferring gene regulatory networks from time series data using the minimum description length principle. Bioinformatics, 22(17):2129–2135, 2006.</p>
<p>[14] T Sandmann, L Jensen, J Jakobsen, M Karzynski, M Eichenlaub, P Bork, and E Furlong. A temporal map of transcription factor activity: mef2 directly regulates target genes at all stages of muscle development. Developmental Cell, 10(6):797–807, 2006.  8</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
