<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>11 nips-2008-A spatially varying two-sample recombinant coalescent, with applications to HIV escape response</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2008" href="../home/nips2008_home.html">nips2008</a> <a title="nips-2008-11" href="../nips2008/nips-2008-A_spatially_varying_two-sample_recombinant_coalescent%2C_with_applications_to_HIV_escape_response.html">nips2008-11</a> <a title="nips-2008-11-reference" href="#">nips2008-11-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>11 nips-2008-A spatially varying two-sample recombinant coalescent, with applications to HIV escape response</h1>
<br/><p>Source: <a title="nips-2008-11-pdf" href="http://papers.nips.cc/paper/3603-a-spatially-varying-two-sample-recombinant-coalescent-with-applications-to-hiv-escape-response.pdf">pdf</a></p><p>Author: Alexander Braunstein, Zhi Wei, Shane T. Jensen, Jon D. Mcauliffe</p><p>Abstract: Statistical evolutionary models provide an important mechanism for describing and understanding the escape response of a viral population under a particular therapy. We present a new hierarchical model that incorporates spatially varying mutation and recombination rates at the nucleotide level. It also maintains separate parameters for treatment and control groups, which allows us to estimate treatment effects explicitly. We use the model to investigate the sequence evolution of HIV populations exposed to a recently developed antisense gene therapy, as well as a more conventional drug therapy. The detection of biologically relevant and plausible signals in both therapy studies demonstrates the effectiveness of the method. 1</p><br/>
<h2>reference text</h2><p>[1] P. J. Green. Reversible jump Markov chain Monte Carlo computation and Bayesian model determination. Biometrika, 82:711–731, 1995.</p>
<p>[2] R. C. Grifﬁths and P. Marjoram. An ancestral recombination graph. In Progress in Population Genetics and Human Evolution, pages 257–270. Springer Verlag, 1997.</p>
<p>[3] J. Hein, M. Schierup, and C. Wiuf. Gene Genealogies, Variation and Evolution: A Primer in Coalescent Theory. Oxford University Press, 2005.</p>
<p>[4] R. R. Hudson. Properties of a neutral allele model with intragenic recombination. Theoretical Population Biology, 23:183–201, 1983.</p>
<p>[5] J. F. C. Kingman. The coalescent. Stochastic Processes and Their Applications, 13:235–248, 1982.</p>
<p>[6] N. Li and M. Stephens. Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data. Genetics, 165:2213–2233, December 2003.</p>
<p>[7] X. Lu, Q. Yu, G. Binder, Z. Chen, T. Slepushkina, J. Rossi, and B. Dropulic. Antisensemediated inhibition of human immunodeﬁciency virus (HIV) replication by use of an HIV type 1-based vector results in severely attenuated mutants incapable of developing resistance. Journal of Virology, 78:7079–7088, 2004.</p>
<p>[8] G. McGuire, M. Denham, and D. Balding. Models of sequence evolution for DNA sequences containing gaps. Molecular Biology and Evolution, 18(4):481–490, 2001.</p>
<p>[9] N. Ray, J. Harrison, L. Blackburn, J. Martin, S. Deeks, and R. Doms. Clinical resistance to enfuvirtide does not affect susceptibility of human immunodeﬁciency virus type 1 to other classes of entry inhibitors. Journal of Virology, 81:3240–3250, 2007.</p>
<p>[10] M. H. Schierup and J. Hein. Consequences of recombination on traditional phylogenetic analysis. Genetics, 156:879–891, 2000.</p>
<p>[11] C. Wild, T. Greenwell, and T. Matthews. A synthetic peptide from HIV-1 gp41 is a potent inhibitor of virus mediated cell-cell fusion. AIDS Research and Human Retroviruses, 9:1051– 1053, 1993.</p>
<p>[12] D. Wilson and G. McVean. Estimating diversifying selection and functional constraint in the presence of recombination. Genetics, 172:1411–1425, 2006. 8</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
