<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>203 nips-2008-Scalable Algorithms for String Kernels with Inexact Matching</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2008" href="../home/nips2008_home.html">nips2008</a> <a title="nips-2008-203" href="../nips2008/nips-2008-Scalable_Algorithms_for_String_Kernels_with_Inexact_Matching.html">nips2008-203</a> <a title="nips-2008-203-reference" href="#">nips2008-203-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>203 nips-2008-Scalable Algorithms for String Kernels with Inexact Matching</h1>
<br/><p>Source: <a title="nips-2008-203-pdf" href="http://papers.nips.cc/paper/3441-scalable-algorithms-for-string-kernels-with-inexact-matching.pdf">pdf</a></p><p>Author: Pavel P. Kuksa, Pai-hsi Huang, Vladimir Pavlovic</p><p>Abstract: We present a new family of linear time algorithms for string comparison with mismatches under the string kernels framework. Based on sufﬁcient statistics, our algorithms improve theoretical complexity bounds of existing approaches while scaling well in sequence alphabet size, the number of allowed mismatches and the size of the dataset. In particular, on large alphabets and under loose mismatch constraints our algorithms are several orders of magnitude faster than the existing algorithms for string comparison under the mismatch similarity measure. We evaluate our algorithms on synthetic data and real applications in music genre classiﬁcation, protein remote homology detection and protein fold prediction. The scalability of the algorithms allows us to consider complex sequence transformations, modeled using longer string features and larger numbers of mismatches, leading to a state-of-the-art performance with signiﬁcantly reduced running times. 1</p><br/>
<h2>reference text</h2><p>[1] Jianlin Cheng and Pierre Baldi. A machine learning information retrieval approach to protein fold recognition. Bioinformatics, 22(12):1456–1463, June 2006.</p>
<p>[2] Rui Kuang, Eugene Ie, Ke Wang, Kai Wang, Mahira Siddiqi, Yoav Freund, and Christina S. Leslie. Proﬁle-based string kernels for remote homology detection and motif extraction. In CSB, pages 152– 160, 2004.</p>
<p>[3] Christina S. Leslie, Eleazar Eskin, Jason Weston, and William Stafford Noble. Mismatch string kernels for SVM protein classiﬁcation. In NIPS, pages 1417–1424, 2002.</p>
<p>[4] S¨ ren Sonnenburg, Gunnar R¨ tsch, and Bernhard Sch¨ lkopf. Large scale genomic sequence SVM claso a o siﬁers. In ICML ’05, pages 848–855, New York, NY, USA, 2005.</p>
<p>[5] John Shawe-Taylor and Nello Cristianini. Kernel Methods for Pattern Analysis. Cambridge University Press, New York, NY, USA, 2004.</p>
<p>[6] Christina Leslie and Rui Kuang. Fast string kernels using inexact matching for protein sequences. J. Mach. Learn. Res., 5:1435–1455, 2004.</p>
<p>[7] Jason Weston, Christina Leslie, Eugene Ie, Dengyong Zhou, Andre Elisseeff, and William Stafford Noble. Semi-supervised protein classiﬁcation using cluster kernels. Bioinformatics, 21(15):3241–3247, 2005.</p>
<p>[8] Vladimir N. Vapnik. Statistical Learning Theory. Wiley-Interscience, September 1998.</p>
<p>[9] Christina S. Leslie, Eleazar Eskin, and William Stafford Noble. The spectrum kernel: A string kernel for SVM protein classiﬁcation. In Paciﬁc Symposium on Biocomputing, pages 566–575, 2002.</p>
<p>[10] S. V. N. Vishwanathan and Alex Smola. Fast kernels for string and tree matching. Advances in Neural Information Processing Systems, 15, 2002.</p>
<p>[11] M. Gribskov, A.D. McLachlan, and D. Eisenberg. Proﬁle analysis: detection of distantly related proteins. Proceedings of the National Academy of Sciences, 84:4355–4358, 1987.</p>
<p>[12] Juho Rousu and John Shawe-Taylor. Efﬁcient computation of gapped substring kernels on large alphabets. J. Mach. Learn. Res., 6:1323–1344, 2005.</p>
<p>[13] Pavel Kuksa, Pai-Hsi Huang, and Vladimir Pavlovic. Fast protein homology and fold detection with sparse spatial sample kernels. In ICPR 2008, 2008.</p>
<p>[14] Chris H.Q. Ding and Inna Dubchak. Multi-class protein fold recognition using support vector machines and neural networks. Bioinformatics, 17(4):349–358, 2001.</p>
<p>[15] Iain Melvin, Eugene Ie, Jason Weston, William Stafford Noble, and Christina Leslie. Multi-class protein classiﬁcation using adaptive codes. J. Mach. Learn. Res., 8:1557–1581, 2007.</p>
<p>[16] Tao Li, Mitsunori Ogihara, and Qi Li. A comparative study on content-based music genre classiﬁcation. In SIGIR ’03, pages 282–289, New York, NY, USA, 2003. ACM.</p>
<p>[17] Pavel Kuksa, Pai-Hsi Huang, and Vladimir Pavlovic. On the role of local matching for efﬁcient semisupervised protein sequence classiﬁcation. In BIBM, 2008.</p>
<p>[18] Tommi Jaakkola, Mark Diekhans, and David Haussler. A discriminative framework for detecting remote protein homologies. In Journal of Computational Biology, volume 7, pages 95–114, 2000.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
