<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>9 nips-2008-A mixture model for the evolution of gene expression in non-homogeneous datasets</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2008" href="../home/nips2008_home.html">nips2008</a> <a title="nips-2008-9" href="../nips2008/nips-2008-A_mixture_model_for_the_evolution_of_gene_expression_in_non-homogeneous_datasets.html">nips2008-9</a> <a title="nips-2008-9-reference" href="#">nips2008-9-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>9 nips-2008-A mixture model for the evolution of gene expression in non-homogeneous datasets</h1>
<br/><p>Source: <a title="nips-2008-9-pdf" href="http://papers.nips.cc/paper/3384-a-mixture-model-for-the-evolution-of-gene-expression-in-non-homogeneous-datasets.pdf">pdf</a></p><p>Author: Gerald Quon, Yee W. Teh, Esther Chan, Timothy Hughes, Michael Brudno, Quaid D. Morris</p><p>Abstract: We address the challenge of assessing conservation of gene expression in complex, non-homogeneous datasets. Recent studies have demonstrated the success of probabilistic models in studying the evolution of gene expression in simple eukaryotic organisms such as yeast, for which measurements are typically scalar and independent. Models capable of studying expression evolution in much more complex organisms such as vertebrates are particularly important given the medical and scientiﬁc interest in species such as human and mouse. We present Brownian Factor Phylogenetic Analysis, a statistical model that makes a number of signiﬁcant extensions to previous models to enable characterization of changes in expression among highly complex organisms. We demonstrate the efﬁcacy of our method on a microarray dataset proﬁling diverse tissues from multiple vertebrate species. We anticipate that the model will be invaluable in the study of gene expression patterns in other diverse organisms as well, such as worms and insects. 1</p><br/>
<h2>reference text</h2><p>[1] Li, W., Yang, J., Gu, X. (2005) Expression divergence between duplicate genes. Trends Genet., 21, 602-607.</p>
<p>[2] Chen, K., Rajewsky, N. (2007) The evolution of gene regulation by transcription factors and microRNAs. Nature Rev. Genet., 8, 93-103.</p>
<p>[3] Yergeau, D.A. et al. (2005) bloodthirsty, an RBCC/TRIM gene required for erythropoiesis in zebraﬁsh. Dev. Biol., 283, 97-112.</p>
<p>[4] Stuart, J.M., Segal, E., Koller, D., Kim, S.K. (2003) A gene-coexpression network for global discovery of conserved genetic modules. Science, 302, 249-255.</p>
<p>[5] Tirosh, I., Weinberger, A., Carmi, M., Barkai, N. (2006) A genetic signature of interspecies variations in gene expression. Nat. Genet., 38, 830-834.</p>
<p>[6] Khaitovich, P. et al. (2005) A neutral model of transcriptome evolution. PLoS. Biol., 2, 682-689.</p>
<p>[7] Ghahramani, Z., & Hinton, G.E. (1996) The EM algorithm for mixtures of factor analyzers. Technical Report CRG-TR-96-2, University of Toronto.</p>
<p>[8] Gu, X. (2004) Statistical framework for phylogenomic analysis of gene family expression proﬁles. Genetics, 167, 531-542.</p>
<p>[9] Oakley, T.H. et al. (2005) Comparative methods for the analysis of gene-expression evolution: an example using yeast functional genomic data. Mol. Biol. Evol., 22, 40-50.</p>
<p>[10] Felsenstein, J. (2004) Inferring phylogenies. Sunderland (Massachusetts): Sinauer Associates. 664 p.</p>
<p>[11] Felsenstein, J. (1981) Evolutionary trees from gene-frequencies and quantitative characters - ﬁnding maximum likelihood estimates. Evolution, 35, 1229-1242.</p>
<p>[12] Khaitovich et al. (2006) Evolution of primate gene expression. Nat. Rev. Genet., 7, 693-702.</p>
<p>[13] Yang, Z. (1994) Maximum likelihood phylogenetic estimation from DNA sequences with variable rates over sites: approximate methods. J. Mol. Evol., 39, 306-314.</p>
<p>[14] Kardong, K.V. (2006) Vertebrates: comparative anatomy, function, evolution. McGraw-Hill. 782 p.</p>
<p>[15] Zhang, W., Morris, Q.D. et al. (2004) The functional landscape of mouse gene expression. J. Biol., 3, 21.</p>
<p>[16] Huber, W. et al. (2002) Variance stabilization applied to microarray data calibration and to the quantiﬁcation of differential expression. Bioinformatics, 18, S96-104.</p>
<p>[17] The Gene Ontology Consortium. (2000) Gene Ontology: tool for the uniﬁcation of biology. Nature Genet., 25, 25-29.</p>
<p>[18] Edgar, R.C. (2004) MUSCLE: multiple sequence alignment with high accuracy and high throughput. Nucleic Acids Res., 32, 1792-1797.</p>
<p>[19] Yang, Z. (2007) PAML 4: phylogenetic analysis by maximum likelihood. Mol. Biol. Evol., 24, 1586-1591.  8</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
