<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>183 nips-2008-Predicting the Geometry of Metal Binding Sites from Protein Sequence</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2008" href="../home/nips2008_home.html">nips2008</a> <a title="nips-2008-183" href="../nips2008/nips-2008-Predicting_the_Geometry_of_Metal_Binding_Sites_from_Protein_Sequence.html">nips2008-183</a> <a title="nips-2008-183-reference" href="#">nips2008-183-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>183 nips-2008-Predicting the Geometry of Metal Binding Sites from Protein Sequence</h1>
<br/><p>Source: <a title="nips-2008-183-pdf" href="http://papers.nips.cc/paper/3498-predicting-the-geometry-of-metal-binding-sites-from-protein-sequence.pdf">pdf</a></p><p>Author: Paolo Frasconi, Andrea Passerini</p><p>Abstract: Metal binding is important for the structural and functional characterization of proteins. Previous prediction efforts have only focused on bonding state, i.e. deciding which protein residues act as metal ligands in some binding site. Identifying the geometry of metal-binding sites, i.e. deciding which residues are jointly involved in the coordination of a metal ion is a new prediction problem that has been never attempted before from protein sequence alone. In this paper, we formulate it in the framework of learning with structured outputs. Our solution relies on the fact that, from a graph theoretical perspective, metal binding has the algebraic properties of a matroid, enabling the application of greedy algorithms for learning structured outputs. On a data set of 199 non-redundant metalloproteins, we obtained precision/recall levels of 75%/46% correct ligand-ion assignments, which improves to 88%/88% in the setting where the metal binding state is known. 1</p><br/>
<h2>reference text</h2><p>[1] M. Babor, S. Gerzon, B. Raveh, V. Sobolev, and M. Edelman. Prediction of transition metal-binding sites from apo protein structures. Proteins, 70(1):208–217, 2008.</p>
<p>[2] G. Bakir, T. Hofmann, B. Sch¨ lkopf, A. Smola, B. Taskar, and S. Vishwanathan, editors. Predicting o Structured Data. The MIT Press, 2007.</p>
<p>[3] A. Bordes, S. Ertekin, J. Weston, and L. Bottou. Fast kernel classiﬁers with online and active learning. Journal of Machine Learning Research, 6:1579–1619, 2005.</p>
<p>[4] H. Daume III and D. Marcu. Learning as search optimization: Approximate large margin methods for structured prediction. In Proc. of the 22nd Int. Conf. on Machine Learning (ICML’05), 2005.</p>
<p>[5] J. C. Ebert and R. B. Altman. Robust recognition of zinc binding sites in proteins. Protein Sci, 17(1):54– 65, 2008.</p>
<p>[6] F. Ferr` and P. Clote. DiANNA 1.1: an extension of the DiANNA web server for ternary cysteine classie ﬁcation. Nucleic Acids Res, 34:W182–W185, 2006.</p>
<p>[7] P. Helman, B. M. E. Moret, and H. D. Shapiro. An exact characterization of greedy structures. SIAM J. Disc. Math., 6(2):274–283, 1993.</p>
<p>[8] N. Hulo, A. Bairoch, V. Bulliard, L. Cerutti, B. A. Cuche, E. de Castro, C. Lachaize, P. S. LangendijkGenevaux, and C. J. A. Sigrist. The 20 years of prosite. Nucleic Acids Res, 36:D245–9, 2008.</p>
<p>[9] E. L. Lawler. Combinatorial Optimization: Networks and Matroids. Holt, Rinehart and Winston, 1976.</p>
<p>[10] A. Messerschmidt, R. Huber, K. Wieghardt, and T. Poulos, editors. Handbook of Metalloproteins. John Wiley & Sons, 2004.</p>
<p>[11] A. Passerini, C. Andreini, S. Menchetti, A. Rosato, and P. Frasconi. Predicting zinc binding at the proteome level. BMC Bioinformatics, 8:39, 2007.</p>
<p>[12] A. Passerini, M. Punta, A. Ceroni, B. Rost, and P. Frasconi. Identifying cysteines and histidines in transition-metal-binding sites using support vector machines and neural networks. Proteins, 65(2):305– 316, 2006.</p>
<p>[13] W. Shi, C. Zhan, A. Ignatov, B. A. Manjasetty, N. Marinkovic, M. Sullivan, R. Huang, and M. R. Chance. Metalloproteomics: high-throughput structural and functional annotation of proteins in structural genomics. Structure, 13(10):1473–1486, 2005.</p>
<p>[14] N. Shu, T. Zhou, and S. Hovmoller. Prediction of zinc-binding sites in proteins from sequence. Bioinformatics, 24(6):775–782, 2008.</p>
<p>[15] B. Taskar, V. Chatalbashev, D. Koller, and C. Guestrin. Learning structured prediction models: a large margin approach. Proc. of the 22nd Int. Conf. on Machine Learning (ICML’05), pages 896–903, 2005.</p>
<p>[16] I. Tsochantaridis, T. Joachims, T. Hofmann, and Y. Altun. Large Margin Methods for Structured and Interdependent Output Variables. The Journal of Machine Learning Research, 6:1453–1484, 2005.</p>
<p>[17] J. Weston, O. Chapelle, A. Elisseeff, B. Scholkopf, and V. Vapnik. Kernel dependency estimation. Advances in Neural Information Processing Systems, 15:873–880, 2003.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
