<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>6 jmlr-2009-An Algorithm for Reading Dependencies from the Minimal Undirected Independence Map of a Graphoid that Satisfies Weak Transitivity</title>
</head>

<body>
<p><a title="jmlr" href="../jmlr_home.html">jmlr</a> <a title="jmlr-2009" href="../home/jmlr2009_home.html">jmlr2009</a> <a title="jmlr-2009-6" href="../jmlr2009/jmlr-2009-An_Algorithm_for_Reading_Dependencies_from_the_Minimal_Undirected_Independence_Map_of_a_Graphoid_that_Satisfies_Weak_Transitivity.html">jmlr2009-6</a> <a title="jmlr-2009-6-reference" href="#">jmlr2009-6-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>6 jmlr-2009-An Algorithm for Reading Dependencies from the Minimal Undirected Independence Map of a Graphoid that Satisfies Weak Transitivity</h1>
<br/><p>Source: <a title="jmlr-2009-6-pdf" href="http://jmlr.org/papers/volume10/pena09a/pena09a.pdf">pdf</a></p><p>Author: Jose M. Peña, Roland Nilsson, Johan Björkegren, Jesper Tegnér</p><p>Abstract: We present a sound and complete graphical criterion for reading dependencies from the minimal undirected independence map G of a graphoid M that satisﬁes weak transitivity. Here, complete means that it is able to read all the dependencies in M that can be derived by applying the graphoid properties and weak transitivity to the dependencies used in the construction of G and the independencies obtained from G by vertex separation. We argue that assuming weak transitivity is not too restrictive. As an intermediate step in the derivation of the graphical criterion, we prove that for any undirected graph G there exists a strictly positive discrete probability distribution with the prescribed sample spaces that is faithful to G. We also report an algorithm that implements the graphical criterion and whose running time is considered to be at most O(n2 (e + n)) for n nodes and e edges. Finally, we illustrate how the graphical criterion can be used within bioinformatics to identify biologically meaningful gene dependencies. Keywords: graphical models, vertex separation, graphoids, weak transitivity, bioinformatics</p><br/>
<h2>reference text</h2><p>Ann Becker, Dan Geiger and Christopher Meek. Perfect tree-like Markovian distributions. In Proceedings on the Sixteenth Conference on Uncertainty in Artiﬁcial Intelligence, pages 19-23, 2000. Allister Bernard and Alexander J. Hartemink. Informative structure priors: Joint learning of dynamic regulatory networks from multiple types of data. In Paciﬁc Symposium on Biocomputing 2005, pages 459-470, 2005. 1091  ˜ ¨ ´ P E NA , N ILSSON , B J ORKEGREN AND T EGN E R  Remco R. Bouckaert. Bayesian belief networks: From construction to inference. PhD Thesis, University of Utrecht, 1995. Robert Castelo and Alberto Roverato. A robust procedure for Gaussian graphical model search from microarray data with p larger than n. Journal of Machine Learning Research, 7:2621-2650, 2006. David M. Chickering and Christopher Meek. Finding optimal Bayesian networks. In Proceedings of the Eighteenth Conference on Uncertainty in Artiﬁcial Intelligence, pages 94-102, 2002. Adrian Dobra, Chris Hans, Beatrix Jones, Joseph R. Nevins, Guang Yao and Mike West. Sparse graphical models for exploring gene expression data. Journal of Multivariate Analysis, 90:196212, 2004. Nir Friedman, Kevin Murphy and Stuart Russell. Learning the structure of dynamic probabilistic networks. In Proceedings of the Fourteenth Conference on Uncertainty in Artiﬁcial Intelligence, pages 139-147, 1998. Dan Geiger and Judea Pearl. Logical and algorithmic properties of conditional independence and graphical models. The Annals of Statistics, 21:2001-2021, 1993. Paul R. Halmos. Measure Theory. Van Nostrand, 1966. Dirk Husmeier. Sensitivity and speciﬁcity of inferring genetic regulatory interactions from microarray experiments with dynamic Bayesian networks. Bioinformatics, 19:2271-2282, 2003. Sun Yong Kim, Seiya Imoto and Satoru Miyano. Dynamic Bayesian network and nonparametric regression for nonlinear modeling of gene networks from time series gene expression data. In Proceedings of the First International Workshop on Computational Methods in Systems Biology, pages 104-113, 2003. Hirohisa Kishino and Peter J. Waddell. Correspondence analysis of genes and tissue types and ﬁnding genetic links from microarray data. Genome Informatics, 11:83-95, 2000. Steffen L. Lauritzen. Graphical Models. Oxford University Press, 1996. Hongzhe Li and Jiang Gui. Gradient directed regularization for sparse Gaussian concentration graphs, with applications to inference of genetic networks. Biostatistics, 7:302-317, 2006. Radim Lnˇ niˇ ka and Frantiˇek Mat´ s. On Gaussian conditional independence structures. Kybere c s uˇ netika, 43:327-342, 2007. Christopher Meek. Strong completeness and faithfulness in Bayesian networks. In Proceedings of the Eleventh Conference on Uncertainty in Artiﬁcial Intelligence, pages 411-418, 1995. Kevin Murphy and Saira Mian. Modelling gene expression data using dynamic Bayesian networks. Technical Report, University of California, 1999. Richard E. Neapolitan. Learning Bayesian Networks. Prentice Hall, 2003. Masashi Okamoto. Distinctness of the eigenvalues of a quadratic form in a multivariate sample. The Annals of Statistics, 1:763-765, 1973. 1092  A N A LGORITHM FOR R EADING D EPENDENCIES FROM THE MUI M AP OF A WT G RAPHOID  Irene M. Ong, Jeremy D. Glasner and David Page. Modelling regulatory pathways in E. Coli from time series expression proﬁles. Bioinformatics, 18:S241-S248, 2002. Judea Pearl. Probabilistic Reasoning in Intelligent Systems: Networks of Plausible Inference. Morgan Kaufmann, 1988. Jose M. Pe˜ a. Faithfulness in chain graphs: The discrete case. International Journal of Approximate n Reasoning. Under review, 2009. Jose M. Pe˜ a, Roland Nilsson, Johan Bj¨ rkegren and Jesper Tegn´ r. Identifying the relevant nodes n o e without learning the model. In Proceedings of the Twentysecond Conference on Uncertainy in Artiﬁcial Intelligence, pages 367-374, 2006. Jose M. Pe˜ a. Reading dependencies from polytree-like Bayesian networks. In Proceedings of the n Twentythird Conference on Uncertainty in Artiﬁcial Intelligence, pages 303-309, 2007. Bruno-Edouard Perrin, Liva Ralaivola, Aur´ lien Mazurie, Samuele Bottani, Jacques Mallet and Floe rence d’Alch´ -Buc. Gene networks inference using dynamic Bayesian networks. Bioinformatics, e 19:ii138-ii148, 2003. Juliane Sch¨ fer and Korbinian Strimmer. Learning large-scale graphical Gaussian models from gea nomic data. In Science of Complex Networks: From Biology to the Internet and WWW, 2005a. Juliane Sch¨ fer and Korbinian Strimmer. An empirical Bayes approach to inferring large-scale gene a association networks. Bioinformatics, 21:754-764, 2005b. Juliane Sch¨ fer and Korbinian Strimmer. A shrinkage approach to large-scale covariance matrix a estimation and implications for functional genomics. Statistical Applications in Genetics and Molecular Biology, 4, 2005c. Paola Sebastiani, Emanuela Gussoni, Isaac S. Kohane and Marco F. Ramoni. Statistical challenges in functional genomics (with discussion). Statistical Science, 18:33-60, 2003. P. Stein. A note on the volume of a simplex. The American Mathematical Monthly, 73:299-301, 1966. Milan Studen´ . Probabilistic Conditional Independence Structures. Springer, 2005. y Milan Studen´ and Remco R. Bouckaert. On chain graph models for description of conditional y independence structures. The Annals of Statistics, 26:1434-1495, 1998. Hiroyuki Toh and Katsuhisa Horimoto. Inference of a genetic network by a combined approach of cluster analysis and graphical Gaussian modeling. Bioinformatics, 18:287-297, 2002. Peter J. Waddell and Hirohisa Kishino. Cluster inference methods and graphical models evaluated on NCI60 microarray gene expression data. Genome Informatics, 11:129-140, 2000. Junbai Wang, Ola Myklebost and Eivind Hovig. MGraph: Graphical models for microarray analysis. Bioinformatics, 19:2210-2211, 2003. Joe Whittaker. Graphical Models in Applied Multivariate Statistics. John Wiley & Sons, 1990. 1093  ˜ ¨ ´ P E NA , N ILSSON , B J ORKEGREN AND T EGN E R  Xintao Wu, Yong Ye and Kalpathi R. Subramanian. Interactive analysis of gene interactions using graphical Gaussian model. In Proceedings of the Third ACM SIGKDD Workshop on Data Mining in Bioinformatics, pages 63-69, 2003. Min Zou and Suzanne D. Conzen. A new dynamic Bayesian network (DBN) approach for identifying gene regulatory networks from time course microarray data. Bioinformatics, 21:71-79, 2005.  1094</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
