<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>357 nips-2013-k-Prototype Learning for 3D Rigid Structures</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2013" href="../home/nips2013_home.html">nips2013</a> <a title="nips-2013-357" href="../nips2013/nips-2013-k-Prototype_Learning_for_3D_Rigid_Structures.html">nips2013-357</a> <a title="nips-2013-357-reference" href="#">nips2013-357-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>357 nips-2013-k-Prototype Learning for 3D Rigid Structures</h1>
<br/><p>Source: <a title="nips-2013-357-pdf" href="http://papers.nips.cc/paper/5095-k-prototype-learning-for-3d-rigid-structures.pdf">pdf</a></p><p>Author: Hu Ding, Ronald Berezney, Jinhui Xu</p><p>Abstract: In this paper, we study the following new variant of prototype learning, called k-prototype learning problem for 3D rigid structures: Given a set of 3D rigid structures, ﬁnd a set of k rigid structures so that each of them is a prototype for a cluster of the given rigid structures and the total cost (or dissimilarity) is minimized. Prototype learning is a core problem in machine learning and has a wide range of applications in many areas. Existing results on this problem have mainly focused on the graph domain. In this paper, we present the ﬁrst algorithm for learning multiple prototypes from 3D rigid structures. Our result is based on a number of new insights to rigid structures alignment, clustering, and prototype reconstruction, and is practically efﬁcient with quality guarantee. We validate our approach using two type of data sets, random data and biological data of chromosome territories. Experiments suggest that our approach can effectively learn prototypes in both types of data. 1</p><br/>
<h2>reference text</h2><p>[1] H. Alt and L. Guibas, Discrete geometric shapes: matching, interpolation, and approximation, in: J.-R. Sack, J. Urrutia (Eds.), Handbook of Computational Geometry, Elsevier, Amsterdam, 1999, pp. 121-153.</p>
<p>[2] K. S. Arun, T. S. Huang and S. D. Blostein: Least-Squares Fitting of Two 3-D Point Sets. IEEE Trans. Pattern Anal. Mach. Intell. (PAMI) 9(5):698-700, 1987.</p>
<p>[3] R. Berezney. Regulating the mammalian genome: the role of nuclear architecture. Advances in Enzyme Regulation, 42:39-52, 2002.</p>
<p>[4] P.J. Besl and N.D. McKay, A method for registration of 3-d shapes, IEEE Trans. Pattern Anal. Mach. Intell. 14 (2) 239-256, 1992.</p>
<p>[5] S. Belongie, J. Malik and J. Puzicha. Shape Matching and Object Recognition Using Shape Contexts. IEEE Trans. Pattern Anal. Mach. Intell. 24(4): 509-522 (2002)</p>
<p>[6] J. Croft, J. Bridger, S. Boyle, P. Perry, P. Teague and W. Bickmore. Differences in the localization and morphology of chromosomes in the human nucleus. J. Cell. Biol., 145(6):1119-1131, 1999.</p>
<p>[7] T. Cremer, M. Cremer, S. Dietzel, S. Mller, I. Solovei, and S. Fakan. Chromosome territoriesa functional nuclear landscape. Curr. Opin. Cell. Biol., 18(3):307-316, 2006.</p>
<p>[8] S. D. Cohen and L. J. Guibas. The Earth Mover’s Distance under Transformation Sets. In ICCV, 1076-1083, 1999.</p>
<p>[9] M. Charikar, S. Guha, E. Tardos and D. B. Shmoys. A constant-factor approximation algorithm for the k-median problem (extended abstract). In Proceedings of the thirtieth annual ACM symposium on Theory of computing, STOC ’99, pages 1-10, New York, NY, USA, 1999.</p>
<p>[10] H. Ding, B. Stojkovic, R. Berezney and J. Xu. Gauging Association Patterns of Chromosome Territories via Chromatic Median. In CVPR 2013: 1296-1303</p>
<p>[11] M. F. Demirci, A. Shokoufandeh and S. J. Dickinson. Skeletal Shape Abstraction from Examples. IEEE Trans. Pattern Anal. Mach. Intell. 31(5): 944-952, 2009.</p>
<p>[12] H. Ding and J. Xu. Solving the Chromatic Cone Clustering Problem via Minimum Spanning Sphere. In ICALP (1) 2011: 773-784</p>
<p>[13] H. Ding and J. Xu. FPTAS for Minimizing Earth Mover’s Distance under Rigid Transformations. In ESA 2013: 397-408</p>
<p>[14] M. Ferrer, D. Karatzas, E. Valveny, I. Bardaj and H. Bunke. A generic framework for median graph computation based on a recursive embedding approach. Computer Vision and Image Understanding 115(7): 919-928, 2011.</p>
<p>[15] J. C. Gower. Generalized procrustes analysis. Psychometrika 40: 3331, 1975.</p>
<p>[16] R. I. Hartley and F. Kahl. Global Optimization through Rotation Space Search. International Journal of Computer Vision 82(1): 64-79 (2009)</p>
<p>[17] T. Jiang, F. Jurie and C. Schmid. Learning shape prior models for object matching. In CVPR 2009: 848-855</p>
<p>[18] X. Jiang, A. Munger and H. Bunke. On Median Graphs: Properties, Algorithms, and Applications, IEEE TPAMI, vol. 23, no. 10, pp. 1144-1151, Oct. 2001.</p>
<p>[19] S. Li and O. Svensson: Approximating k-median via pseudo-approximation. In STOC 2013: 901-910.</p>
<p>[20] D. Macrini, K. Siddiqi and S. J. Dickinson. From skeletons to bone graphs: Medial abstraction for object recognition. In CVPR, 2008.</p>
<p>[21] T. B. Sebastian, P. N. Klein and B. B. Kimia. Recognition of Shapes by Editing Shock Graphs. In ICCV, 755-762, 2001.</p>
<p>[22] N. H. Trinh and B. B. Kimia. Learning Prototypical Shapes for Object Categories. In SMiCV, 2010.</p>
<p>[23] E. Weiszfeld. On the point for which the sum of the distances to n given points is minimum. Tohoku. Math. Journal., 43:355-386, 1937.</p>
<p>[24] D. B. West, Introduction to Graph Theory , Prentice Hall, Chapter 3, ISBN 0-13-014400-2, 1999.</p>
<p>[25] M.J. Zeitz, L. Mukherjee, J. Xu, S. Bhattacharya and R. Berezney. A Probabilistic Model for the Arrangement of a Subset of Chromosome Territories in WI38 Human Fibroblasts. Journal of Cellular Physiology, 221, 120-129, 2009.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
