<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>55 nips-2010-Cross Species Expression Analysis using a Dirichlet Process Mixture Model with Latent Matchings</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2010" href="../home/nips2010_home.html">nips2010</a> <a title="nips-2010-55" href="../nips2010/nips-2010-Cross_Species_Expression_Analysis_using_a_Dirichlet_Process_Mixture_Model_with_Latent_Matchings.html">nips2010-55</a> <a title="nips-2010-55-reference" href="#">nips2010-55-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>55 nips-2010-Cross Species Expression Analysis using a Dirichlet Process Mixture Model with Latent Matchings</h1>
<br/><p>Source: <a title="nips-2010-55-pdf" href="http://papers.nips.cc/paper/4153-cross-species-expression-analysis-using-a-dirichlet-process-mixture-model-with-latent-matchings.pdf">pdf</a></p><p>Author: Ziv Bar-joseph, Hai-son P. Le</p><p>Abstract: Recent studies compare gene expression data across species to identify core and species speciﬁc genes in biological systems. To perform such comparisons researchers need to match genes across species. This is a challenging task since the correct matches (orthologs) are not known for most genes. Previous work in this area used deterministic matchings or reduced multidimensional expression data to binary representation. Here we develop a new method that can utilize soft matches (given as priors) to infer both, unique and similar expression patterns across species and a matching for the genes in both species. Our method uses a Dirichlet process mixture model which includes a latent data matching variable. We present learning and inference algorithms based on variational methods for this model. Applying our method to immune response data we show that it can accurately identify common and unique response patterns by improving the matchings between human and mouse genes. 1</p><br/>
<h2>reference text</h2><p>[1] M. Kellis, N. Patterson, M. Endrizzi, B. Birren, and E. S. Lander. Sequencing and comparison of yeast species to identify genes and regulatory elements. Nature, 423:241–254, May 2003.</p>
<p>[2] L. J. Jensen, T. S. Jensen, U. de Lichtenberg, S. Brunak, and P. Bork. Co-evolution of transcriptional and post-translational cell-cycle regulation. Nature, 443:594–597, Oct 2006.</p>
<p>[3] G. Lelandais et al. Genome adaptation to chemical stress: clues from comparative transcriptomics in Saccharomyces cerevisiae and Candida glabrata. Genome Biol., 9:R164, 2008.</p>
<p>[4] G. Quon, Y. W. Teh, E. Chan, M. Brudno, T. Hughes, and Q. D. Morris. A mixture model for the evolution of gene expression in non-homogeneous datasets. In Advances in Neural Information Processing Systems, volume 21, 2009.</p>
<p>[5] J. M. Stuart, E. Segal, D. Koller, and S. K. Kim. A gene-coexpression network for global discovery of conserved genetic modules. Science, 302:249–255, Oct 2003.</p>
<p>[6] Sven Bergmann, Jan Ihmels, and Naama Barkai. Similarities and differences in genome-wide expression data of six organisms. PLoS Biol, 2(1):e9, 12 2003.</p>
<p>[7] Y. Lu, R. Rosenfeld, and Z. Bar-Joseph. Identifying cycling genes by combining sequence homology and expression data. Bioinformatics, 22:e314–322, Jul 2006.</p>
<p>[8] Y. Lu, R. Rosenfeld, G. J. Nau, and Z. Bar-Joseph. Cross species expression analysis of innate immune response. J. Comput. Biol., 17:253–268, Mar 2010.</p>
<p>[9] R. Sharan et al. Conserved patterns of protein interaction in multiple species. Proc. Natl. Acad. Sci. U.S.A., 102:1974–1979, Feb 2005.</p>
<p>[10] Thomas S. Ferguson. A bayesian analysis of some nonparametric problems. The Annals of Statistics, 1(2):209–230, 1973.</p>
<p>[11] J. Sethuraman. A constructive deﬁnition of dirichlet priors. Statistica Sinica, 4:639–650, 1994.</p>
<p>[12] Michael I. Jordan, Zoubin Ghahramani, Tommi S. Jaakkola, and Lawrence K. Saul. An introduction to variational methods for graphical models. Machine Learning, 37(2):183–233, November 1999.</p>
<p>[13] Martin J. Wainwright and Michael I. Jordan. Graphical models, exponential families, and variational inference. Found. Trends Mach. Learn., 1(1-2):1–305, 2008.</p>
<p>[14] H. Ishwaran and James. Gibbs sampling methods for stick breaking priors. Journal of the American Statistical Association, pages 161–173, March 2001.</p>
<p>[15] Zoubin Ghahramani and Matthew J. Beal. Propagation algorithms for variational bayesian learning. In In Advances in Neural Information Processing Systems 13, pages 507–513. MIT Press, 2001.</p>
<p>[16] Sanjoy Dasgupta. Learning mixtures of gaussians. In FOCS ’99: Proceedings of the 40th Annual Symposium on Foundations of Computer Science, Washington, DC, USA, 1999.</p>
<p>[17] M. Meila. Comparing clusterings by the variation of information. In Learning theory and Kernel machines: 16th Annual Conference on Learning Theory and 7th Kernel Workshop, COLT/Kernel 2003, Washington, DC, USA, August 24-27, 2003: proceedings, page 173. Springer Verlag, 2003.</p>
<p>[18] C. S. Detweiler et al. Host microarray analysis reveals a role for the Salmonella response regulator phoP in human macrophage cell death. Proc. Natl. Acad. Sci. U.S.A., 98:5850–5855, May 2001.</p>
<p>[19] K. van Erp et al. Role of strain differences on host resistance and the transcriptional response of macrophages to infection with Yersinia enterocolitica. Physiol. Genomics, 25:75–84, 2006.</p>
<p>[20] D. M. Monack, B. Raupach, et al. Salmonella typhimurium invasion induces apoptosis in infected macrophages. Proc. Natl. Acad. Sci. U.S.A., 93:9833–9838, Sep 1996.</p>
<p>[21] O. O. Zharskaia et al. [Activation of transcription of ribosome genes following human embryo ﬁbroblast infection with cytomegalovirus in vitro]. Tsitologiia, 45:690–701, 2003.</p>
<p>[22] J. W. Gow, S. Hagan, P. Herzyk, C. Cannon, P. O. Behan, and A. Chaudhuri. A gene signature for post-infectious chronic fatigue syndrome. BMC Med Genomics, 2:38, 2009.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
