<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>129 nips-2010-Inter-time segment information sharing for non-homogeneous dynamic Bayesian networks</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2010" href="../home/nips2010_home.html">nips2010</a> <a title="nips-2010-129" href="../nips2010/nips-2010-Inter-time_segment_information_sharing_for_non-homogeneous_dynamic_Bayesian_networks.html">nips2010-129</a> <a title="nips-2010-129-reference" href="#">nips2010-129-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>129 nips-2010-Inter-time segment information sharing for non-homogeneous dynamic Bayesian networks</h1>
<br/><p>Source: <a title="nips-2010-129-pdf" href="http://papers.nips.cc/paper/3944-inter-time-segment-information-sharing-for-non-homogeneous-dynamic-bayesian-networks.pdf">pdf</a></p><p>Author: Dirk Husmeier, Frank Dondelinger, Sophie Lebre</p><p>Abstract: Conventional dynamic Bayesian networks (DBNs) are based on the homogeneous Markov assumption, which is too restrictive in many practical applications. Various approaches to relax the homogeneity assumption have recently been proposed, allowing the network structure to change with time. However, unless time series are very long, this ﬂexibility leads to the risk of overﬁtting and inﬂated inference uncertainty. In the present paper we investigate three regularization schemes based on inter-segment information sharing, choosing different prior distributions and different coupling schemes between nodes. We apply our method to gene expression time series obtained during the Drosophila life cycle, and compare the predicted segmentation with other state-of-the-art techniques. We conclude our evaluation with an application to synthetic biology, where the objective is to predict a known in vivo regulatory network of ﬁve genes in yeast. 1</p><br/>
<h2>reference text</h2><p>[1] J. W. Robinson and A. J. Hartemink. Non-stationary dynamic Bayesian networks. In D. Koller, D. Schuurmans, Y. Bengio, and L. Bottou, editors, Advances in Neural Information Processing Systems (NIPS), volume 21, pages 1369–1376. Morgan Kaufmann Publishers, 2009.</p>
<p>[2] M. Grzegorczyk and D. Husmeier. Non-stationary continuous dynamic Bayesian networks. In Y. Bengio, D. Schuurmans, J. Lafferty, C. K. I. Williams, and A. Culotta, editors, Advances in Neural Information Processing Systems (NIPS), volume 22, pages 682–690. 2009.</p>
<p>[3] A. Ahmed and E. P. Xing. Recovering time-varying networks of dependencies in social and biological studies. Proceedings of the National Academy of Sciences, 106:11878–11883, 2009.</p>
<p>[4] M. Talih and N. Hengartner. Structural learning with time-varying components: Tracking the cross-section of ﬁnancial time series. Journal of the Royal Statistical Society B, 67(3):321–341, 2005.</p>
<p>[5] X. Xuan and K. Murphy. Modeling changing dependency structure in multivariate time series. In Zoubin Ghahramani, editor, Proceedings of the 24th Annual International Conference on Machine Learning (ICML 2007), pages 1055–1062. Omnipress, 2007.</p>
<p>[6] S. L` bre. Stochastic process analysis for Genomics and Dynamic Bayesian Networks inference. e PhD thesis, Universit´ d‘Evry-Val-d‘Essonne, France, 2007. e</p>
<p>[7] S. L` bre, J. Becq, F. Devaux, G. Lelandais, and M.P.H. Stumpf. Statistical inference of the e time-varying structure of gene-regulation networks. BMC Systems Biology, 4(130), 2010.</p>
<p>[8] M. Kolar, L. Song, and E. Xing. Sparsistent learning of varying-coefﬁcient models with structural changes. In Y. Bengio, D. Schuurmans, J. Lafferty, C. K. I. Williams, and A. Culotta, editors, Advances in Neural Information Processing Systems (NIPS), volume 22, pages 1006– 1014. 2009.</p>
<p>[9] B. Larget and D. L. Simon. Markov chain Monte Carlo algorithms for the Bayesian analysis of phylogenetic trees. Molecular Biology and Evolution, 16(6):750–759, 1999.</p>
<p>[10] C. Andrieu and A. Doucet. Joint Bayesian model selection and estimation of noisy sinusoids via reversible jump MCMC. IEEE Transactions on Signal Processing, 47(10):2667–2676, 1999.</p>
<p>[11] P. Green. Reversible jump Markov chain Monte Carlo computation and Bayesian model determination. Biometrika, 82:711–732, 1995.</p>
<p>[12] A. Zellner. On assessing prior distributions and Bayesian regression analysis with g-prior distributions. In P. Goel and A. Zellner, editors, Bayesian Inference and Decision Techniques, pages 233–243. Elsevier, 1986.</p>
<p>[13] A. V. Werhli and D. Husmeier. Gene regulatory network reconstruction by Bayesian integration of prior knowledge and/or different experimental conditions. Journal of Bioinformatics and Computational Biology, 6(3):543–572, 2008.</p>
<p>[14] A. Gelman and D.B. Rubin. Inference from iterative simulation using multiple sequences. Statistical science, 7(4):457–472, 1992.</p>
<p>[15] J. Davis and M. Goadrich. The relationship between precision-recall and ROC curves. In Proceedings of the 23rd international conference on Machine Learning, page 240. ACM, 2006.</p>
<p>[16] M.N. Arbeitman, E.E.M. Furlong, F. Imam, E. Johnson, B.H. Null, B.S. Baker, M.A. Krasnow, M.P. Scott, R.W. Davis, and K.P. White. Gene expression during the life cycle of Drosophila melanogaster. Science, 297(5590):2270–2275, 2002.</p>
<p>[17] I. Cantone, L. Marucci, F. Iorio, M. A Ricci, V. Belcastro, M. Bansal, S. Santini, M. di Bernardo, D. di Bernardo, and M. P Cosma. A yeast synthetic network for in vivo assessment of reverse-engineering and modeling approaches. Cell, 137(1):172181, 2009.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
