<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>11 nips-2008-A spatially varying two-sample recombinant coalescent, with applications to HIV escape response</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2008" href="../home/nips2008_home.html">nips2008</a> <a title="nips-2008-11" href="#">nips2008-11</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>11 nips-2008-A spatially varying two-sample recombinant coalescent, with applications to HIV escape response</h1>
<br/><p>Source: <a title="nips-2008-11-pdf" href="http://papers.nips.cc/paper/3603-a-spatially-varying-two-sample-recombinant-coalescent-with-applications-to-hiv-escape-response.pdf">pdf</a></p><p>Author: Alexander Braunstein, Zhi Wei, Shane T. Jensen, Jon D. Mcauliffe</p><p>Abstract: Statistical evolutionary models provide an important mechanism for describing and understanding the escape response of a viral population under a particular therapy. We present a new hierarchical model that incorporates spatially varying mutation and recombination rates at the nucleotide level. It also maintains separate parameters for treatment and control groups, which allows us to estimate treatment effects explicitly. We use the model to investigate the sequence evolution of HIV populations exposed to a recently developed antisense gene therapy, as well as a more conventional drug therapy. The detection of biologically relevant and plausible signals in both therapy studies demonstrates the effectiveness of the method. 1</p><p>Reference: <a title="nips-2008-11-reference" href="../nips2008_reference/nips-2008-A_spatially_varying_two-sample_recombinant_coalescent%2C_with_applications_to_HIV_escape_response_reference.html">text</a></p><br/><h2>Summary: the most important sentenses genereted by tfidf model</h2><p>sentIndex sentText sentNum sentScore</p><p>1 A spatially varying two-sample recombinant coalescent, with applications to HIV escape response  Alexander Braunstein Statistics Department University of Pennsylvania Wharton School Philadelphia, PA 19104 braunsf@wharton. [sent-1, score-0.297]
</p><p>2 edu  Abstract Statistical evolutionary models provide an important mechanism for describing and understanding the escape response of a viral population under a particular therapy. [sent-10, score-0.502]
</p><p>3 We present a new hierarchical model that incorporates spatially varying mutation and recombination rates at the nucleotide level. [sent-11, score-1.165]
</p><p>4 It also maintains separate parameters for treatment and control groups, which allows us to estimate treatment effects explicitly. [sent-12, score-0.46]
</p><p>5 We use the model to investigate the sequence evolution of HIV populations exposed to a recently developed antisense gene therapy, as well as a more conventional drug therapy. [sent-13, score-0.637]
</p><p>6 The detection of biologically relevant and plausible signals in both therapy studies demonstrates the effectiveness of the method. [sent-14, score-0.215]
</p><p>7 The rapid evolution of HIV and other viruses gives rise to a so-called escape response when infected cells are subjected to therapy. [sent-17, score-0.292]
</p><p>8 Widespread availability of genome sequencing technology has had a profound effect on the study of viral escape response. [sent-18, score-0.383]
</p><p>9 Increasingly, virologists are gathering twosample data sets of viral genome sequences: a control sample contains genomes from a set of virions gathered before therapy, and a treatment sample consists of genomes from the post-therapeutic viral population. [sent-19, score-0.81]
</p><p>10 HIV treatment samples gathered just days after the start of therapy can exhibit a signiﬁcant escape response. [sent-20, score-0.561]
</p><p>11 Up to now, statistical analyses of two-sample viral sequence data sets have been mainly rudimentary. [sent-21, score-0.203]
</p><p>12 As a representative example, [7] presents tabulated counts of mutation occurrences (relative to a reference wild-type sequence) in the control group and the treatment group, without attempting any statistical inference. [sent-22, score-0.572]
</p><p>13 1  In this paper we develop a model which allows for a detailed quantiﬁcation of the escape response present in a two-sample data set. [sent-23, score-0.173]
</p><p>14 The model incorporates mutation and recombination rate parameters which vary positionally along the viral genome, and which differ between the treatment and control samples. [sent-24, score-1.293]
</p><p>15 The resulting posterior distribution suggests speciﬁc regions of the genome where the treatment sample’s evolutionary dynamics differ from the control’s: this is the putative escape response. [sent-26, score-0.567]
</p><p>16 The ﬁrst study consists of a control sample of viral sequences obtained from HIV-infected individuals before a drug treatment, and a corresponding post-treatment sample [9]. [sent-30, score-0.545]
</p><p>17 The second study set is an in vitro investigation of a new gene therapy for HIV; it contains a control sample of untreated virions and a treatment sample of virions challenged with the therapy [7]. [sent-31, score-1.021]
</p><p>18 2  Methods  We begin by brieﬂy describing the standard statistical genetics framework for populations evolving under mutation and recombination. [sent-32, score-0.442]
</p><p>19 In what follows, each “individual” in a population is a sequence of L nucleotides (plus a gap symbol, used when sequences have insertions or deletions relative to each other). [sent-36, score-0.296]
</p><p>20 An alignment is a matrix in which rows are sequences, columns are sites, and the (i, j)th entry is individual i’s nucleotide at site j. [sent-38, score-0.231]
</p><p>21 1  The coalescent with recombination  The genome sequences in the control sample were drawn at random from a large population of sequences at a ﬁxed point in time. [sent-40, score-1.156]
</p><p>22 We approximate the evolution of this population using the WrightFisher evolutionary model with recombination [3]. [sent-41, score-0.825]
</p><p>23 Similarly, the treatment sample sequences are viewed as randomly drawn from a Wright-Fisher recombining population, but governed by different evolutionary parameters. [sent-42, score-0.502]
</p><p>24 Each sequence in the gth generation is determined by randomly choosing a sequence from the (g − 1)th generation, mutating it at one position with probability u, and leaving it unchanged with probability 1 − u. [sent-44, score-0.166]
</p><p>25 Kingman showed that the continuous-time limit of the Wright-Fisher model induces a simple distribution, called the coalescent process, on the topology and branch lengths of the resulting tree. [sent-49, score-0.153]
</p><p>26 Mutation events in the coalescent can be viewed as a separate point process marking locations on the branches of a given coalescent tree. [sent-50, score-0.331]
</p><p>27 This point process is independent of the tree-generating coalescent process. [sent-51, score-0.153]
</p><p>28 The Wright-Fisher dynamics are extended to model recombination as follows. [sent-53, score-0.567]
</p><p>29 With probability r, their child sequence in generation g is a recombinant: a juncture between two adjacent sites is chosen uniformly at random, and the child is formed by joining the paternal sequence to the left of the juncture with the maternal sequence to the right. [sent-55, score-0.351]
</p><p>30 2  Now look backwards in time at the ancestors of a sample: we ﬁnd both coalescence events, where two sequences merge into a common ancestor, and recombination events, where a single sequence splits into the two parent sequences that formed it. [sent-57, score-0.892]
</p><p>31 Thus the genealogy is not a tree but a graph, the ancestral recombination graph (ARG). [sent-58, score-0.61]
</p><p>32 The continuous-time limit of the Wright-Fisher model with recombination induces a distribution over ARGs called the recombinant coalescent [4, 2]. [sent-59, score-0.815]
</p><p>33 In fact, the ARG is the union of L coalescent trees. [sent-60, score-0.153]
</p><p>34 A single site is never split by recombination, so we can follow that site in the sample backwards in time through coalescence events to its MRCA. [sent-61, score-0.206]
</p><p>35 But recombination causes the sample to have a possibly different ancestral tree (and different MRCA) at each site. [sent-62, score-0.646]
</p><p>36 The higher the rate of recombination (corresponding to the parameter r), the more often the tree changes along the alignment. [sent-63, score-0.595]
</p><p>37 For this reason, methods that estimate a ﬁxed, global phylogeny are badly biased in samples from highly recombinant populations, like viruses [10]. [sent-64, score-0.127]
</p><p>38 But experience has shown that the coalescent and the recombinant coalescent can give reasonable results when applied to samples from populations not matching the Wright-Fisher model, such as populations of increasing size [3]. [sent-66, score-0.537]
</p><p>39 2  A two-sample hierarchical recombinant coalescent  We now present the components of our new hierarchical model for a control sample and a treatment sample of nucleotide sequences drawn from two recombining populations. [sent-68, score-0.986]
</p><p>40 There are four parameter vectors of primary interest in the model: a control-population mutation rate µC which varies along the sequence, a corresponding spatially varying treatment-population vector µT , and analogous recombination rate parameter vectors ρC and ρT . [sent-70, score-0.993]
</p><p>41 (3)  T  This prior is designed to give µ and µ a block structure: the Blocks distribution divides the L sequence positions into Bµ adjacent subsequences, with the index of each subsequence’s rightmost B B 1 1 site given by Sµ = (Sµ , . [sent-79, score-0.2]
</p><p>42 The sites in the ith block all mutate at the same rate µC (in the control population) or µT (in the treatment population). [sent-87, score-0.353]
</p><p>43 We lose no generality in sharing i i the same block structure between the populations: two separate block structures can be replaced with a single block structure formed from the union of their Sµ ’s. [sent-88, score-0.144]
</p><p>44 To generate the per-population mutation rates within a block, we ﬁrst draw a lognormally distributed variable µi , which then furnishes the mean for the independent lognormal variables µC and µT . [sent-89, score-0.347]
</p><p>45 The recombination rate parameters (ρC , ρT ) are independent of (µC , µT ) and have the same form of prior distribution (1)–(3), mutatis mutandis. [sent-94, score-0.595]
</p><p>46 Let H C be the alignment of control-sample sequences and H T the treatment-sample sequence alignment. [sent-98, score-0.147]
</p><p>47 However, using the nucleotide mutation model described below, even Monte Carlo approximation of this probability is computationally intractable [12]. [sent-102, score-0.508]
</p><p>48 The kth HMM evaluates 3  the probability that sequence k was produced by mutating and recombining sequences 1 through k − 1. [sent-105, score-0.242]
</p><p>49 (4) (5)  In order to apply PAC, we must specify a nucleotide substitution model, that is, the probability that a nucleotide i mutates to a nucleotide j over evolutionary distance t. [sent-107, score-0.696]
</p><p>50 To incorporate the extended Felsenstein model in PAC, it is necessary to integrate evolutionary distance out of the substitution process p(j | i, 2t), using the exponential distribution induced by the coalescent on the evolutionary distance 2t between pairs of sampled individuals. [sent-111, score-0.383]
</p><p>51 (6) k + 2β k + 2(α + β) πi + πj  Here k is the number of sampled individuals, and πi , πj , α, and β are Felsenstein model parameters (the last two depending on the mutation rate at the site in question). [sent-113, score-0.389]
</p><p>52 For example, in one of our empirical analyses, the treatment population receives RNA antisense gene therapy. [sent-119, score-0.602]
</p><p>53 The target of this therapy is the primary HIV genome sequence itself, not its protein products. [sent-120, score-0.351]
</p><p>54 So we would expect the escape response to manifest at the nucleotide level, in the targeted region of the genome. [sent-121, score-0.389]
</p><p>55 Second, we perform simultaneous hierarchical inference about the control and treatment sample, which encourages the parameter estimates to differ between the samples only where strongly justiﬁed by the data. [sent-123, score-0.286]
</p><p>56 Because the Blocks prior generates mutation and recombination parameters with piecewise-constant proﬁles along the sequence, we call our sampler implementation PICOMAP. [sent-129, score-0.908]
</p><p>57 The block updates consider extending a block to the left or right, merging two adjacent blocks, and splitting a block. [sent-131, score-0.139]
</p><p>58 To illustrate one of the parameter updates within a block, let (µC , µT ) be the current values of the i i control and treatment mutation rates in block i. [sent-133, score-0.67]
</p><p>59 We sample proposal values log µC ∼ N (log µC , τ 2 ) , ˜i i log µT ˜i  ∼  N (log µT , τ 2 ) i 4  ,  (7) (8)  Figure 1: Posterior estimate of the effect of enfuvirtide drug therapy on mutation rates. [sent-134, score-0.88]
</p><p>60 In the second study, the HIV virus was exposed in vitro to a novel antisense gene therapy. [sent-143, score-0.439]
</p><p>61 In both cases, our analysis extracts biologically relevant features of the evolutionary response of HIV to these therapeutic challenges. [sent-144, score-0.182]
</p><p>62 In the interest of brevity, we include only plots of the posterior treatment-effect estimates for both mutation and recombination rates. [sent-147, score-0.944]
</p><p>63 1  Drug therapy study  In this study, ﬁve patients had blood samples taken both before and after treatment with the drug enfuvirtide, also known as Fuzeon or T-20 [11]. [sent-149, score-0.627]
</p><p>64 Pooling across these patients, we have 28 pre-exposure Env sequences which we label as the control sample, and 29 post-exposure Env sequences which we label as the treatment sample. [sent-151, score-0.443]
</p><p>65 We quantify the treatment effect of exposure 5  Figure 2: Posterior estimate of the effect of enfuvirtide drug therapy on recombination rates. [sent-152, score-1.271]
</p><p>66 to the drug by calculating the posterior mean and 95% highest-posterior-density (HPD) intervals of the difference in recombination rates ρT − ρC and mutation rates µT − µC at each position of the genomic sequence. [sent-154, score-1.13]
</p><p>67 The very existence in the patient of a post-exposure HIV population indicates the evolution of sequence changes that have conferred resistance to the action of the drug enfuvirtide. [sent-155, score-0.371]
</p><p>68 In fact, resistance-conferring mutations are known a priori to occur at nucleotide locations 1639-1668 in the Env sequence. [sent-156, score-0.277]
</p><p>69 Figure 1 shows the posterior estimate of the treatment effect on mutation rates over the length of the Env sequence. [sent-157, score-0.644]
</p><p>70 From nucleotide positions 1590-1700, the entire 95% HPD interval of the mutation rate treatment effect is above zero, which suggests our model is able to detect elevated levels of mutation in the resistance-conferring region, among individuals in the treatment sample. [sent-158, score-1.415]
</p><p>71 Another preliminary observation from this study was that both the pre-exposure and post-exposure sequences are mixtures of several different HIV subtypes. [sent-159, score-0.119]
</p><p>72 Subtype identity is speciﬁed by the V3 loop subsequence of the Env sequence, which corresponds to nucleotide positions 887-995. [sent-160, score-0.226]
</p><p>73 Since it is unlikely that resistance-conferring mutations developed independently in each subtype, we suspect that the resistance-conferring mutations were passed to the different subtypes via recombination. [sent-161, score-0.176]
</p><p>74 Recombination is the primary means by which drug resistance is transferred in vivo between strains of HIV, so recombination at these locations involving drug resistant strains would allow successful transfer of the resistance-conferring mutations between types of HIV. [sent-162, score-1.025]
</p><p>75 Figure 2 shows the spatial posterior estimate of the treatment effect on recombination. [sent-163, score-0.297]
</p><p>76 We see two areas of increased recombination, one from nucleotide positions 1020-1170 and another from nucleotide positions 1900-2200. [sent-164, score-0.452]
</p><p>77 As an interesting side note, we see a marked decrease in mutation and recombination in the V3 loop that determines sequence speciﬁcity. [sent-165, score-0.938]
</p><p>78 2  Antisense gene therapy study  In the VIRxSYS antisense gene therapy study, we have two populations of wild type HIV in vitro. [sent-167, score-0.935]
</p><p>79 The samples consist of 19 Env sequences from a control HIV population that was allowed to evolve neutrally in cell culture, along with 48 Env sequences sampled from an HIV population evolving in cell cultures that were transfected with the VIRxSYS antisense vector [7]. [sent-168, score-0.645]
</p><p>80 The antisense gene therapy vector targets nucleotide positions 1325 - 2249. [sent-169, score-0.734]
</p><p>81 Unlike drug therapy treatments, whose effect can be nulliﬁed by just one or two well placed mutations, a relatively large number of mutations are required to escape the effects of antisense gene therapy. [sent-170, score-0.897]
</p><p>82 We again quantify the treatment effect of exposure to the antisense vector by calculating the posterior mean and 95% HPD interval of the 6  Figure 3: Posterior estimate of the effect of VIRxSYS antisense gene therapy on mutation rates. [sent-171, score-1.386]
</p><p>83 difference in recombination rates ρT − ρC and mutation rates µT − µC at each position of the Env sequence. [sent-173, score-0.942]
</p><p>84 Figures 3 and 4 show the posterior estimate of the treatment’s effect on mutation and recombination, respectively. [sent-174, score-0.409]
</p><p>85 The most striking feature of the plots is the area of signiﬁcantly elevated mutation in the treatment sequences. [sent-175, score-0.551]
</p><p>86 The leftmost region of the highest plateau corresponds to nucleotide position 1325, the 5’ boundary of the antisense target region. [sent-176, score-0.421]
</p><p>87 This area of heightened mutation overlaps with the target region for around 425 nucleotides in the 3’ direction. [sent-177, score-0.393]
</p><p>88 We see fewer differences in the recombination rate, suggesting that mutation is the primary mechanism of evolutionary response to the antisense vector. [sent-178, score-1.248]
</p><p>89 In fact, we estimate lower recombination rates in the target region of the treatment sequences relative to the control sequences. [sent-179, score-0.97]
</p><p>90 4  Discussion  We have introduced a hierarchical model for the estimation of evolutionary escape response in a population exposed to therapeutic challenge. [sent-180, score-0.495]
</p><p>91 The escape response is quantiﬁed by mutation and recombination rate parameters. [sent-181, score-1.087]
</p><p>92 Our method allows for spatial heterogeneity in these mutation and recombination rates. [sent-182, score-0.886]
</p><p>93 It estimates differences between treatment and control sample parameters, with parameter values encouraged to be similar between the two populations except where the data suggests otherwise. [sent-183, score-0.357]
</p><p>94 We applied our procedure to sequence data from two different HIV therapy studies, detecting evolutionary responses in both studies that are of biological interest and may be relevant to the design of future HIV treatments. [sent-184, score-0.368]
</p><p>95 Although virological problems motivated the creation of our model, it applies more generally to twosample data sets of nucleic acid sequences drawn from any population. [sent-185, score-0.127]
</p><p>96 The model is particularly relevant for populations in which the recombination rate is a substantial fraction of the mutation rate, since simpler models which ignore recombination can produce seriously misleading results. [sent-186, score-1.549]
</p><p>97 7  Figure 4: Posterior estimate of the effect of VIRxSYS antisense gene therapy on recombination rates. [sent-189, score-1.107]
</p><p>98 Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data. [sent-223, score-0.567]
</p><p>99 Models of sequence evolution for DNA sequences containing gaps. [sent-239, score-0.202]
</p><p>100 Clinical resistance to enfuvirtide does not affect susceptibility of human immunodeﬁciency virus type 1 to other classes of entry inhibitors. [sent-248, score-0.174]
</p>
<br/>
<h2>similar papers computed by tfidf model</h2><h3>tfidf for this paper:</h3><p>wordName wordTfidf (topN-words)</p>
<p>[('recombination', 0.567), ('mutation', 0.319), ('hiv', 0.29), ('therapy', 0.215), ('treatment', 0.207), ('antisense', 0.205), ('nucleotide', 0.189), ('coalescent', 0.153), ('env', 0.142), ('escape', 0.139), ('drug', 0.13), ('viral', 0.126), ('hpd', 0.11), ('population', 0.102), ('evolutionary', 0.101), ('sequences', 0.095), ('recombinant', 0.095), ('mutations', 0.088), ('gene', 0.088), ('virus', 0.079), ('pac', 0.069), ('populations', 0.068), ('enfuvirtide', 0.063), ('omegamap', 0.063), ('recombining', 0.063), ('virions', 0.063), ('virxsys', 0.063), ('genome', 0.062), ('posterior', 0.058), ('evolution', 0.055), ('felsenstein', 0.055), ('genetics', 0.055), ('individuals', 0.052), ('sequence', 0.052), ('block', 0.048), ('immunode', 0.047), ('therapeutic', 0.047), ('wharton', 0.047), ('nucleotides', 0.047), ('control', 0.046), ('ancestral', 0.043), ('site', 0.042), ('blocks', 0.042), ('exposed', 0.039), ('genomes', 0.038), ('pennsylvania', 0.037), ('positions', 0.037), ('sample', 0.036), ('response', 0.034), ('backwards', 0.033), ('hierarchical', 0.033), ('resistance', 0.032), ('effect', 0.032), ('args', 0.032), ('infected', 0.032), ('maternal', 0.032), ('mutating', 0.032), ('paternal', 0.032), ('picomap', 0.032), ('schierup', 0.032), ('twosample', 0.032), ('virology', 0.032), ('viruses', 0.032), ('wild', 0.032), ('log', 0.03), ('generation', 0.03), ('spatially', 0.029), ('substitution', 0.028), ('rates', 0.028), ('rate', 0.028), ('binder', 0.028), ('coalescence', 0.028), ('juncture', 0.028), ('mrca', 0.028), ('replication', 0.028), ('strains', 0.028), ('subtype', 0.028), ('vitro', 0.028), ('region', 0.027), ('patients', 0.027), ('events', 0.025), ('philadelphia', 0.025), ('aids', 0.025), ('blocking', 0.025), ('elevated', 0.025), ('exposure', 0.025), ('harrison', 0.025), ('proposal', 0.025), ('analyses', 0.025), ('sites', 0.024), ('blood', 0.024), ('kingman', 0.024), ('ray', 0.024), ('study', 0.024), ('primary', 0.022), ('ancestors', 0.022), ('updates', 0.022), ('sampler', 0.022), ('adjacent', 0.021), ('pa', 0.02)]</p>
<h3>similar papers list:</h3><p>simIndex simValue paperId paperTitle</p>
<p>same-paper 1 1.0000002 <a title="11-tfidf-1" href="./nips-2008-A_spatially_varying_two-sample_recombinant_coalescent%2C_with_applications_to_HIV_escape_response.html">11 nips-2008-A spatially varying two-sample recombinant coalescent, with applications to HIV escape response</a></p>
<p>Author: Alexander Braunstein, Zhi Wei, Shane T. Jensen, Jon D. Mcauliffe</p><p>Abstract: Statistical evolutionary models provide an important mechanism for describing and understanding the escape response of a viral population under a particular therapy. We present a new hierarchical model that incorporates spatially varying mutation and recombination rates at the nucleotide level. It also maintains separate parameters for treatment and control groups, which allows us to estimate treatment effects explicitly. We use the model to investigate the sequence evolution of HIV populations exposed to a recently developed antisense gene therapy, as well as a more conventional drug therapy. The detection of biologically relevant and plausible signals in both therapy studies demonstrates the effectiveness of the method. 1</p><p>2 0.10376346 <a title="11-tfidf-2" href="./nips-2008-An_Efficient_Sequential_Monte_Carlo_Algorithm_for_Coalescent_Clustering.html">18 nips-2008-An Efficient Sequential Monte Carlo Algorithm for Coalescent Clustering</a></p>
<p>Author: Dilan Gorur, Yee W. Teh</p><p>Abstract: We propose an efﬁcient sequential Monte Carlo inference scheme for the recently proposed coalescent clustering model [1]. Our algorithm has a quadratic runtime while those in [1] is cubic. In experiments, we were surprised to ﬁnd that in addition to being more efﬁcient, it is also a better sequential Monte Carlo sampler than the best in [1], when measured in terms of variance of estimated likelihood and effective sample size. 1</p><p>3 0.096818276 <a title="11-tfidf-3" href="./nips-2008-The_Infinite_Hierarchical_Factor_Regression_Model.html">235 nips-2008-The Infinite Hierarchical Factor Regression Model</a></p>
<p>Author: Piyush Rai, Hal Daume</p><p>Abstract: We propose a nonparametric Bayesian factor regression model that accounts for uncertainty in the number of factors, and the relationship between factors. To accomplish this, we propose a sparse variant of the Indian Buffet Process and couple this with a hierarchical model over factors, based on Kingman’s coalescent. We apply this model to two problems (factor analysis and factor regression) in gene-expression data analysis. 1</p><p>4 0.078324459 <a title="11-tfidf-4" href="./nips-2008-Efficient_Inference_in_Phylogenetic_InDel_Trees.html">70 nips-2008-Efficient Inference in Phylogenetic InDel Trees</a></p>
<p>Author: Alexandre Bouchard-côté, Dan Klein, Michael I. Jordan</p><p>Abstract: Accurate and efﬁcient inference in evolutionary trees is a central problem in computational biology. While classical treatments have made unrealistic site independence assumptions, ignoring insertions and deletions, realistic approaches require tracking insertions and deletions along the phylogenetic tree—a challenging and unsolved computational problem. We propose a new ancestry resampling procedure for inference in evolutionary trees. We evaluate our method in two problem domains—multiple sequence alignment and reconstruction of ancestral sequences—and show substantial improvement over the current state of the art. 1</p><p>5 0.077040836 <a title="11-tfidf-5" href="./nips-2008-A_mixture_model_for_the_evolution_of_gene_expression_in_non-homogeneous_datasets.html">9 nips-2008-A mixture model for the evolution of gene expression in non-homogeneous datasets</a></p>
<p>Author: Gerald Quon, Yee W. Teh, Esther Chan, Timothy Hughes, Michael Brudno, Quaid D. Morris</p><p>Abstract: We address the challenge of assessing conservation of gene expression in complex, non-homogeneous datasets. Recent studies have demonstrated the success of probabilistic models in studying the evolution of gene expression in simple eukaryotic organisms such as yeast, for which measurements are typically scalar and independent. Models capable of studying expression evolution in much more complex organisms such as vertebrates are particularly important given the medical and scientiﬁc interest in species such as human and mouse. We present Brownian Factor Phylogenetic Analysis, a statistical model that makes a number of signiﬁcant extensions to previous models to enable characterization of changes in expression among highly complex organisms. We demonstrate the efﬁcacy of our method on a microarray dataset proﬁling diverse tissues from multiple vertebrate species. We anticipate that the model will be invaluable in the study of gene expression patterns in other diverse organisms as well, such as worms and insects. 1</p><p>6 0.048959784 <a title="11-tfidf-6" href="./nips-2008-Efficient_Sampling_for_Gaussian_Process_Inference_using_Control_Variables.html">71 nips-2008-Efficient Sampling for Gaussian Process Inference using Control Variables</a></p>
<p>7 0.045917835 <a title="11-tfidf-7" href="./nips-2008-Temporal_Dynamics_of_Cognitive_Control.html">231 nips-2008-Temporal Dynamics of Cognitive Control</a></p>
<p>8 0.045712721 <a title="11-tfidf-8" href="./nips-2008-Accelerating_Bayesian_Inference_over_Nonlinear_Differential_Equations_with_Gaussian_Processes.html">12 nips-2008-Accelerating Bayesian Inference over Nonlinear Differential Equations with Gaussian Processes</a></p>
<p>9 0.04067238 <a title="11-tfidf-9" href="./nips-2008-Optimal_Response_Initiation%3A_Why_Recent_Experience_Matters.html">172 nips-2008-Optimal Response Initiation: Why Recent Experience Matters</a></p>
<p>10 0.036089186 <a title="11-tfidf-10" href="./nips-2008-Psychiatry%3A_Insights_into_depression_through_normative_decision-making_models.html">187 nips-2008-Psychiatry: Insights into depression through normative decision-making models</a></p>
<p>11 0.035946231 <a title="11-tfidf-11" href="./nips-2008-Tighter_Bounds_for_Structured_Estimation.html">239 nips-2008-Tighter Bounds for Structured Estimation</a></p>
<p>12 0.035123069 <a title="11-tfidf-12" href="./nips-2008-Phase_transitions_for_high-dimensional_joint_support_recovery.html">179 nips-2008-Phase transitions for high-dimensional joint support recovery</a></p>
<p>13 0.033844396 <a title="11-tfidf-13" href="./nips-2008-Non-stationary_dynamic_Bayesian_networks.html">152 nips-2008-Non-stationary dynamic Bayesian networks</a></p>
<p>14 0.032951806 <a title="11-tfidf-14" href="./nips-2008-An_Empirical_Analysis_of_Domain_Adaptation_Algorithms_for_Genomic_Sequence_Analysis.html">19 nips-2008-An Empirical Analysis of Domain Adaptation Algorithms for Genomic Sequence Analysis</a></p>
<p>15 0.032159466 <a title="11-tfidf-15" href="./nips-2008-Transfer_Learning_by_Distribution_Matching_for_Targeted_Advertising.html">241 nips-2008-Transfer Learning by Distribution Matching for Targeted Advertising</a></p>
<p>16 0.03205077 <a title="11-tfidf-16" href="./nips-2008-Hierarchical_Fisher_Kernels_for_Longitudinal_Data.html">97 nips-2008-Hierarchical Fisher Kernels for Longitudinal Data</a></p>
<p>17 0.030630944 <a title="11-tfidf-17" href="./nips-2008-MDPs_with_Non-Deterministic_Policies.html">131 nips-2008-MDPs with Non-Deterministic Policies</a></p>
<p>18 0.029850163 <a title="11-tfidf-18" href="./nips-2008-Bayesian_Experimental_Design_of_Magnetic_Resonance_Imaging_Sequences.html">30 nips-2008-Bayesian Experimental Design of Magnetic Resonance Imaging Sequences</a></p>
<p>19 0.029193288 <a title="11-tfidf-19" href="./nips-2008-Scalable_Algorithms_for_String_Kernels_with_Inexact_Matching.html">203 nips-2008-Scalable Algorithms for String Kernels with Inexact Matching</a></p>
<p>20 0.027569572 <a title="11-tfidf-20" href="./nips-2008-The_Gaussian_Process_Density_Sampler.html">233 nips-2008-The Gaussian Process Density Sampler</a></p>
<br/>
<h2>similar papers computed by <a title="lsi-model" href="../home/nips2008_lsi.html">lsi model</a></h2><h3>lsi for this paper:</h3><p>topicId topicWeight</p>
<p>[(0, -0.091), (1, 0.024), (2, 0.049), (3, 0.015), (4, 0.036), (5, -0.036), (6, 0.011), (7, 0.067), (8, 0.043), (9, -0.006), (10, -0.019), (11, 0.041), (12, 0.039), (13, -0.061), (14, 0.054), (15, 0.074), (16, 0.011), (17, -0.075), (18, -0.005), (19, 0.008), (20, -0.009), (21, 0.156), (22, -0.113), (23, -0.031), (24, -0.16), (25, 0.028), (26, -0.074), (27, 0.058), (28, 0.044), (29, 0.007), (30, 0.022), (31, 0.049), (32, 0.103), (33, 0.031), (34, 0.012), (35, 0.02), (36, -0.028), (37, -0.005), (38, -0.0), (39, 0.084), (40, -0.007), (41, 0.046), (42, 0.081), (43, 0.044), (44, 0.007), (45, -0.019), (46, 0.009), (47, 0.013), (48, -0.002), (49, -0.038)]</p>
<h3>similar papers list:</h3><p>simIndex simValue paperId paperTitle</p>
<p>same-paper 1 0.91187805 <a title="11-lsi-1" href="./nips-2008-A_spatially_varying_two-sample_recombinant_coalescent%2C_with_applications_to_HIV_escape_response.html">11 nips-2008-A spatially varying two-sample recombinant coalescent, with applications to HIV escape response</a></p>
<p>Author: Alexander Braunstein, Zhi Wei, Shane T. Jensen, Jon D. Mcauliffe</p><p>Abstract: Statistical evolutionary models provide an important mechanism for describing and understanding the escape response of a viral population under a particular therapy. We present a new hierarchical model that incorporates spatially varying mutation and recombination rates at the nucleotide level. It also maintains separate parameters for treatment and control groups, which allows us to estimate treatment effects explicitly. We use the model to investigate the sequence evolution of HIV populations exposed to a recently developed antisense gene therapy, as well as a more conventional drug therapy. The detection of biologically relevant and plausible signals in both therapy studies demonstrates the effectiveness of the method. 1</p><p>2 0.72702986 <a title="11-lsi-2" href="./nips-2008-A_mixture_model_for_the_evolution_of_gene_expression_in_non-homogeneous_datasets.html">9 nips-2008-A mixture model for the evolution of gene expression in non-homogeneous datasets</a></p>
<p>Author: Gerald Quon, Yee W. Teh, Esther Chan, Timothy Hughes, Michael Brudno, Quaid D. Morris</p><p>Abstract: We address the challenge of assessing conservation of gene expression in complex, non-homogeneous datasets. Recent studies have demonstrated the success of probabilistic models in studying the evolution of gene expression in simple eukaryotic organisms such as yeast, for which measurements are typically scalar and independent. Models capable of studying expression evolution in much more complex organisms such as vertebrates are particularly important given the medical and scientiﬁc interest in species such as human and mouse. We present Brownian Factor Phylogenetic Analysis, a statistical model that makes a number of signiﬁcant extensions to previous models to enable characterization of changes in expression among highly complex organisms. We demonstrate the efﬁcacy of our method on a microarray dataset proﬁling diverse tissues from multiple vertebrate species. We anticipate that the model will be invaluable in the study of gene expression patterns in other diverse organisms as well, such as worms and insects. 1</p><p>3 0.71094859 <a title="11-lsi-3" href="./nips-2008-An_Efficient_Sequential_Monte_Carlo_Algorithm_for_Coalescent_Clustering.html">18 nips-2008-An Efficient Sequential Monte Carlo Algorithm for Coalescent Clustering</a></p>
<p>Author: Dilan Gorur, Yee W. Teh</p><p>Abstract: We propose an efﬁcient sequential Monte Carlo inference scheme for the recently proposed coalescent clustering model [1]. Our algorithm has a quadratic runtime while those in [1] is cubic. In experiments, we were surprised to ﬁnd that in addition to being more efﬁcient, it is also a better sequential Monte Carlo sampler than the best in [1], when measured in terms of variance of estimated likelihood and effective sample size. 1</p><p>4 0.64730418 <a title="11-lsi-4" href="./nips-2008-Efficient_Inference_in_Phylogenetic_InDel_Trees.html">70 nips-2008-Efficient Inference in Phylogenetic InDel Trees</a></p>
<p>Author: Alexandre Bouchard-côté, Dan Klein, Michael I. Jordan</p><p>Abstract: Accurate and efﬁcient inference in evolutionary trees is a central problem in computational biology. While classical treatments have made unrealistic site independence assumptions, ignoring insertions and deletions, realistic approaches require tracking insertions and deletions along the phylogenetic tree—a challenging and unsolved computational problem. We propose a new ancestry resampling procedure for inference in evolutionary trees. We evaluate our method in two problem domains—multiple sequence alignment and reconstruction of ancestral sequences—and show substantial improvement over the current state of the art. 1</p><p>5 0.63148439 <a title="11-lsi-5" href="./nips-2008-The_Infinite_Hierarchical_Factor_Regression_Model.html">235 nips-2008-The Infinite Hierarchical Factor Regression Model</a></p>
<p>Author: Piyush Rai, Hal Daume</p><p>Abstract: We propose a nonparametric Bayesian factor regression model that accounts for uncertainty in the number of factors, and the relationship between factors. To accomplish this, we propose a sparse variant of the Indian Buffet Process and couple this with a hierarchical model over factors, based on Kingman’s coalescent. We apply this model to two problems (factor analysis and factor regression) in gene-expression data analysis. 1</p><p>6 0.45450023 <a title="11-lsi-6" href="./nips-2008-Non-stationary_dynamic_Bayesian_networks.html">152 nips-2008-Non-stationary dynamic Bayesian networks</a></p>
<p>7 0.42246708 <a title="11-lsi-7" href="./nips-2008-The_Mondrian_Process.html">236 nips-2008-The Mondrian Process</a></p>
<p>8 0.38072744 <a title="11-lsi-8" href="./nips-2008-Fast_Computation_of_Posterior_Mode_in_Multi-Level_Hierarchical_Models.html">82 nips-2008-Fast Computation of Posterior Mode in Multi-Level Hierarchical Models</a></p>
<p>9 0.38054413 <a title="11-lsi-9" href="./nips-2008-Efficient_Sampling_for_Gaussian_Process_Inference_using_Control_Variables.html">71 nips-2008-Efficient Sampling for Gaussian Process Inference using Control Variables</a></p>
<p>10 0.37826118 <a title="11-lsi-10" href="./nips-2008-Optimal_Response_Initiation%3A_Why_Recent_Experience_Matters.html">172 nips-2008-Optimal Response Initiation: Why Recent Experience Matters</a></p>
<p>11 0.37143809 <a title="11-lsi-11" href="./nips-2008-Stress%2C_noradrenaline%2C_and_realistic_prediction_of_mouse_behaviour_using_reinforcement_learning.html">222 nips-2008-Stress, noradrenaline, and realistic prediction of mouse behaviour using reinforcement learning</a></p>
<p>12 0.35429803 <a title="11-lsi-12" href="./nips-2008-Learning_Bounded_Treewidth_Bayesian_Networks.html">115 nips-2008-Learning Bounded Treewidth Bayesian Networks</a></p>
<p>13 0.34005255 <a title="11-lsi-13" href="./nips-2008-Temporal_Dynamics_of_Cognitive_Control.html">231 nips-2008-Temporal Dynamics of Cognitive Control</a></p>
<p>14 0.33666277 <a title="11-lsi-14" href="./nips-2008-Probabilistic_detection_of_short_events%2C_with_application_to_critical_care_monitoring.html">186 nips-2008-Probabilistic detection of short events, with application to critical care monitoring</a></p>
<p>15 0.33401072 <a title="11-lsi-15" href="./nips-2008-Nonparametric_regression_and_classification_with_joint_sparsity_constraints.html">155 nips-2008-Nonparametric regression and classification with joint sparsity constraints</a></p>
<p>16 0.32182395 <a title="11-lsi-16" href="./nips-2008-A_Scalable_Hierarchical_Distributed_Language_Model.html">4 nips-2008-A Scalable Hierarchical Distributed Language Model</a></p>
<p>17 0.31723559 <a title="11-lsi-17" href="./nips-2008-Predicting_the_Geometry_of_Metal_Binding_Sites_from_Protein_Sequence.html">183 nips-2008-Predicting the Geometry of Metal Binding Sites from Protein Sequence</a></p>
<p>18 0.30979225 <a title="11-lsi-18" href="./nips-2008-Psychiatry%3A_Insights_into_depression_through_normative_decision-making_models.html">187 nips-2008-Psychiatry: Insights into depression through normative decision-making models</a></p>
<p>19 0.30220169 <a title="11-lsi-19" href="./nips-2008-The_Infinite_Factorial_Hidden_Markov_Model.html">234 nips-2008-The Infinite Factorial Hidden Markov Model</a></p>
<p>20 0.29767653 <a title="11-lsi-20" href="./nips-2008-QUIC-SVD%3A_Fast_SVD_Using_Cosine_Trees.html">188 nips-2008-QUIC-SVD: Fast SVD Using Cosine Trees</a></p>
<br/>
<h2>similar papers computed by <a title="lda-model" href="../home/nips2008_lda.html">lda model</a></h2><h3>lda for this paper:</h3><p>topicId topicWeight</p>
<p>[(6, 0.028), (7, 0.059), (12, 0.017), (15, 0.014), (28, 0.135), (57, 0.057), (59, 0.015), (63, 0.013), (71, 0.484), (77, 0.024), (83, 0.04), (87, 0.013)]</p>
<h3>similar papers list:</h3><p>simIndex simValue paperId paperTitle</p>
<p>1 0.87426591 <a title="11-lda-1" href="./nips-2008-Spike_Feature_Extraction_Using_Informative_Samples.html">220 nips-2008-Spike Feature Extraction Using Informative Samples</a></p>
<p>Author: Zhi Yang, Qi Zhao, Wentai Liu</p><p>Abstract: This paper presents a spike feature extraction algorithm that targets real-time spike sorting and facilitates miniaturized microchip implementation. The proposed algorithm has been evaluated on synthesized waveforms and experimentally recorded sequences. When compared with many spike sorting approaches our algorithm demonstrates improved speed, accuracy and allows unsupervised execution. A preliminary hardware implementation has been realized using an integrated microchip interfaced with a personal computer. 1</p><p>2 0.8098219 <a title="11-lda-2" href="./nips-2008-On_the_Complexity_of_Linear_Prediction%3A_Risk_Bounds%2C_Margin_Bounds%2C_and_Regularization.html">161 nips-2008-On the Complexity of Linear Prediction: Risk Bounds, Margin Bounds, and Regularization</a></p>
<p>Author: Sham M. Kakade, Karthik Sridharan, Ambuj Tewari</p><p>Abstract: This work characterizes the generalization ability of algorithms whose predictions are linear in the input vector. To this end, we provide sharp bounds for Rademacher and Gaussian complexities of (constrained) linear classes, which directly lead to a number of generalization bounds. This derivation provides simpliﬁed proofs of a number of corollaries including: risk bounds for linear prediction (including settings where the weight vectors are constrained by either L2 or L1 constraints), margin bounds (including both L2 and L1 margins, along with more general notions based on relative entropy), a proof of the PAC-Bayes theorem, and upper bounds on L2 covering numbers (with Lp norm constraints and relative entropy constraints). In addition to providing a uniﬁed analysis, the results herein provide some of the sharpest risk and margin bounds. Interestingly, our results show that the uniform convergence rates of empirical risk minimization algorithms tightly match the regret bounds of online learning algorithms for linear prediction, up to a constant factor of 2. 1</p><p>same-paper 3 0.79467213 <a title="11-lda-3" href="./nips-2008-A_spatially_varying_two-sample_recombinant_coalescent%2C_with_applications_to_HIV_escape_response.html">11 nips-2008-A spatially varying two-sample recombinant coalescent, with applications to HIV escape response</a></p>
<p>Author: Alexander Braunstein, Zhi Wei, Shane T. Jensen, Jon D. Mcauliffe</p><p>Abstract: Statistical evolutionary models provide an important mechanism for describing and understanding the escape response of a viral population under a particular therapy. We present a new hierarchical model that incorporates spatially varying mutation and recombination rates at the nucleotide level. It also maintains separate parameters for treatment and control groups, which allows us to estimate treatment effects explicitly. We use the model to investigate the sequence evolution of HIV populations exposed to a recently developed antisense gene therapy, as well as a more conventional drug therapy. The detection of biologically relevant and plausible signals in both therapy studies demonstrates the effectiveness of the method. 1</p><p>4 0.76006484 <a title="11-lda-4" href="./nips-2008-MDPs_with_Non-Deterministic_Policies.html">131 nips-2008-MDPs with Non-Deterministic Policies</a></p>
<p>Author: Mahdi M. Fard, Joelle Pineau</p><p>Abstract: Markov Decision Processes (MDPs) have been extensively studied and used in the context of planning and decision-making, and many methods exist to ﬁnd the optimal policy for problems modelled as MDPs. Although ﬁnding the optimal policy is sufﬁcient in many domains, in certain applications such as decision support systems where the policy is executed by a human (rather than a machine), ﬁnding all possible near-optimal policies might be useful as it provides more ﬂexibility to the person executing the policy. In this paper we introduce the new concept of non-deterministic MDP policies, and address the question of ﬁnding near-optimal non-deterministic policies. We propose two solutions to this problem, one based on a Mixed Integer Program and the other one based on a search algorithm. We include experimental results obtained from applying this framework to optimize treatment choices in the context of a medical decision support system. 1</p><p>5 0.48020163 <a title="11-lda-5" href="./nips-2008-Rademacher_Complexity_Bounds_for_Non-I.I.D._Processes.html">189 nips-2008-Rademacher Complexity Bounds for Non-I.I.D. Processes</a></p>
<p>Author: Mehryar Mohri, Afshin Rostamizadeh</p><p>Abstract: This paper presents the ﬁrst Rademacher complexity-based error bounds for noni.i.d. settings, a generalization of similar existing bounds derived for the i.i.d. case. Our bounds hold in the scenario of dependent samples generated by a stationary β-mixing process, which is commonly adopted in many previous studies of noni.i.d. settings. They beneﬁt from the crucial advantages of Rademacher complexity over other measures of the complexity of hypothesis classes. In particular, they are data-dependent and measure the complexity of a class of hypotheses based on the training sample. The empirical Rademacher complexity can be estimated from such ﬁnite samples and lead to tighter generalization bounds. We also present the ﬁrst margin bounds for kernel-based classiﬁcation in this non-i.i.d. setting and brieﬂy study their convergence.</p><p>6 0.46464494 <a title="11-lda-6" href="./nips-2008-Dependent_Dirichlet_Process_Spike_Sorting.html">59 nips-2008-Dependent Dirichlet Process Spike Sorting</a></p>
<p>7 0.45754704 <a title="11-lda-7" href="./nips-2008-Hebbian_Learning_of_Bayes_Optimal_Decisions.html">96 nips-2008-Hebbian Learning of Bayes Optimal Decisions</a></p>
<p>8 0.45628503 <a title="11-lda-8" href="./nips-2008-Fast_Rates_for_Regularized_Objectives.html">85 nips-2008-Fast Rates for Regularized Objectives</a></p>
<p>9 0.44632292 <a title="11-lda-9" href="./nips-2008-A_Transductive_Bound_for_the_Voted_Classifier_with_an_Application_to_Semi-supervised_Learning.html">5 nips-2008-A Transductive Bound for the Voted Classifier with an Application to Semi-supervised Learning</a></p>
<p>10 0.42782423 <a title="11-lda-10" href="./nips-2008-Adaptive_Template_Matching_with_Shift-Invariant_Semi-NMF.html">16 nips-2008-Adaptive Template Matching with Shift-Invariant Semi-NMF</a></p>
<p>11 0.41643587 <a title="11-lda-11" href="./nips-2008-Improved_Moves_for_Truncated_Convex_Models.html">104 nips-2008-Improved Moves for Truncated Convex Models</a></p>
<p>12 0.40843242 <a title="11-lda-12" href="./nips-2008-Mind_the_Duality_Gap%3A_Logarithmic_regret_algorithms_for_online_optimization.html">133 nips-2008-Mind the Duality Gap: Logarithmic regret algorithms for online optimization</a></p>
<p>13 0.40430331 <a title="11-lda-13" href="./nips-2008-Phase_transitions_for_high-dimensional_joint_support_recovery.html">179 nips-2008-Phase transitions for high-dimensional joint support recovery</a></p>
<p>14 0.40363863 <a title="11-lda-14" href="./nips-2008-Bounds_on_marginal_probability_distributions.html">40 nips-2008-Bounds on marginal probability distributions</a></p>
<p>15 0.39647365 <a title="11-lda-15" href="./nips-2008-Modeling_Short-term_Noise_Dependence_of_Spike_Counts_in_Macaque_Prefrontal_Cortex.html">137 nips-2008-Modeling Short-term Noise Dependence of Spike Counts in Macaque Prefrontal Cortex</a></p>
<p>16 0.39629146 <a title="11-lda-16" href="./nips-2008-Scalable_Algorithms_for_String_Kernels_with_Inexact_Matching.html">203 nips-2008-Scalable Algorithms for String Kernels with Inexact Matching</a></p>
<p>17 0.39562461 <a title="11-lda-17" href="./nips-2008-A_mixture_model_for_the_evolution_of_gene_expression_in_non-homogeneous_datasets.html">9 nips-2008-A mixture model for the evolution of gene expression in non-homogeneous datasets</a></p>
<p>18 0.39506972 <a title="11-lda-18" href="./nips-2008-A_general_framework_for_investigating_how_far_the_decoding_process_in_the_brain_can_be_simplified.html">8 nips-2008-A general framework for investigating how far the decoding process in the brain can be simplified</a></p>
<p>19 0.39422944 <a title="11-lda-19" href="./nips-2008-On_the_Generalization_Ability_of_Online_Strongly_Convex_Programming_Algorithms.html">164 nips-2008-On the Generalization Ability of Online Strongly Convex Programming Algorithms</a></p>
<p>20 0.39370275 <a title="11-lda-20" href="./nips-2008-Short-Term_Depression_in_VLSI_Stochastic_Synapse.html">209 nips-2008-Short-Term Depression in VLSI Stochastic Synapse</a></p>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
