<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>17 jmlr-2010-Bayesian Learning in Sparse Graphical Factor Models via Variational Mean-Field Annealing</title>
</head>

<body>
<p><a title="jmlr" href="../jmlr_home.html">jmlr</a> <a title="jmlr-2010" href="../home/jmlr2010_home.html">jmlr2010</a> <a title="jmlr-2010-17" href="../jmlr2010/jmlr-2010-Bayesian_Learning_in_Sparse_Graphical_Factor_Models_via_Variational_Mean-Field_Annealing.html">jmlr2010-17</a> <a title="jmlr-2010-17-reference" href="#">jmlr2010-17-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>17 jmlr-2010-Bayesian Learning in Sparse Graphical Factor Models via Variational Mean-Field Annealing</h1>
<br/><p>Source: <a title="jmlr-2010-17-pdf" href="http://jmlr.org/papers/volume11/yoshida10a/yoshida10a.pdf">pdf</a></p><p>Author: Ryo Yoshida, Mike West</p><p>Abstract: We describe a class of sparse latent factor models, called graphical factor models (GFMs), and relevant sparse learning algorithms for posterior mode estimation. Linear, Gaussian GFMs have sparse, orthogonal factor loadings matrices, that, in addition to sparsity of the implied covariance matrices, also induce conditional independence structures via zeros in the implied precision matrices. We describe the models and their use for robust estimation of sparse latent factor structure and data/signal reconstruction. We develop computational algorithms for model exploration and posterior mode search, addressing the hard combinatorial optimization involved in the search over a huge space of potential sparse conﬁgurations. A mean-ﬁeld variational technique coupled with annealing is developed to successively generate “artiﬁcial” posterior distributions that, at the limiting temperature in the annealing schedule, deﬁne required posterior modes in the GFM parameter space. Several detailed empirical studies and comparisons to related approaches are discussed, including analyses of handwritten digit image and cancer gene expression data. Keywords: annealing, graphical factor models, variational mean-ﬁeld method, MAP estimation, sparse factor analysis, gene expression proﬁling</p><br/>
<h2>reference text</h2><p>T.W. Anderson. An Introduction to Multivariate Statistical Analysis, 3rd ed. Wiley-Interscience; New Jersey, 2003. C. Archambeau and F. Bach. Sparse probabilistic projections. In D. Koller, D. Schuurmans, Y. Bengio, and L. Bottou, editors, Advances in Neural Information Processing Systems 21, pages 73–80, Cambridge, MA, 2009. MIT Press. H. Attias. Inferring parameters and structure of latent variable models by variational Bayes. Proceedings of the 15th Conference on Uncertainty in Artiﬁcial Intelligence (UAI99), pages 21–30, 1999. C.M. Bishop. Probabilistic principal component analysis. Journal of the Royal Statistical Society (Series B), 61:611–622, 1999. C.M. Bishop. Pattern Recognition and Machine Learning, 1st ed. Springer: Singapore, 2006. C.M. Carvalho and M. West. Dynamic matrix-variate graphical models. Bayesian Analysis, 2: 69–98, 2007. C.M. Carvalho, J.E. Lucas, Q. Wang, J.T. Chang, J.R. Nevins, and M. West. High-dimensional sparse factor modelling: Applications in gene expression genomics. Journal of the American Statistical Association, 103:1438–1456, 2008. J. Chang, C.M. Carvalho, S. Mori, A. Bild, Q. Wang, M. West, and J.R. Nevins. Decomposing cellular signaling pathways into functional units: A genomic strategy. Molecular Cell, 34:104– 114, 2009. 1796  G RAPHICAL FACTOR M ODELS AND VARIATIONAL M EAN -F IELD A NNEALING  S.H. Cheng, M. West C.F.Horng, E. Huang, J. Pittman, H. Dressman M.H. Tsou, C.M. Chen, S.Y. Tsail, J.J. Jian, J.R. Nevins M.C. Liu, and A.T. Huang. Genomic prediction of loco-regional recurrence following mastectomy in breast cancer. Journal of Clinical Oncology, 24:4594–4602, 2006. A. d’Aspremont, L. El Ghaoui, M.I. Jordan, and G.R.G. Lanckriet. A direct formulation for sparse PCA using semideﬁnite programming. SIAM Review, 49(3):434–448, 2007. A. Dobra, B. Jones, C. Hans, J.R. Nevins, and M. West. Sparse graphical models for exploring gene expression data. Journal of Multivariate Analysis, 90:196–212, 2004. T.L. Grifﬁths and Z Ghahramani. Inﬁnite latent feature models and the indian buffet process. In Y. Weiss, B. Sch¨ lkopf, and J. Platt, editors, Advances in Neural Information Processing Systems o 18, pages 475–482, Cambridge, MA, 2006. MIT Press. Y. Guan and J Dy. Sparse probabilistic principal component analysis. Proceedings of AISTATS 2009, 5:185–192, 2009. T. Hastie, R. Tibshirani, and J. Friedman. The Elements of Statistical Learning: Data Mining, Inference, and Prediction. Newv York: Springer, 2001. O. Hirose, R. Yoshida, S. Imoto, R. Yamaguchi, T. Higuchi, D. S. Charnock-Jones, C. Print, and S. Miyano. Statistical inference of transcriptional module-based gene networks from time course gene expression proﬁles by using state space models. Bioinformatics, 24(7):932–942, 2008. E. Huang, S Chen, H. K. Dressman, J. Pittman, M. H. Tsou, C. F Horng, A. Bild, E. S. Iversen, M. Liao, C. M. Chen, M. West, J. R. Nevins, and A. T. Huang. Gene expression predictors of breast cancer outcomes. The Lancet, 361:1590–1596, 2003. I. T. Jolliffe, N. T. Trendaﬁlov, and M. Uddin. A modiﬁed principal component technique based on the lasso. Journal of Computational and Graphical Statistics, 112(3):531–547, 2003. B. Jones, A. Dobra, C.M. Carvalho, C. Hans, C. Carter, and M. West. Experiments in stochastic computation for high-dimensional graphical models. Statistical Science, 20:388–400, 2005. M.I. Jordan, editor. Learning in Graphical Models. Cambridge MA: MIT Press, 1999. M.I. Jordan. Graphical models. Statistical Science, 19:140–155, 2004. M. Katoh and M. Katoh. Evolutionary recombination hotspot around GSDML-GSDM locus is closely linked to the oncogenomic recombination hotspot around the PPP1R1B-ERBB2-GRB7 amplicon. International Journal of Oncology, 24:757–63, 2004. J.E. Lucas, C.M. Carvalho, Q. Wang, A.H. Bild, J.R. Nevins, and M. West. Sparse statistical modelling in gene expression genomics. In P. M¨ ller, K.A. Do, and M. Vannucci, editors, Bayesian u Inference for Gene Expression and Proteomics, pages 155–176. Cambridge University Press, 2006. J.E. Lucas, C.M. Carvalho, J-T.A. Chi, and M. West. Cross-study projections of genomic biomarkers: An evaluation in cancer genomics. PLoS ONE, 4(2):e4523, 2009. 1797  YOSHIDA AND W EST  D.J.C. Mackay. Probable networks and plausible predictions - a review of practical Bayesian methods for supervised neural networks. Network: Computation in Neural Systems, 6:469–505, 1995. J. Mairal, F. Bach, J. Ponce, and G. Sapiro. Online dictionary learning for sparse coding. In Proceedings of the 26th Annual International Conference on Machine Learning (ICML2009), pages 689–696, New York, NY, USA, 2009. ACM. J. Pittman, E. Huang, H. Dressman, C. F. Horng, S. H. Cheng, M. H. Tsou, C. M. Chen, A. Bild, E. S. Iversen, A. T. Huang, J. R. Nevins, and M. West. Integrated modeling of clinical and gene expression information for personalized prediction of disease outcomes. Proceedings of the National Academy of Sciences, 101:8431–8436, 2004. P. Rai and H. Daum´ . The inﬁnite hierarchical factor regression model. In D. Koller, D. Schuurmans, e Y. Bengio, and L. Bottou, editors, Advances in Neural Information Processing Systems 21, pages 1321–1328, Cambridge, MA, 2009. MIT Press. N. Ueda and R. Nakano. Deterministic annealing EM algorithm. Neural Networks, 11:271–282, 1998. M.J. Wainwright and M.I. Jordan. Graphical models, exponential families, and variational inference. Foundations and Trends in Machine Learning, 1:1–305, 2008. Q. Wang, C.M. Carvalho, J.E. Lucas, and M. West. BFRM: Bayesian factor regression modelling. Bulletin of the International Society for Bayesian Analysis, 14(2):4–5, 2007. M. West. Bayesian factor regression models in the “large p, small n” paradigm. In J.M. Bernardo, M.J. Bayarri, J.O. Berger, A.P. Dawid, D. Heckerman, A.F.M. Smith, and M. West, editors, Bayesian Statistics 7, pages 723–732. Oxford University Press, 2003. M. West, C. Blanchette, H. Dressman, E. Huang, S. Ishida, H. Zuzan R. Spang, J.R. Marks, and J.R. Nevins. Predicting the clinical status of human breast cancer utilizing gene expression proﬁles. Proceedings of the National Academy of Sciences, 98:11462–11467, 2001. R. Yoshida, T. Higuchi, and S. Imoto. A mixed factors model for dimension reduction and extraction of a group structure in gene expression data. Proceedings of the 2004 IEEE Computational Systems Bioinformatics Conference, pages 161–172, 2004. R. Yoshida, T. Higuchi, S. Imoto, and S. Miyano. Arraycluster: an analytic tool for clustering, data visualization and module ﬁnder on gene expression proﬁles. Bioinformatics, 22(12):1538–1539, 2006. H. Zou, T. Hastie, and R. Tibshirani. Sparse principal component analysis. Journal of Computational and Graphical Statistics, 15(2):265–286, 2006.  1798</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
