<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>184 cvpr-2013-Gauging Association Patterns of Chromosome Territories via Chromatic Median</title>
</head>

<body>
<p><a title="cvpr" href="../cvpr_home.html">cvpr</a> <a title="cvpr-2013" href="../home/cvpr2013_home.html">cvpr2013</a> <a title="cvpr-2013-184" href="#">cvpr2013-184</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>184 cvpr-2013-Gauging Association Patterns of Chromosome Territories via Chromatic Median</h1>
<br/><p>Source: <a title="cvpr-2013-184-pdf" href="http://www.cv-foundation.org/openaccess/content_cvpr_2013/papers/Ding_Gauging_Association_Patterns_2013_CVPR_paper.pdf">pdf</a></p><p>Author: Hu Ding, Branislav Stojkovic, Ronald Berezney, Jinhui Xu</p><p>Abstract: Computing accurate and robust organizational patterns of chromosome territories inside the cell nucleus is critical for understanding several fundamental genomic processes, such as co-regulation of gene activation, gene silencing, X chromosome inactivation, and abnormal chromosome rearrangement in cancer cells. The usage of advanced fluorescence labeling and image processing techniques has enabled researchers to investigate interactions of chromosome territories at large spatial resolution. The resulting high volume of generated data demands for high-throughput and automated image analysis methods. In this paper, we introduce a novel algorithmic tool for investigating association patterns of chromosome territories in a population of cells. Our method takes as input a set of graphs, one for each cell, containing information about spatial interaction of chromosome territories, and yields a single graph that contains essential information for the whole population and stands as its structural representative. We formulate this combinato- rial problem as a semi-definite programming and present novel techniques to efficiently solve it. We validate our approach on both artificial and real biological data; the experimental results suggest that our approach yields a nearoptimal solution, and can handle large-size datasets, which are significant improvements over existing techniques.</p><p>Reference: <a title="cvpr-2013-184-reference" href="../cvpr2013_reference/cvpr-2013-Gauging_Association_Patterns_of_Chromosome_Territories_via_Chromatic_Median_reference.html">text</a></p><br/><h2>Summary: the most important sentenses genereted by tfidf model</h2><p>sentIndex sentText sentNum sentScore</p><p>1 The usage of advanced fluorescence labeling and image processing techniques has enabled researchers to investigate interactions of chromosome territories at large spatial resolution. [sent-5, score-0.846]
</p><p>2 In this paper, we introduce a novel algorithmic tool for investigating association patterns of chromosome territories in a population of cells. [sent-7, score-1.023]
</p><p>3 Our method takes as input a set of graphs, one for each cell, containing information about spatial interaction of chromosome territories, and yields a single graph that contains essential information for the whole population and stands as its structural representative. [sent-8, score-0.661]
</p><p>4 Chromosome Territories (CTs) are distinct regions within the cell nu∗This research was supported in part by NSF through a grant IIS1115220. [sent-14, score-0.101]
</p><p>5 Chromosome territories constitute a staple feature of nuclear architecture and are in constant dynamic interaction with other components of cell nucleus. [sent-16, score-0.406]
</p><p>6 Recent studies show influence of arrangements of chromosome territories on some fundamental cell molecular processes (e. [sent-17, score-0.949]
</p><p>7 For decades, determining how chromosomes are organized inside the cellular nucleus was a technically challenging process largely relying on manual measurements and observation ofexperts (see Fig. [sent-21, score-0.241]
</p><p>8 Thus, obtaining accurate and robust chromosome interaction patterns from cell nucleus images will significantly facilitate analysis and im-  prove overall understanding of molecular processes in cell nucleus [5, 6, 8, 10]. [sent-23, score-1.135]
</p><p>9 An effective way of representing high level chromosome territory organization in each cell is to use a graph data structure, where every vertex is an individual chromosome territory and every edge represents the association (spatial proximity) of a pair of neighboring chromosome territories. [sent-24, score-2.201]
</p><p>10 In a pair graph, each chromosome territory is associated with a chromosome number, and each chromosome number has multiplicity of two (one for each chromosome homolog). [sent-26, score-2.305]
</p><p>11 1a, each chromosome number has two chromosome homologs which share the same color. [sent-28, score-1.11]
</p><p>12 A graph that can be seen as representative for the set of pair graphs is called a Chromosome Association Pattern (CAP). [sent-29, score-0.188]
</p><p>13 Despite recent developments in microscopy imaging, and labeling techniques, our abilities to calculate CAP for a given cell population are still limited. [sent-30, score-0.132]
</p><p>14 1d  from a set of  is a label  3D microscopic nucleus images. [sent-33, score-0.201]
</p><p>15 Related Works Several models in literature are applicable to finding chromosome territories association patterns. [sent-40, score-0.973]
</p><p>16 The most popular one is median graph [16], which has received considerable attentions in recent years [11, 12, 14, 15]. [sent-41, score-0.296]
</p><p>17 Given a set of input graphs with possible labels associated with vertices, the objective of a median graph problem is to compute a new graph, called median graph, that has the minimum distance to the input graphs. [sent-42, score-0.654]
</p><p>18 Distance between two graphs is usually defined as their edit distance. [sent-43, score-0.142]
</p><p>19 In the context of finding association patterns of chromosome territories, GMG, as well as any other median graph method, has two major limitations. [sent-45, score-1.018]
</p><p>20 While this is in accordance with the edit distance definition, it gives mis-  leading semantic interpretation in our biological application where vertex labels denote chromosome territories. [sent-48, score-0.707]
</p><p>21 Secondly, GMG requires graphs with unique predefined labeling, this is in direct contrast with the uncertainty that we are facing in dealing with chromosome territory homologs. [sent-49, score-0.753]
</p><p>22 Our Model Aiming to fix the aforementioned problems in existing approaches, we first enumerate 2k labeled graphs for each k-pair association graph, with each labeled graph corresponding to a permutation of the original k-pair graph. [sent-58, score-0.434]
</p><p>23 Note that k is normally a small constant no more than 9 (this is due to the limitation of current labeling techniques in cell biology). [sent-59, score-0.101]
</p><p>24 Then we map the 2kn labeled graphs to points in some metric space, such that the pairwise distance of any two points is equal to the Jaccard distance  between the corresponding two labeled graphs. [sent-61, score-0.285]
</p><p>25 To solve the chromatic median problem, we first give a Quadratic Integer Programming model, then relax it to a Semi-definite Programming (SDP) problem, and propose a multi-level rounding technique to solve the SDP problem. [sent-66, score-0.559]
</p><p>26 It should be pointed out that although several rounding techniques exist for SDP [1, 13], the multi-level rounding technique is new, to the best of our knowledge. [sent-67, score-0.274]
</p><p>27 e An a unweighed graph I (V, E) is a k-pair association graph = for IM, if it satisfies  1r. [sent-75, score-0.298]
</p><p>28 Any two vertices from V are connect by an edge if the corresponding two chromosome territories are close 1 1 12 2 29 9 957 5  enough to each other in 3D space (based on some biological threshold). [sent-78, score-0.917]
</p><p>29 Figure 1c shows the k-pair association graph, where k = 8, for the nucleus image given in Figure 1a. [sent-79, score-0.316]
</p><p>30 From the above definition, it is easy to see that every kpair association graph has 2k different label differentiations. [sent-90, score-0.266]
</p><p>31 For any two given graphs G1 = (V, E1) and G2 = (V, E2) with the same labeled vertex set V , the Jaccard Distance between them is defined as  JD(G1,G2) = 1 −||EE11? [sent-94, score-0.195]
</p><p>32 First, we recall that median point in geometry is also called Fermat Weber point. [sent-106, score-0.221]
</p><p>33 For any set of n points {p1, p2, · · · ,pn} in Rd space, its median point is defined {asp  Med = argx m∈iRndn1i? [sent-107, score-0.244]
</p><p>34 Consequently, median point is often approximated by using some iterative procedure, such as Weiszfeld’s algorithm [21]. [sent-111, score-0.221]
</p><p>35 In Section 5, we present a semi-definite programming formulation, which helps us to identify each pili and yields a 2-approximation solution for Chromatic Median, where the approximation ratio is over the cost function (3). [sent-119, score-0.268]
</p><p>36 Formulation: from cap to chromatic median To solve the chromosome association pattern (CAP) problem, we use the following reduction from CAP to the Chromatic Median problem. [sent-121, score-1.21]
</p><p>37 For each of the n nucleus images IMi, build a kpair a esascohci oaftio thne graph. [sent-123, score-0.211]
</p><p>38 Enumerate the 2k possible Label Differentiations for each Ii, and generate the corresponding 2k labeled graphs {Ii1, · · · , I2ik }. [sent-126, score-0.153]
</p><p>39 For any two labeled graphs Iis1 and Iit2 , we compute the Jaccard distance between them. [sent-131, score-0.177]
</p><p>40 Since the Jac-  card distance satisfies triangle inequality [7], we map the 2kn labeled graphs to 2kn points in some metric space. [sent-132, score-0.222]
</p><p>41 For each group oflabeled graphs {Ii1, · · · , I2ik }, we dee. [sent-133, score-0.113]
</p><p>42 Find the n points {pl11 , · · · ,plnn } by the semi-definite programming model given in Section 5, where each pili is the nearest point to Chromatic Median among Pi. [sent-137, score-0.268]
</p><p>43 Mapping graphs into points: In Step 3 of the above reduction, we need to map all labeled graphs into points in some metric space. [sent-141, score-0.286]
</p><p>44 Our idea is not to explicitly build the metric space; instead, we formulate the Chromatic Median problem as a semi-definite programming (in Section 5) which only requires the pairwise distances among the set of mapped points (i. [sent-143, score-0.162]
</p><p>45 For a given set of labeled graphs {DGef1i , iGti2o , n· n· 6· , jGdn-}M wGit)h. [sent-150, score-0.153]
</p><p>46 2(bc)showsanexampleof the column graph corresponding to the 0-1 Quadratic Programming, where the vertices sharing the same column of vertices. [sent-152, score-0.11]
</p><p>47 n=1JD(G˜,Gi),  E˜),  (4)  where JD(·) denotes the Jaccard Distance between two graphs ( JseDe Definition s3) th. [sent-155, score-0.113]
</p><p>48 Ifwe consider each Gi as one point in some metric space, and the distance in the space is defined as the Jaccard Distance, then jd-MG is similar to the geometric median point otafn tchee, point-sets {GG is1 , Gim2i , ·a ·r ·t , Gthen g}e. [sent-156, score-0.265]
</p><p>49 Compute the median point of {ν1 , ν2 , · · · , νn} using W. [sent-165, score-0.221]
</p><p>50 Correspondingly, denote the graph as = (V, where the edges set is indicated by ˜ν (ρ). [sent-175, score-0.104]
</p><p>51 Semi-definite Model for Chromatic Median In this section, we show that it is not necessary to find the exact chromatic median CMed. [sent-182, score-0.422]
</p><p>52 Instead, it is sufficient to first find the n points {pl11 , · · · ,plnn }, where each plii is the nearest to CMed among all the points in Pi, and then output the Median Graph under Jaccard Distance for {I1l1, ··· , Ilnn} using the idea described in Section 4. [sent-183, score-0.241]
</p><p>53 jn=1 | |pjlj − plii | |, so as to avoid finding CMed. [sent-188, score-0.241]
</p><p>54 Since CMed is the geometric median point of {pl11 , · · · ,plnn }, by (2), we know that for any 1 ≤ j ≤ n, ? [sent-198, score-0.221]
</p><p>55 Mhaevaen ||pjlj − plii | | ≤ ||pjlj − CMed| | + | |pili − CMed| |. [sent-217, score-0.241]
</p><p>56 bove lemma suggests that minimizing the total pairwise distances enables us to obtain a 2-approximation for chromatic median. [sent-327, score-0.295]
</p><p>57 Next, we introduce a quadratic programming model for finding the desired n points which minimize the total pairwise distances. [sent-328, score-0.151]
</p><p>58 Finally, the 0-1 Quadratic Programming is to find a subgraph of the column graph with minimum total weight, where the subgraph contains exactly one ver-  =  n−y  wis1,,ti2  tex from each column. [sent-349, score-0.121]
</p><p>59 j=1  (15)  The above 0-1 quadratic programming indicates that if we compute its optimal solution, we immediately obtain exactly one indicator variable, say xlii? [sent-372, score-0.146]
</p><p>60 Then by Lemma 2, we know that the geometric median point of {pl11? [sent-376, score-0.221]
</p><p>61 The optimal solution of the above 0-1 quadratic programming yields a 2-approximation for the chromatic median problem. [sent-382, score-0.549]
</p><p>62 Semi-definite Programming Model Since the 0-1 quadratic programming is challenging to solve optimally [19], we convert it to a semi-definite programming model. [sent-385, score-0.225]
</p><p>63 We first build an equivalent 0-1 semidefinite programming (SDP), and then use rounding technique to solve it. [sent-386, score-0.288]
</p><p>64 The optimal solution of the above 0-1 semidefinite programming is equivalent to the optimal solution of the 0-1 quadratic programming. [sent-405, score-0.18]
</p><p>65 Consequently, it yields a 2-approximation for the chromatic median problem. [sent-406, score-0.422]
</p><p>66 Since X is not necessarily a 0d-e1fi mnitaetr mixa, troix xo Xbtai ∈n a feasible solution to CAP, we need to perform a rounding procedure on X to convert it into a 0-1 matrix. [sent-411, score-0.137]
</p><p>67 Below, we introduce a multiple-level rounding algorithm to achieve a better solution. [sent-414, score-0.137]
</p><p>68 Algorithm overview: Our algorithm performs log λ rounding steps iteratively. [sent-415, score-0.137]
</p><p>69 Finally, after log λ iterations, there is only one positive number among the diagonal elements for each Xi,i, which is the  final rounding result. [sent-418, score-0.163]
</p><p>70 In the first part we evaluate the performance of our method on synthetic datasets, with each dataset consisting of a number of randomly generated k-pair association graphs. [sent-454, score-0.177]
</p><p>71 In the second part, we apply our method for gauging the associations of chromosome territories in a population of cells belonging to WI 38 human lung fibroblast cell line. [sent-455, score-1.087]
</p><p>72 For each synthetic dataset, we randomly generate a k-pair association graph, I= (V, E), as the ground truth. [sent-461, score-0.177]
</p><p>73 Then we generate a set of kpair association graphs based on the ground truth with some percentage of input noise. [sent-462, score-0.304]
</p><p>74 Athec ground truth graph for each input graph would be 1−11−+pp = tMruothre gorvaeprh, we regard nitp as gthraep expected objective value for the CAP-graph obtained by our method. [sent-465, score-0.15]
</p><p>75 section we denote the number of (chromosome territory) pairs as k, the number of input k-pair association graphs in each dataset as n, and the input noise percentage as p. [sent-467, score-0.28]
</p><p>76 Table 1: Results for the multi-level rounding  12+pp). [sent-481, score-0.137]
</p><p>77 To determine the performance of the multilevel rounding technique, we show its comparison with the results obtained by single level round1 1 12 3239 90 91 9  aeAcngidrjevtas0 5. [sent-491, score-0.192]
</p><p>78 The Jaccard distance values for multilevel rounding are significantly lower than those obtained by single level rounding. [sent-502, score-0.216]
</p><p>79 Given two labeled graphs G1 = (V, E1) and G2 = (V, E2), Sorensen index is defined as Results are shown in Table 2, where our method achieves more than 10% improvements over all the three datasets, and the improvement becomes larger when the noise level increases. [sent-518, score-0.172]
</p><p>80 The usage of advanced fluorescence labeling and image processing techniques has enabled researchers to investigate interactions of  chromosome territories at large spatial resolution. [sent-532, score-0.846]
</p><p>81 Current limits of microscopic image acquisition process allow at most 9 pairs of chromosome territories to be labeled per cell nucleus. [sent-533, score-0.999]
</p><p>82 In this experiment, we focus on the study of associations of 8 chromosome territory pairs (with chromosome numbers from 1 to 9, except for 5) in WI 38 human lung fibroblast cells. [sent-534, score-1.3]
</p><p>83 We run our algorithm on dataset consisting of 90 microscope nucleus images. [sent-535, score-0.19]
</p><p>84 Each raw microscope image is a set of 2-D slices, where each slice corresponds to an image of the 3-D cell nucleus acquired at a certain focal plane. [sent-536, score-0.291]
</p><p>85 Individual k-pair association graph for each cell is derived using nearest border to border distances among chromosome territories. [sent-538, score-0.899]
</p><p>86 Like many biological problems, there is no ground truth for the CAP-graph on the cell nucleus images. [sent-540, score-0.326]
</p><p>87 For the association pattern, we compare the edges in our association pattern with those in the graph [22] generated by GMG [18] using the same dataset. [sent-559, score-0.4]
</p><p>88 Additionally, our method finds the following new association edges, where Table 4 shows them and their frequency among the dataset. [sent-561, score-0.148]
</p><p>89 Note that since the nucleus images are in 3D and the association patterns may appear in different slices, we only show the associations in some slices. [sent-565, score-0.375]
</p><p>90 From Figure 4b, it is easy to see that the labeled pairs of chromosome territories are indeed close to each other, which also confirms the effectiveness of our approach. [sent-566, score-0.865]
</p><p>91 Table 4: Additional association edges discovered by our method Edges4 − 63 − 46 − 77 − 2 4  −  63  Frequency  −  56. [sent-567, score-0.177]
</p><p>92 Summary and Discussion In this paper we present an efficient method for recognizing the pattern of associations for chromosome territories in the cell nucleus. [sent-572, score-0.966]
</p><p>93 Experiments using synthetic datasets reveal the scalability and high efficiency of our method, while the experiment on a cell nucleus image dataset shows the accuracy of our method for finding the association pattern of human chromosome terrirtories. [sent-575, score-1.001]
</p><p>94 A linear programming approach for the weighted graph matching problem. [sent-583, score-0.173]
</p><p>95 Three-dimensional maps of all chromosomes in human male fibroblast nuclei and prometaphase rosettes. [sent-618, score-0.116]
</p><p>96 Chromosome territories, nuclear architecture and gene regulation in mamalian cells. [sent-639, score-0.104]
</p><p>97 A generic framework for median graph computation based on a recursive embedding approach. [sent-660, score-0.296]
</p><p>98 Generalized median graph computation by means of graph embedding in vector spaces. [sent-667, score-0.371]
</p><p>99 A binary linear programming formulation of the graph edit distance. [sent-686, score-0.202]
</p><p>100 A probabilistic model for the arrangement of a subset of human chromosome territories in wi38 human fibroblasts. [sent-743, score-0.825]
</p>
<br/>
<h2>similar papers computed by tfidf model</h2><h3>tfidf for this paper:</h3><p>wordName wordTfidf (topN-words)</p>
<p>[('chromosome', 0.555), ('cmed', 0.313), ('territories', 0.27), ('plii', 0.241), ('median', 0.221), ('chromatic', 0.201), ('jaccard', 0.201), ('pili', 0.17), ('nucleus', 0.168), ('association', 0.148), ('rounding', 0.137), ('pljj', 0.128), ('gmg', 0.114), ('graphs', 0.113), ('cell', 0.101), ('programming', 0.098), ('sdp', 0.091), ('mag', 0.085), ('territory', 0.085), ('cap', 0.085), ('graph', 0.075), ('sorensen', 0.071), ('pli', 0.07), ('biological', 0.057), ('cremer', 0.057), ('multilevel', 0.055), ('semidefinite', 0.053), ('med', 0.052), ('chromosomes', 0.05), ('lemma', 0.05), ('gene', 0.044), ('fibroblast', 0.043), ('kpair', 0.043), ('pjlj', 0.043), ('stojkovic', 0.043), ('zeitz', 0.043), ('vertex', 0.042), ('xji', 0.04), ('associations', 0.04), ('labeled', 0.04), ('buffalo', 0.038), ('deletion', 0.038), ('plj', 0.038), ('fal', 0.035), ('nuclear', 0.035), ('jd', 0.035), ('vertices', 0.035), ('microscopic', 0.033), ('population', 0.031), ('bu', 0.031), ('synthetic', 0.029), ('quadratic', 0.029), ('edit', 0.029), ('edges', 0.029), ('arora', 0.028), ('boyle', 0.028), ('bridger', 0.028), ('ferrer', 0.028), ('gcap', 0.028), ('homolog', 0.028), ('mller', 0.028), ('solovei', 0.028), ('williamson', 0.028), ('definition', 0.028), ('jn', 0.027), ('pi', 0.027), ('speedup', 0.027), ('diagonal', 0.026), ('adaptive', 0.026), ('goemans', 0.025), ('genome', 0.025), ('regulation', 0.025), ('gauging', 0.025), ('ilnn', 0.025), ('inequality', 0.025), ('slices', 0.024), ('pairwise', 0.024), ('distance', 0.024), ('valveny', 0.023), ('molecular', 0.023), ('hardness', 0.023), ('nuclei', 0.023), ('cellular', 0.023), ('argx', 0.023), ('subgraph', 0.023), ('microscope', 0.022), ('indi', 0.022), ('lung', 0.022), ('fluorescence', 0.021), ('mukherjee', 0.02), ('metric', 0.02), ('distances', 0.02), ('immediately', 0.019), ('noise', 0.019), ('patterns', 0.019), ('sampling', 0.018), ('side', 0.018), ('ei', 0.018), ('enumerate', 0.018), ('pp', 0.018)]</p>
<h3>similar papers list:</h3><p>simIndex simValue paperId paperTitle</p>
<p>same-paper 1 1.0000014 <a title="184-tfidf-1" href="./cvpr-2013-Gauging_Association_Patterns_of_Chromosome_Territories_via_Chromatic_Median.html">184 cvpr-2013-Gauging Association Patterns of Chromosome Territories via Chromatic Median</a></p>
<p>Author: Hu Ding, Branislav Stojkovic, Ronald Berezney, Jinhui Xu</p><p>Abstract: Computing accurate and robust organizational patterns of chromosome territories inside the cell nucleus is critical for understanding several fundamental genomic processes, such as co-regulation of gene activation, gene silencing, X chromosome inactivation, and abnormal chromosome rearrangement in cancer cells. The usage of advanced fluorescence labeling and image processing techniques has enabled researchers to investigate interactions of chromosome territories at large spatial resolution. The resulting high volume of generated data demands for high-throughput and automated image analysis methods. In this paper, we introduce a novel algorithmic tool for investigating association patterns of chromosome territories in a population of cells. Our method takes as input a set of graphs, one for each cell, containing information about spatial interaction of chromosome territories, and yields a single graph that contains essential information for the whole population and stands as its structural representative. We formulate this combinato- rial problem as a semi-definite programming and present novel techniques to efficiently solve it. We validate our approach on both artificial and real biological data; the experimental results suggest that our approach yields a nearoptimal solution, and can handle large-size datasets, which are significant improvements over existing techniques.</p><p>2 0.13568354 <a title="184-tfidf-2" href="./cvpr-2013-A_Genetic_Algorithm-Based_Solver_for_Very_Large_Jigsaw_Puzzles.html">11 cvpr-2013-A Genetic Algorithm-Based Solver for Very Large Jigsaw Puzzles</a></p>
<p>Author: Dror Sholomon, Omid David, Nathan S. Netanyahu</p><p>Abstract: In thispaper wepropose thefirst effective automated, genetic algorithm (GA)-based jigsaw puzzle solver. We introduce a novel procedure of merging two ”parent” solutions to an improved ”child” solution by detecting, extracting, and combining correctly assembled puzzle segments. The solver proposed exhibits state-of-the-art performance solving previously attempted puzzles faster and far more accurately, and also puzzles of size never before attempted. Other contributions include the creation of a benchmark of large images, previously unavailable. We share the data sets and all of our results for future testing and comparative evaluation of jigsaw puzzle solvers.</p><p>3 0.08389312 <a title="184-tfidf-3" href="./cvpr-2013-Multi-target_Tracking_by_Rank-1_Tensor_Approximation.html">301 cvpr-2013-Multi-target Tracking by Rank-1 Tensor Approximation</a></p>
<p>Author: Xinchu Shi, Haibin Ling, Junling Xing, Weiming Hu</p><p>Abstract: In this paper we formulate multi-target tracking (MTT) as a rank-1 tensor approximation problem and propose an ?1 norm tensor power iteration solution. In particular, a high order tensor is constructed based on trajectories in the time window, with each tensor element as the affinity of the corresponding trajectory candidate. The local assignment variables are the ?1 normalized vectors, which are used to approximate the rank-1 tensor. Our approach provides a flexible and effective formulation where both pairwise and high-order association energies can be used expediently. We also show the close relation between our formulation and the multi-dimensional assignment (MDA) model. To solve the optimization in the rank-1 tensor approximation, we propose an algorithm that iteratively powers the intermediate solution followed by an ?1 normalization. Aside from effectively capturing high-order motion information, the proposed solver runs efficiently with proved convergence. The experimental validations are conducted on two challenging datasets and our method demonstrates promising performances on both.</p><p>4 0.077589117 <a title="184-tfidf-4" href="./cvpr-2013-A_Fast_Semidefinite_Approach_to_Solving_Binary_Quadratic_Problems.html">9 cvpr-2013-A Fast Semidefinite Approach to Solving Binary Quadratic Problems</a></p>
<p>Author: Peng Wang, Chunhua Shen, Anton van_den_Hengel</p><p>Abstract: Many computer vision problems can be formulated as binary quadratic programs (BQPs). Two classic relaxation methods are widely used for solving BQPs, namely, spectral methods and semidefinite programming (SDP), each with their own advantages and disadvantages. Spectral relaxation is simple and easy to implement, but its bound is loose. Semidefinite relaxation has a tighter bound, but its computational complexity is high for large scale problems. We present a new SDP formulation for BQPs, with two desirable properties. First, it has a similar relaxation bound to conventional SDP formulations. Second, compared with conventional SDP methods, the new SDP formulation leads to a significantly more efficient and scalable dual optimization approach, which has the same degree of complexity as spectral methods. Extensive experiments on various applications including clustering, image segmentation, co-segmentation and registration demonstrate the usefulness of our SDP formulation for solving large-scale BQPs.</p><p>5 0.055882324 <a title="184-tfidf-5" href="./cvpr-2013-Graph_Matching_with_Anchor_Nodes%3A_A_Learning_Approach.html">192 cvpr-2013-Graph Matching with Anchor Nodes: A Learning Approach</a></p>
<p>Author: Nan Hu, Raif M. Rustamov, Leonidas Guibas</p><p>Abstract: In this paper, we consider the weighted graph matching problem with partially disclosed correspondences between a number of anchor nodes. Our construction exploits recently introduced node signatures based on graph Laplacians, namely the Laplacian family signature (LFS) on the nodes, and the pairwise heat kernel map on the edges. In this paper, without assuming an explicit form of parametric dependence nor a distance metric between node signatures, we formulate an optimization problem which incorporates the knowledge of anchor nodes. Solving this problem gives us an optimized proximity measure specific to the graphs under consideration. Using this as a first order compatibility term, we then set up an integer quadratic program (IQP) to solve for a near optimal graph matching. Our experiments demonstrate the superior performance of our approach on randomly generated graphs and on two widelyused image sequences, when compared with other existing signature and adjacency matrix based graph matching methods.</p><p>6 0.051912531 <a title="184-tfidf-6" href="./cvpr-2013-Graph-Based_Discriminative_Learning_for_Location_Recognition.html">189 cvpr-2013-Graph-Based Discriminative Learning for Location Recognition</a></p>
<p>7 0.050351359 <a title="184-tfidf-7" href="./cvpr-2013-Unsupervised_Joint_Object_Discovery_and_Segmentation_in_Internet_Images.html">450 cvpr-2013-Unsupervised Joint Object Discovery and Segmentation in Internet Images</a></p>
<p>8 0.049831994 <a title="184-tfidf-8" href="./cvpr-2013-Evaluation_of_Color_STIPs_for_Human_Action_Recognition.html">149 cvpr-2013-Evaluation of Color STIPs for Human Action Recognition</a></p>
<p>9 0.049157858 <a title="184-tfidf-9" href="./cvpr-2013-Perceptual_Organization_and_Recognition_of_Indoor_Scenes_from_RGB-D_Images.html">329 cvpr-2013-Perceptual Organization and Recognition of Indoor Scenes from RGB-D Images</a></p>
<p>10 0.047949392 <a title="184-tfidf-10" href="./cvpr-2013-Information_Consensus_for_Distributed_Multi-target_Tracking.html">224 cvpr-2013-Information Consensus for Distributed Multi-target Tracking</a></p>
<p>11 0.047826324 <a title="184-tfidf-11" href="./cvpr-2013-Fast_Energy_Minimization_Using_Learned_State_Filters.html">165 cvpr-2013-Fast Energy Minimization Using Learned State Filters</a></p>
<p>12 0.045831818 <a title="184-tfidf-12" href="./cvpr-2013-Multi-target_Tracking_by_Lagrangian_Relaxation_to_Min-cost_Network_Flow.html">300 cvpr-2013-Multi-target Tracking by Lagrangian Relaxation to Min-cost Network Flow</a></p>
<p>13 0.043461997 <a title="184-tfidf-13" href="./cvpr-2013-Bilinear_Programming_for_Human_Activity_Recognition_with_Unknown_MRF_Graphs.html">62 cvpr-2013-Bilinear Programming for Human Activity Recognition with Unknown MRF Graphs</a></p>
<p>14 0.043127637 <a title="184-tfidf-14" href="./cvpr-2013-Efficient_Color_Boundary_Detection_with_Color-Opponent_Mechanisms.html">140 cvpr-2013-Efficient Color Boundary Detection with Color-Opponent Mechanisms</a></p>
<p>15 0.042041767 <a title="184-tfidf-15" href="./cvpr-2013-Learning_to_Detect_Partially_Overlapping_Instances.html">264 cvpr-2013-Learning to Detect Partially Overlapping Instances</a></p>
<p>16 0.04162766 <a title="184-tfidf-16" href="./cvpr-2013-Tracking_Sports_Players_with_Context-Conditioned_Motion_Models.html">441 cvpr-2013-Tracking Sports Players with Context-Conditioned Motion Models</a></p>
<p>17 0.03981882 <a title="184-tfidf-17" href="./cvpr-2013-Learning_for_Structured_Prediction_Using_Approximate_Subgradient_Descent_with_Working_Sets.html">262 cvpr-2013-Learning for Structured Prediction Using Approximate Subgradient Descent with Working Sets</a></p>
<p>18 0.039484121 <a title="184-tfidf-18" href="./cvpr-2013-Detection-_and_Trajectory-Level_Exclusion_in_Multiple_Object_Tracking.html">121 cvpr-2013-Detection- and Trajectory-Level Exclusion in Multiple Object Tracking</a></p>
<p>19 0.03901216 <a title="184-tfidf-19" href="./cvpr-2013-Incorporating_User_Interaction_and_Topological_Constraints_within_Contour_Completion_via_Discrete_Calculus.html">222 cvpr-2013-Incorporating User Interaction and Topological Constraints within Contour Completion via Discrete Calculus</a></p>
<p>20 0.03845666 <a title="184-tfidf-20" href="./cvpr-2013-PDM-ENLOR%3A_Learning_Ensemble_of_Local_PDM-Based_Regressions.html">321 cvpr-2013-PDM-ENLOR: Learning Ensemble of Local PDM-Based Regressions</a></p>
<br/>
<h2>similar papers computed by <a title="lsi-model" href="../home/cvpr2013_lsi.html">lsi model</a></h2><h3>lsi for this paper:</h3><p>topicId topicWeight</p>
<p>[(0, 0.089), (1, 0.013), (2, 0.002), (3, 0.011), (4, 0.032), (5, -0.004), (6, -0.007), (7, -0.026), (8, -0.031), (9, 0.009), (10, 0.029), (11, 0.007), (12, -0.049), (13, -0.024), (14, -0.021), (15, -0.007), (16, 0.01), (17, 0.004), (18, 0.062), (19, 0.006), (20, -0.018), (21, 0.038), (22, -0.054), (23, 0.035), (24, 0.057), (25, -0.015), (26, 0.047), (27, -0.025), (28, -0.01), (29, 0.017), (30, -0.001), (31, 0.002), (32, 0.035), (33, -0.06), (34, 0.057), (35, 0.033), (36, 0.033), (37, 0.019), (38, 0.055), (39, 0.018), (40, 0.003), (41, 0.005), (42, 0.001), (43, 0.054), (44, 0.042), (45, -0.037), (46, 0.034), (47, -0.048), (48, -0.003), (49, -0.033)]</p>
<h3>similar papers list:</h3><p>simIndex simValue paperId paperTitle</p>
<p>same-paper 1 0.89397788 <a title="184-lsi-1" href="./cvpr-2013-Gauging_Association_Patterns_of_Chromosome_Territories_via_Chromatic_Median.html">184 cvpr-2013-Gauging Association Patterns of Chromosome Territories via Chromatic Median</a></p>
<p>Author: Hu Ding, Branislav Stojkovic, Ronald Berezney, Jinhui Xu</p><p>Abstract: Computing accurate and robust organizational patterns of chromosome territories inside the cell nucleus is critical for understanding several fundamental genomic processes, such as co-regulation of gene activation, gene silencing, X chromosome inactivation, and abnormal chromosome rearrangement in cancer cells. The usage of advanced fluorescence labeling and image processing techniques has enabled researchers to investigate interactions of chromosome territories at large spatial resolution. The resulting high volume of generated data demands for high-throughput and automated image analysis methods. In this paper, we introduce a novel algorithmic tool for investigating association patterns of chromosome territories in a population of cells. Our method takes as input a set of graphs, one for each cell, containing information about spatial interaction of chromosome territories, and yields a single graph that contains essential information for the whole population and stands as its structural representative. We formulate this combinato- rial problem as a semi-definite programming and present novel techniques to efficiently solve it. We validate our approach on both artificial and real biological data; the experimental results suggest that our approach yields a nearoptimal solution, and can handle large-size datasets, which are significant improvements over existing techniques.</p><p>2 0.74025595 <a title="184-lsi-2" href="./cvpr-2013-Deformable_Graph_Matching.html">106 cvpr-2013-Deformable Graph Matching</a></p>
<p>Author: Feng Zhou, Fernando De_la_Torre</p><p>Abstract: Graph matching (GM) is a fundamental problem in computer science, and it has been successfully applied to many problems in computer vision. Although widely used, existing GM algorithms cannot incorporate global consistence among nodes, which is a natural constraint in computer vision problems. This paper proposes deformable graph matching (DGM), an extension of GM for matching graphs subject to global rigid and non-rigid geometric constraints. The key idea of this work is a new factorization of the pair-wise affinity matrix. This factorization decouples the affinity matrix into the local structure of each graph and the pair-wise affinity edges. Besides the ability to incorporate global geometric transformations, this factorization offers three more benefits. First, there is no need to compute the costly (in space and time) pair-wise affinity matrix. Second, it provides a unified view of many GM methods and extends the standard iterative closest point algorithm. Third, it allows to use the path-following optimization algorithm that leads to improved optimization strategies and matching performance. Experimental results on synthetic and real databases illustrate how DGM outperforms state-of-the-art algorithms for GM. The code is available at http : / / human s en s ing . c s . cmu .edu / fgm.</p><p>3 0.72987044 <a title="184-lsi-3" href="./cvpr-2013-Graph_Matching_with_Anchor_Nodes%3A_A_Learning_Approach.html">192 cvpr-2013-Graph Matching with Anchor Nodes: A Learning Approach</a></p>
<p>Author: Nan Hu, Raif M. Rustamov, Leonidas Guibas</p><p>Abstract: In this paper, we consider the weighted graph matching problem with partially disclosed correspondences between a number of anchor nodes. Our construction exploits recently introduced node signatures based on graph Laplacians, namely the Laplacian family signature (LFS) on the nodes, and the pairwise heat kernel map on the edges. In this paper, without assuming an explicit form of parametric dependence nor a distance metric between node signatures, we formulate an optimization problem which incorporates the knowledge of anchor nodes. Solving this problem gives us an optimized proximity measure specific to the graphs under consideration. Using this as a first order compatibility term, we then set up an integer quadratic program (IQP) to solve for a near optimal graph matching. Our experiments demonstrate the superior performance of our approach on randomly generated graphs and on two widelyused image sequences, when compared with other existing signature and adjacency matrix based graph matching methods.</p><p>4 0.6886999 <a title="184-lsi-4" href="./cvpr-2013-A_Genetic_Algorithm-Based_Solver_for_Very_Large_Jigsaw_Puzzles.html">11 cvpr-2013-A Genetic Algorithm-Based Solver for Very Large Jigsaw Puzzles</a></p>
<p>Author: Dror Sholomon, Omid David, Nathan S. Netanyahu</p><p>Abstract: In thispaper wepropose thefirst effective automated, genetic algorithm (GA)-based jigsaw puzzle solver. We introduce a novel procedure of merging two ”parent” solutions to an improved ”child” solution by detecting, extracting, and combining correctly assembled puzzle segments. The solver proposed exhibits state-of-the-art performance solving previously attempted puzzles faster and far more accurately, and also puzzles of size never before attempted. Other contributions include the creation of a benchmark of large images, previously unavailable. We share the data sets and all of our results for future testing and comparative evaluation of jigsaw puzzle solvers.</p><p>5 0.63292801 <a title="184-lsi-5" href="./cvpr-2013-Graph_Transduction_Learning_with_Connectivity_Constraints_with_Application_to_Multiple_Foreground_Cosegmentation.html">193 cvpr-2013-Graph Transduction Learning with Connectivity Constraints with Application to Multiple Foreground Cosegmentation</a></p>
<p>Author: Tianyang Ma, Longin Jan Latecki</p><p>Abstract: The proposed approach is based on standard graph transduction, semi-supervised learning (SSL) framework. Its key novelty is the integration of global connectivity constraints into this framework. Although connectivity leads to higher order constraints and their number is an exponential, finding the most violated connectivity constraint can be done efficiently in polynomial time. Moreover, each such constraint can be represented as a linear inequality. Based on this fact, we design a cutting-plane algorithm to solve the integrated problem. It iterates between solving a convex quadraticproblem of labelpropagation with linear inequality constraints, and finding the most violated constraint. We demonstrate the benefits of the proposed approach on a realistic and very challenging problem of cosegmentation of multiple foreground objects in photo collections in which the foreground objects are not present in all photos. The obtained results not only demonstrate performance boost induced by the connectivity constraints, but also show a significant improvement over the state-of-the-art methods.</p><p>6 0.61422503 <a title="184-lsi-6" href="./cvpr-2013-Reconstructing_Loopy_Curvilinear_Structures_Using_Integer_Programming.html">350 cvpr-2013-Reconstructing Loopy Curvilinear Structures Using Integer Programming</a></p>
<p>7 0.59578663 <a title="184-lsi-7" href="./cvpr-2013-A_Fast_Semidefinite_Approach_to_Solving_Binary_Quadratic_Problems.html">9 cvpr-2013-A Fast Semidefinite Approach to Solving Binary Quadratic Problems</a></p>
<p>8 0.57899791 <a title="184-lsi-8" href="./cvpr-2013-Bilinear_Programming_for_Human_Activity_Recognition_with_Unknown_MRF_Graphs.html">62 cvpr-2013-Bilinear Programming for Human Activity Recognition with Unknown MRF Graphs</a></p>
<p>9 0.56099802 <a title="184-lsi-9" href="./cvpr-2013-Maximum_Cohesive_Grid_of_Superpixels_for_Fast_Object_Localization.html">280 cvpr-2013-Maximum Cohesive Grid of Superpixels for Fast Object Localization</a></p>
<p>10 0.55547148 <a title="184-lsi-10" href="./cvpr-2013-Multi-target_Tracking_by_Lagrangian_Relaxation_to_Min-cost_Network_Flow.html">300 cvpr-2013-Multi-target Tracking by Lagrangian Relaxation to Min-cost Network Flow</a></p>
<p>11 0.55434626 <a title="184-lsi-11" href="./cvpr-2013-Information_Consensus_for_Distributed_Multi-target_Tracking.html">224 cvpr-2013-Information Consensus for Distributed Multi-target Tracking</a></p>
<p>12 0.55050713 <a title="184-lsi-12" href="./cvpr-2013-Joint_Spectral_Correspondence_for_Disparate_Image_Matching.html">234 cvpr-2013-Joint Spectral Correspondence for Disparate Image Matching</a></p>
<p>13 0.54939336 <a title="184-lsi-13" href="./cvpr-2013-Graph-Based_Optimization_with_Tubularity_Markov_Tree_for_3D_Vessel_Segmentation.html">190 cvpr-2013-Graph-Based Optimization with Tubularity Markov Tree for 3D Vessel Segmentation</a></p>
<p>14 0.54463881 <a title="184-lsi-14" href="./cvpr-2013-Consensus_of_k-NNs_for_Robust_Neighborhood_Selection_on_Graph-Based_Manifolds.html">91 cvpr-2013-Consensus of k-NNs for Robust Neighborhood Selection on Graph-Based Manifolds</a></p>
<p>15 0.54320914 <a title="184-lsi-15" href="./cvpr-2013-The_Generalized_Laplacian_Distance_and_Its_Applications_for_Visual_Matching.html">429 cvpr-2013-The Generalized Laplacian Distance and Its Applications for Visual Matching</a></p>
<p>16 0.53533125 <a title="184-lsi-16" href="./cvpr-2013-Incorporating_User_Interaction_and_Topological_Constraints_within_Contour_Completion_via_Discrete_Calculus.html">222 cvpr-2013-Incorporating User Interaction and Topological Constraints within Contour Completion via Discrete Calculus</a></p>
<p>17 0.53469998 <a title="184-lsi-17" href="./cvpr-2013-Keypoints_from_Symmetries_by_Wave_Propagation.html">240 cvpr-2013-Keypoints from Symmetries by Wave Propagation</a></p>
<p>18 0.53141659 <a title="184-lsi-18" href="./cvpr-2013-Constraints_as_Features.html">93 cvpr-2013-Constraints as Features</a></p>
<p>19 0.52930695 <a title="184-lsi-19" href="./cvpr-2013-Towards_Efficient_and_Exact_MAP-Inference_for_Large_Scale_Discrete_Computer_Vision_Problems_via_Combinatorial_Optimization.html">436 cvpr-2013-Towards Efficient and Exact MAP-Inference for Large Scale Discrete Computer Vision Problems via Combinatorial Optimization</a></p>
<p>20 0.5189749 <a title="184-lsi-20" href="./cvpr-2013-Fast_Energy_Minimization_Using_Learned_State_Filters.html">165 cvpr-2013-Fast Energy Minimization Using Learned State Filters</a></p>
<br/>
<h2>similar papers computed by <a title="lda-model" href="../home/cvpr2013_lda.html">lda model</a></h2><h3>lda for this paper:</h3><p>topicId topicWeight</p>
<p>[(3, 0.016), (10, 0.097), (16, 0.017), (26, 0.042), (33, 0.193), (39, 0.014), (59, 0.012), (65, 0.012), (67, 0.03), (69, 0.046), (87, 0.046), (94, 0.37)]</p>
<h3>similar papers list:</h3><p>simIndex simValue paperId paperTitle</p>
<p>same-paper 1 0.68742526 <a title="184-lda-1" href="./cvpr-2013-Gauging_Association_Patterns_of_Chromosome_Territories_via_Chromatic_Median.html">184 cvpr-2013-Gauging Association Patterns of Chromosome Territories via Chromatic Median</a></p>
<p>Author: Hu Ding, Branislav Stojkovic, Ronald Berezney, Jinhui Xu</p><p>Abstract: Computing accurate and robust organizational patterns of chromosome territories inside the cell nucleus is critical for understanding several fundamental genomic processes, such as co-regulation of gene activation, gene silencing, X chromosome inactivation, and abnormal chromosome rearrangement in cancer cells. The usage of advanced fluorescence labeling and image processing techniques has enabled researchers to investigate interactions of chromosome territories at large spatial resolution. The resulting high volume of generated data demands for high-throughput and automated image analysis methods. In this paper, we introduce a novel algorithmic tool for investigating association patterns of chromosome territories in a population of cells. Our method takes as input a set of graphs, one for each cell, containing information about spatial interaction of chromosome territories, and yields a single graph that contains essential information for the whole population and stands as its structural representative. We formulate this combinato- rial problem as a semi-definite programming and present novel techniques to efficiently solve it. We validate our approach on both artificial and real biological data; the experimental results suggest that our approach yields a nearoptimal solution, and can handle large-size datasets, which are significant improvements over existing techniques.</p><p>2 0.63285732 <a title="184-lda-2" href="./cvpr-2013-Shape_from_Silhouette_Probability_Maps%3A_Reconstruction_of_Thin_Objects_in_the_Presence_of_Silhouette_Extraction_and_Calibration_Error.html">395 cvpr-2013-Shape from Silhouette Probability Maps: Reconstruction of Thin Objects in the Presence of Silhouette Extraction and Calibration Error</a></p>
<p>Author: Amy Tabb</p><p>Abstract: This paper considers the problem of reconstructing the shape ofthin, texture-less objects such as leafless trees when there is noise or deterministic error in the silhouette extraction step or there are small errors in camera calibration. Traditional intersection-based techniques such as the visual hull are not robust to error because they penalize false negative and false positive error unequally. We provide a voxel-based formalism that penalizes false negative and positive error equally, by casting the reconstruction problem as a pseudo-Boolean minimization problem, where voxels are the variables of a pseudo-Boolean function and are labeled occupied or empty. Since the pseudo-Boolean minimization problem is NP-Hard for nonsubmodular functions, we developed an algorithm for an approximate solution using local minimum search. Our algorithm treats input binary probability maps (in other words, silhouettes) or continuously-valued probability maps identically, and places no constraints on camera placement. The algorithm was tested on three different leafless trees and one metal object where the number of voxels is 54.4 million (voxel sides measure 3.6 mm). Results show that our . usda .gov (a)Orignalimage(b)SilhoueteProbabiltyMap approach reconstructs the complicated branching structure of thin, texture-less objects in the presence of error where intersection-based approaches currently fail. 1</p><p>3 0.54912859 <a title="184-lda-3" href="./cvpr-2013-Adaptive_Active_Learning_for_Image_Classification.html">34 cvpr-2013-Adaptive Active Learning for Image Classification</a></p>
<p>Author: Xin Li, Yuhong Guo</p><p>Abstract: Recently active learning has attracted a lot of attention in computer vision field, as it is time and cost consuming to prepare a good set of labeled images for vision data analysis. Most existing active learning approaches employed in computer vision adopt most uncertainty measures as instance selection criteria. Although most uncertainty query selection strategies are very effective in many circumstances, they fail to take information in the large amount of unlabeled instances into account and are prone to querying outliers. In this paper, we present a novel adaptive active learning approach that combines an information density measure and a most uncertainty measure together to select critical instances to label for image classifications. Our experiments on two essential tasks of computer vision, object recognition and scene recognition, demonstrate the efficacy of the proposed approach.</p><p>4 0.54906201 <a title="184-lda-4" href="./cvpr-2013-Histograms_of_Sparse_Codes_for_Object_Detection.html">204 cvpr-2013-Histograms of Sparse Codes for Object Detection</a></p>
<p>Author: Xiaofeng Ren, Deva Ramanan</p><p>Abstract: Object detection has seen huge progress in recent years, much thanks to the heavily-engineered Histograms of Oriented Gradients (HOG) features. Can we go beyond gradients and do better than HOG? Weprovide an affirmative answer byproposing and investigating a sparse representation for object detection, Histograms of Sparse Codes (HSC). We compute sparse codes with dictionaries learned from data using K-SVD, and aggregate per-pixel sparse codes to form local histograms. We intentionally keep true to the sliding window framework (with mixtures and parts) and only change the underlying features. To keep training (and testing) efficient, we apply dimension reduction by computing SVD on learned models, and adopt supervised training where latent positions of roots and parts are given externally e.g. from a HOG-based detector. By learning and using local representations that are much more expressive than gradients, we demonstrate large improvements over the state of the art on the PASCAL benchmark for both root- only and part-based models.</p><p>5 0.54839402 <a title="184-lda-5" href="./cvpr-2013-Analyzing_Semantic_Segmentation_Using_Hybrid_Human-Machine_CRFs.html">43 cvpr-2013-Analyzing Semantic Segmentation Using Hybrid Human-Machine CRFs</a></p>
<p>Author: Roozbeh Mottaghi, Sanja Fidler, Jian Yao, Raquel Urtasun, Devi Parikh</p><p>Abstract: Recent trends in semantic image segmentation have pushed for holistic scene understanding models that jointly reason about various tasks such as object detection, scene recognition, shape analysis, contextual reasoning. In this work, we are interested in understanding the roles of these different tasks in aiding semantic segmentation. Towards this goal, we “plug-in ” human subjects for each of the various components in a state-of-the-art conditional random field model (CRF) on the MSRC dataset. Comparisons among various hybrid human-machine CRFs give us indications of how much “head room ” there is to improve segmentation by focusing research efforts on each of the tasks. One of the interesting findings from our slew of studies was that human classification of isolated super-pixels, while being worse than current machine classifiers, provides a significant boost in performance when plugged into the CRF! Fascinated by this finding, we conducted in depth analysis of the human generated potentials. This inspired a new machine potential which significantly improves state-of-the-art performance on the MRSC dataset.</p><p>6 0.52360892 <a title="184-lda-6" href="./cvpr-2013-Learning_Collections_of_Part_Models_for_Object_Recognition.html">248 cvpr-2013-Learning Collections of Part Models for Object Recognition</a></p>
<p>7 0.52256161 <a title="184-lda-7" href="./cvpr-2013-Integrating_Grammar_and_Segmentation_for_Human_Pose_Estimation.html">225 cvpr-2013-Integrating Grammar and Segmentation for Human Pose Estimation</a></p>
<p>8 0.52245933 <a title="184-lda-8" href="./cvpr-2013-Understanding_Indoor_Scenes_Using_3D_Geometric_Phrases.html">446 cvpr-2013-Understanding Indoor Scenes Using 3D Geometric Phrases</a></p>
<p>9 0.52244884 <a title="184-lda-9" href="./cvpr-2013-Understanding_Bayesian_Rooms_Using_Composite_3D_Object_Models.html">445 cvpr-2013-Understanding Bayesian Rooms Using Composite 3D Object Models</a></p>
<p>10 0.52244252 <a title="184-lda-10" href="./cvpr-2013-Beyond_Point_Clouds%3A_Scene_Understanding_by_Reasoning_Geometry_and_Physics.html">61 cvpr-2013-Beyond Point Clouds: Scene Understanding by Reasoning Geometry and Physics</a></p>
<p>11 0.52183402 <a title="184-lda-11" href="./cvpr-2013-Minimum_Uncertainty_Gap_for_Robust_Visual_Tracking.html">285 cvpr-2013-Minimum Uncertainty Gap for Robust Visual Tracking</a></p>
<p>12 0.5217818 <a title="184-lda-12" href="./cvpr-2013-Structure_Preserving_Object_Tracking.html">414 cvpr-2013-Structure Preserving Object Tracking</a></p>
<p>13 0.52136302 <a title="184-lda-13" href="./cvpr-2013-Accurate_Localization_of_3D_Objects_from_RGB-D_Data_Using_Segmentation_Hypotheses.html">30 cvpr-2013-Accurate Localization of 3D Objects from RGB-D Data Using Segmentation Hypotheses</a></p>
<p>14 0.521025 <a title="184-lda-14" href="./cvpr-2013-Part_Discovery_from_Partial_Correspondence.html">325 cvpr-2013-Part Discovery from Partial Correspondence</a></p>
<p>15 0.52077538 <a title="184-lda-15" href="./cvpr-2013-Robust_Estimation_of_Nonrigid_Transformation_for_Point_Set_Registration.html">360 cvpr-2013-Robust Estimation of Nonrigid Transformation for Point Set Registration</a></p>
<p>16 0.52044362 <a title="184-lda-16" href="./cvpr-2013-Bottom-Up_Segmentation_for_Top-Down_Detection.html">70 cvpr-2013-Bottom-Up Segmentation for Top-Down Detection</a></p>
<p>17 0.52039808 <a title="184-lda-17" href="./cvpr-2013-Robust_Real-Time_Tracking_of_Multiple_Objects_by_Volumetric_Mass_Densities.html">365 cvpr-2013-Robust Real-Time Tracking of Multiple Objects by Volumetric Mass Densities</a></p>
<p>18 0.5202949 <a title="184-lda-18" href="./cvpr-2013-Deep_Convolutional_Network_Cascade_for_Facial_Point_Detection.html">104 cvpr-2013-Deep Convolutional Network Cascade for Facial Point Detection</a></p>
<p>19 0.52017385 <a title="184-lda-19" href="./cvpr-2013-Least_Soft-Threshold_Squares_Tracking.html">267 cvpr-2013-Least Soft-Threshold Squares Tracking</a></p>
<p>20 0.51987755 <a title="184-lda-20" href="./cvpr-2013-Intrinsic_Scene_Properties_from_a_Single_RGB-D_Image.html">227 cvpr-2013-Intrinsic Scene Properties from a Single RGB-D Image</a></p>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
