<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>99 nips-2002-Graph-Driven Feature Extraction From Microarray Data Using Diffusion Kernels and Kernel CCA</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2002" href="../home/nips2002_home.html">nips2002</a> <a title="nips-2002-99" href="../nips2002/nips-2002-Graph-Driven_Feature_Extraction_From_Microarray_Data_Using_Diffusion_Kernels_and_Kernel_CCA.html">nips2002-99</a> <a title="nips-2002-99-reference" href="#">nips2002-99-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>99 nips-2002-Graph-Driven Feature Extraction From Microarray Data Using Diffusion Kernels and Kernel CCA</h1>
<br/><p>Source: <a title="nips-2002-99-pdf" href="http://papers.nips.cc/paper/2273-graph-driven-feature-extraction-from-microarray-data-using-diffusion-kernels-and-kernel-cca.pdf">pdf</a></p><p>Author: Jean-philippe Vert, Minoru Kanehisa</p><p>Abstract: We present an algorithm to extract features from high-dimensional gene expression proﬁles, based on the knowledge of a graph which links together genes known to participate to successive reactions in metabolic pathways. Motivated by the intuition that biologically relevant features are likely to exhibit smoothness with respect to the graph topology, the algorithm involves encoding the graph and the set of expression proﬁles into kernel functions, and performing a generalized form of canonical correlation analysis in the corresponding reproducible kernel Hilbert spaces. Function prediction experiments for the genes of the yeast S. Cerevisiae validate this approach by showing a consistent increase in performance when a state-of-the-art classiﬁer uses the vector of features instead of the original expression proﬁle to predict the functional class of a gene.</p><br/>
<h2>reference text</h2><p>[1] F. R. Bach and M. I. Jordan. Kernel independent component analysis. Journal of Machine Learning Research, 3:1–48, 2002.</p>
<p>[2] Michael P. S. Brown, William Noble Grundy, David Lin, Nello Cristianini, Charles Walsh Sugnet, Terence S. Furey, Jr. Manuel Ares, and David Haussler. Knowledge-based analysis of microarray gene expression data by using support vector machines. Proc. Natl. Acad. Sci. USA, 97:262–267, 2000.</p>
<p>[3] Fan R.K. Chung. Spectral graph theory, volume 92 of CBMS Regional Conference Series. American Mathematical Society, Providence, 1997.</p>
<p>[4] Michael B. Eisen, Paul T. Spellman, Patrick O. Brown, and David Botstein. Cluster analysis and display of genome-wide expression patterns. Proc. Natl. Acad. Sci. USA, 95:14863–14868, Dec 1998.</p>
<p>[5] Frederico Girosi, Michael Jones, and Tomaso Poggio. Regularization theory and neural networks architectures. Neural Computation, 7(2):219–269, 1995.</p>
<p>[6] S. Goto, Y. Okuno, M. Hattori, T. Nishioka, and M. Kanehisa. LIGAND: database of chemical compounds and reactions in biological pathways. Nucleic Acid Research, 30:402–404, 2002.</p>
<p>[7] Thorsten Joachims. Making large-scale svm learning practical. In B. Sch¨ lkopf, C. Burges, o and A. Smola, editors, Advances in Kernel Methods - Support Vector Learning, pages 169–184. MIT Press, 1999.</p>
<p>[8] M. Kanehisa, S. Goto, S. Kawashima, and A. Nakaya. The KEGG databases at GenomeNet. Nucleic Acid Research, 30:42–46, 2002.</p>
<p>[9] R. I. Kondor and J. Lafferty. Diffusion kernels on graphs and other discrete input. In ICML 2002, 2002.</p>
<p>[10] H.W. Mewes, D. Frishman, U. G¨ ldener, G. Mannhaupt, K. Mayer, M. Mokrejs, B. Morgenu stern, M. M¨ nsterkoetter, S. Rudd, and B. Weil. MIPS: a database for genomes and protein u sequences. Nucleic Acid Research, 30(1):31–34, 2002.</p>
<p>[11] B. Mohar. Some applications of laplace eigenvalues of graphs. In G. Hahn and G. Sabidussi, editors, Graph Symmetry: Algebraic Methods and Applications, volume 497 of NATO ASI Series C, pages 227–275. Kluwer, Dordrecht, 1997.</p>
<p>[12] S. Saitoh. Theory of reproducing Kernels and its applications. Longman Scientiﬁc & Technical, Harlow, UK, 1988.</p>
<p>[13] G. Sherlock, T. Hernandez-Boussard, A. Kasarskis, G. Binkley, J.C. Matese, S.S. Dwight, M. Kaloper, S. Weng, H. Jin, C.A. Ball, M.B. Eisen, and P.T. Spellman. The stanford microarray database. Nucleic Acid Research, 29(1):152–155, Jan 2001.</p>
<p>[14] G. Wahba. Spline Models for Observational Data, volume 59 of CBMS-NSF Regional Conference Series in Applied Mathematics. SIAM, Philadelphia, 1990.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
