<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>7 nips-2002-A Hierarchical Bayesian Markovian Model for Motifs in Biopolymer Sequences</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2002" href="../home/nips2002_home.html">nips2002</a> <a title="nips-2002-7" href="../nips2002/nips-2002-A_Hierarchical_Bayesian_Markovian_Model_for_Motifs_in_Biopolymer_Sequences.html">nips2002-7</a> <a title="nips-2002-7-reference" href="#">nips2002-7-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>7 nips-2002-A Hierarchical Bayesian Markovian Model for Motifs in Biopolymer Sequences</h1>
<br/><p>Source: <a title="nips-2002-7-pdf" href="http://papers.nips.cc/paper/2282-a-hierarchical-bayesian-markovian-model-for-motifs-in-biopolymer-sequences.pdf">pdf</a></p><p>Author: Eric P. Xing, Michael I. Jordan, Richard M. Karp, Stuart Russell</p><p>Abstract: We propose a dynamic Bayesian model for motifs in biopolymer sequences which captures rich biological prior knowledge and positional dependencies in motif structure in a principled way. Our model posits that the position-speciﬁc multinomial parameters for monomer distribution are distributed as a latent Dirichlet-mixture random variable, and the position-speciﬁc Dirichlet component is determined by a hidden Markov process. Model parameters can be ﬁt on training motifs using a variational EM algorithm within an empirical Bayesian framework. Variational inference is also used for detecting hidden motifs. Our model improves over previous models that ignore biological priors and positional dependence. It has much higher sensitivity to motifs during detection and a notable ability to distinguish genuine motifs from false recurring patterns.</p><br/>
<h2>reference text</h2><p>[1] T. L. Bailey and C. Elkan. Unsupervised learning of multiple motifs in biopolymers using EM. Machine Learning, 21:51–80, 1995.</p>
<p>[2] T. L. Bailey and C. Elkan. The value of prior knowledge in discovering motifs with MEME. In Proc. of the 3rd International Conf. on Intelligent Systems for Molecular Biology, 1995.</p>
<p>[3] M. J. Beal, Z. Ghahramani, and C. E. Rasmussen. The inﬁnite hidden Markov model. In Proc. of 14th Conference on Advances in Neural Information Processing Systems, 2001.</p>
<p>[4] M. Eisen. Structural properties of transcription factor-DNA interactions and the inference of sequence speciﬁcity. manuscript in preparation.</p>
<p>[5] Z. Ghahramani and M.J. Beal. Propagation algorithms for variational Bayesian learning. In Proc. of 13th Conference on Advances in Neural Information Processing Systems, 2000.</p>
<p>[6] D. Heckerman, D. Geiger, and D. M. Chickering. Learning Bayesian networks: the combination of knowledge and statistics data. Machine Learning, 20:197–243, 1995.</p>
<p>[7] C. Lawrence and A. Reilly. An expectation maximization (EM) algorithm for the identiﬁcation and characterization of common sites in unaligned biopolymer sequences. Proteins, 7:41–51, 1990.</p>
<p>[8] C.E. Lawrence, S.F. Altschul, M.S. Boguski, J.S. Liu, A.F. Neuwald, and J.C. Wootton. Detecting subtle sequence signals: A Gibbs sampling strategy for multiple alignment. Science, 262:208–214, 1993.</p>
<p>[9] J. Liu, X. Liu, and D.L. Brutlag. Bioprospector: Discovering conserved DNA motifs in upstream regulatory regions of co-expressed genes. In Proc. of PSB, 2001.</p>
<p>[10] J.S. Liu, A.F. Neuwald, and C.E. Lawrence. Bayesian models for multiple local sequence alignment and Gibbs sampling strategies. J. Amer. Statistical Assoc, 90:1156–1169, 1995.</p>
<p>[11] A. M. Michelson. Deciphering genetic regulatory codes: A challenge for functional genomics. Proc. Natl. Acad. Sci. USA, 99:546–548, 2002.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
