<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>48 acl-2013-An Open Source Toolkit for Quantitative Historical Linguistics</title>
</head>

<body>
<p><a title="acl" href="../acl_home.html">acl</a> <a title="acl-2013" href="../home/acl2013_home.html">acl2013</a> <a title="acl-2013-48" href="../acl2013/acl-2013-An_Open_Source_Toolkit_for_Quantitative_Historical_Linguistics.html">acl2013-48</a> <a title="acl-2013-48-reference" href="#">acl2013-48-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>48 acl-2013-An Open Source Toolkit for Quantitative Historical Linguistics</h1>
<br/><p>Source: <a title="acl-2013-48-pdf" href="http://aclweb.org/anthology//P/P13/P13-4003.pdf">pdf</a></p><p>Author: Johann-Mattis List ; Steven Moran</p><p>Abstract: Given the increasing interest and development of computational and quantitative methods in historical linguistics, it is important that scholars have a basis for documenting, testing, evaluating, and sharing complex workflows. We present a novel open-source toolkit for quantitative tasks in historical linguistics that offers these features. This toolkit also serves as an interface between existing software packages and frequently used data formats, and it provides implementations of new and existing algorithms within a homogeneous framework. We illustrate the toolkit’s functionality with an exemplary workflow that starts with raw language data and ends with automatically calculated phonetic alignments, cognates and borrowings. We then illustrate evaluation metrics on gold standard datasets that are provided with the toolkit.</p><br/>
<h2>reference text</h2><p>A. Bouchard-Côté, D. Hall, T. L. Griffiths, and D. Klein. 2013. Automated reconstruction of ancient languages using probabilistic models of sound change. PNAS, 110(1 1):4224–4229. R. Bouckaert, P. Lemey, M. Dunn, S. J. Greenhill, A. V. Alekseyenko, A. J. Drummond, R. D. Gray, M. A. Suchard, and Q. D. Atkinson. 2012. Mapping the origins and expansion of the Indo-European language family. Science, 337(6097):957–960, Aug. A. B. Dolgopolsky. 1964. Gipoteza drevnejšego rodstvajazykovych semej Severnoj Evrazii s verojatnostej točky zrenija [A probabilistic hypothesis concerning the oldest relationships among the language families of Northern Eurasia]. Voprosy Jazykoznanija, 2:53–63. J. Felsenstein. 2005. Phylip (phylogeny inference package) version 3.6. Distributed by the author. De-  partment of Genome Sciences, University of Washington, Seattle. J. Heath, S Moran, K. Prokhorov, L. McPherson, and B. Canslter. 2013. Dogon comparative lexicon. URL: http : / /www .dogonlanguage s . org. E. W. Holman, C. H. Brown, S. Wichmann, A. Müller, V. Velupillai, H. Hammarström, S. Sauppe, H. Jung, D. Bakker, P. Brown, O. Belyaev, M. Urban, R. Mailhammer, J.-M. List, and D. Egorov. 2011. Automated dating of the world’s language families based on lexical similarity. Current Anthropology, 52(6):841–875. D. H. Huson. 1998. SplitsTree. Analyzing and visualizing evolutionary data. Bioinformatics, 14(1):68– 73. B. Kessler. 2001 . The significance of word lists. Statistical tests for investigating historical connections between languages. CSLI Publications, Stanford. P. Kleiweg. 2009. RuG/L04. Software for dialectometrics and cartography. Distributed by the Author. Rijksuniversiteit Groningen. Faculteit der Letteren, September. R. Knight, P. Maxwell, A. Birmingham, J. Carnes, J. G. Caporaso, B. Easton, M. Eaton, M. Hamady,  H. Lindsay, Z. Liu, C. Lozupone, D. McDonald, M. Robeson, R. Sammut, S. Smit, M. Wakefield, J. Widmann, S. Wikman, S. Wilson, H. Ying, and G. Huttley. 2007. PyCogent. A toolkit for making sense from sequence. Genome Biology, 8(8):R171 . G. Kondrak. 2000. A new algorithm for the alignment of phonetic sequences. In Proceedings of the 1st North American chapter of the Association for Computational Linguistics conference, NAACL 2000, pages 288–295, Stroudsburg, PA, USA. Association for Computational Linguistics. V. I. Levenshtein. 1966. Binary codes capable of correcting deletions, insertions, and reversals. Soviet Physics Doklady, 10(8):707–710. J.-M. List. 2012a. LexStat. Automatic detection of cognates in multilingual wordlists. InProceedings of the EACL 2012 Joint Workshop of LINGVIS & UNCLH, pages 117–125. Association for Computational Linguistics. J.-M. List. 2012b. SCA. Phonetic alignment based on sound classes. In M. Slavkovik and D. Lassiter, editors, New directions in logic, language, and computation, number 7415 in LNCS, pages 32–5 1. Springer, Berlin and Heidelberg. S. Moran. 2012. Phonetics information base and lexicon. Ph.D. thesis, University of Washington.  S. B. Needleman and C. D. Wunsch. 1970. A gene method applicable to the search for similarities in the amino acid sequence of two proteins. Journal of Molecular Biology, 48:443–453, July. S. Nelson-Sathi, J.-M. List, H. Geisler, H. Fangerau, R. D. Gray, W. Martin, and T. Dagan. 2011. Networks uncover hidden lexical borrowing in IndoEuropean language evolution. Proceedings of the Royal Society B, 278(1713): 1794–1803. J. Prokić, M. Wieling, and J. Nerbonne. 2009. Multiple sequence alignments in linguistics. In Proceedings of the EACL 2009 Workshop on Language Technology and Resources for Cultural Heritage, Social Sciences, Humanities, and Education, pages 18–25. Association for Computational Linguistics. F. Ronquist and J. P. Huelsenbeck. 2003. MrBayes 3. Bayesian phylogenetic inference under mixed models. Bioinformatics, 19(12): 1572–1574. T. F. Smith and M. S. Waterman. 198 1. Identification of common molecular subsequences. Journal of Molecular Biology, 1:195–197. S. A. Starostin. 2000. The STARLING database program. URL: http : / / st arl ing .rinet . ru. L. Steiner, P. F. Stadler, and M. Cysouw.  2011.  A pipeline for computational historical linguistics. Language Dynamics and Change, 1(1):89–127. R. L. Trask. 2000. The dictionary of historical and comparative linguistics. Edinburgh University Press, Edinburgh. P. Turchin, I. Peiros, and M. Gell-Mann. 2010. Analyzing genetic connections between languages by matching consonant classes. Journal of Language Relationship, 3: 117–126. 18</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
