<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>131 nips-2011-Inference in continuous-time change-point models</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2011" href="../home/nips2011_home.html">nips2011</a> <a title="nips-2011-131" href="../nips2011/nips-2011-Inference_in_continuous-time_change-point_models.html">nips2011-131</a> <a title="nips-2011-131-reference" href="#">nips2011-131-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>131 nips-2011-Inference in continuous-time change-point models</h1>
<br/><p>Source: <a title="nips-2011-131-pdf" href="http://papers.nips.cc/paper/4286-inference-in-continuous-time-change-point-models.pdf">pdf</a></p><p>Author: Florian Stimberg, Manfred Opper, Guido Sanguinetti, Andreas Ruttor</p><p>Abstract: We consider the problem of Bayesian inference for continuous-time multi-stable stochastic systems which can change both their diffusion and drift parameters at discrete times. We propose exact inference and sampling methodologies for two speciﬁc cases where the discontinuous dynamics is given by a Poisson process and a two-state Markovian switch. We test the methodology on simulated data, and apply it to two real data sets in ﬁnance and systems biology. Our experimental results show that the approach leads to valid inferences and non-trivial insights. 1</p><br/>
<h2>reference text</h2><p>[1] Neil D. Lawrence, Guido Sanguinetti, and Magnus Rattray. Modelling transcriptional regulation using Gaussian processes. In B. Sch¨ lkopf, J. Platt, and T. Hoffman, editors, Advances in Neural Information o Processing Systems 19. 2007.</p>
<p>[2] Darren J. Wilkinson. Stochastic Modelling for Systems Biology. Chapman & Hall / CRC, London, 2006.</p>
<p>[3] Guido Sanguinetti, Andreas Ruttor, Manfred Opper, and C` dric Archambeau. Switching regulatory mode els of cellular stress response. Bioinformatics, 25:1280–1286, 2009.</p>
<p>[4] Ido Cohn, Tal El-Hay, Nir Friedman, and Raz Kupferman. Mean ﬁeld variational approximation for continuous-time Bayesian networks. In Proceedings of the twenty-ﬁfthth conference on Uncertainty in Artiﬁcial Intelligence (UAI), 2009.</p>
<p>[5] Andreas Ruttor and Manfred Opper. Approximate inference in reaction-diffusion processes. JMLR W&CP;, 9:669–676, 2010.</p>
<p>[6] Tobias Preis, Johannes Schneider, and H. Eugene Stanley. Switching processes in ﬁnancial markets. Proceedings of the National Academy of Sciences USA, 108(19):7674–7678, 2011.</p>
<p>[7] Paul Fearnhead and Zhen Liu. Efﬁcient bayesian analysis of multiple changepoint models with dependence across segments. Statistics and Computing, 21(2):217–229, 2011.</p>
<p>[8] Paolo Giordani and Robert Kohn. Efﬁcient bayesian inference for multiple change-point and mixture innovation models. Journal of Business and Economic Statistics, 26(1):66–77, 2008.</p>
<p>[9] E. B. Fox, E. B. Sudderth, M. I. Jordan, and A. S. Willsky. An HDP-HMM for systems with state persistence. In Proc. International Conference on Machine Learning, July 2008.</p>
<p>[10] Yunus Saatci, Ryan Turner, and Carl Edward Rasmussen. Gaussian process change point models. In ICML, pages 927–934, 2010.</p>
<p>[11] Vahid Shahrezaei and Peter Swain. The stochastic nature of biochemical networks. Curr. Opin. in Biotech., 19(4):369–374, 2008.</p>
<p>[12] Michael B. Elowitz, Arnold J. Levine, Eric D. Siggia, and Peter S. Swain. Stochastic gene expression in a single cell. Science, 297(5584):1129–1131, 2002.</p>
<p>[13] Avigdor Eldar and Michael B. Elowitz. 467(7312):167–173, 2010.  Functional roles for noise in genetic circuits.  Nature,</p>
<p>[14] G¨ rol M. Su¨ l, Jordi Garcia-Ojalvo, Louisa M. Liberman, and Michael B. Elowitz. An excitable gene u e regulatory circuit induces transient cellular differentiation. Nature, 440(7083):545–550, 2006.</p>
<p>[15] Manfred Opper, Andreas Ruttor, and Guido Sanguinetti. Approximate inference in continuous time gaussian-jump processes. In J. Lafferty, C. K. I. Williams, R. Zemel, J. Shawe-Taylor, and A. Culotta, editors, Advances in Neural Information Processing Systems 23, pages 1822–1830. 2010.</p>
<p>[16] N. G. van Kampen. Stochastic Processes in Physics and Chemistry. North-Holland, Amsterdam, 1981.</p>
<p>[17] Manfred Opper and Guido Sanguinetti. Learning combinatorial transcriptional dynamics from gene expression data. Bioinformatics, 26(13):1623–1629, 2010.</p>
<p>[18] H. M. Shahzad Asif and Guido Sanguinetti. Large scale learning of combinatorial transcriptional dynamics from gene expression. Bioinformatics, 27(9):1277–1283, 2011.</p>
<p>[19] Matthew Beal, Zoubin Ghahramani, and Carl Edward Rasmussen. The inﬁnite hidden Markov model. In S. Becker, S. Thrun, and L. Saul, editors, Advances in Neural Information Processing Systems 14, pages 577–584. 2002.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
