<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>120 nips-2011-History distribution matching method for predicting effectiveness of HIV combination therapies</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2011" href="../home/nips2011_home.html">nips2011</a> <a title="nips-2011-120" href="../nips2011/nips-2011-History_distribution_matching_method_for_predicting_effectiveness_of_HIV_combination_therapies.html">nips2011-120</a> <a title="nips-2011-120-reference" href="#">nips2011-120-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>120 nips-2011-History distribution matching method for predicting effectiveness of HIV combination therapies</h1>
<br/><p>Source: <a title="nips-2011-120-pdf" href="http://papers.nips.cc/paper/4206-history-distribution-matching-method-for-predicting-effectiveness-of-hiv-combination-therapies.pdf">pdf</a></p><p>Author: Jasmina Bogojeska</p><p>Abstract: This paper presents an approach that predicts the effectiveness of HIV combination therapies by simultaneously addressing several problems affecting the available HIV clinical data sets: the different treatment backgrounds of the samples, the uneven representation of the levels of therapy experience, the missing treatment history information, the uneven therapy representation and the unbalanced therapy outcome representation. The computational validation on clinical data shows that, compared to the most commonly used approach that does not account for the issues mentioned above, our model has signiﬁcantly higher predictive power. This is especially true for samples stemming from patients with longer treatment history and samples associated with rare therapies. Furthermore, our approach is at least as powerful for the remaining samples. 1</p><br/>
<h2>reference text</h2><p>[1] A. Altmann, M. D¨ umer, N. Beerenwinkel, E. Peres, Y. Sch¨ lter, A. B¨ ch, S. Rhee, a u u A. S¨ nnerborg, WJ. Fessel, M. Shafer, WR. Zazzi, R. Kaiser, and T. Lengauer. Predicting o response to combination antiretroviral therapy: retrospective validation of geno2pheno-THEO on a large clinical database. Journal of Infectious Diseases, 199:999–1006, 2009.</p>
<p>[2] S. Bickel, J. Bogojeska, T. Lengauer, and T. Scheffer. Multi-task learning for HIV therapy screening. In Proceedings of the International Conference on Machine Learning, 2008.</p>
<p>[3] S. Bickel, M. Br¨ ckner, and T. Scheffer. Discriminative learning for differing training and test u distributions. In Proceedings of the International Conference on Machine Learning, 2007.</p>
<p>[4] J. Bogojeska, S. Bickel, A. Altmann, and T. Lengauer. Dealing with sparse data in predicting outcomes of HIV combination therapies. Bioinformatics, 26:2085–2092, 2010.</p>
<p>[5] J. Bogojeska, D. St¨ ckel, M. Zazzi, R. Kaiser, F. Incardona, M. Rosen-Zvi, and T. Lengauer. o History-alignment models for bias-aware prediction of virological response to HIV combination therapy. submitted, 2011.</p>
<p>[6] J. Hanley and B. McNeil. A method of comparing the areas under receiver operating characteristic curves derived from the same cases. Radiology, 148:839–843, 1983.</p>
<p>[7] T. Hastie, R. Tibshirani, and J. Friedman. The Elements of Statistical Learning. Springer, 2009.</p>
<p>[8] VA. Johnson, F. Brun-Vezinet, B. Clotet, HF. G¨ nthrad, DR. Kuritzkes, D. Pillay, JM. u Schapiro, and DD. Richman. Update of the drug resistance mutations in HIV-1: December 2008. Topics in HIV Medicine, 16:138–145, 2008.</p>
<p>[9] L. Kaufman and PJ. Rousseeuw. Finding Groups in Data. An introduction to cluster analysis. John Wiley and Sons, Inc., 1990.</p>
<p>[10] B. Larder, D. Wang, A. Revell, J. Montaner, R. Harrigan, F. De Wolf, J. Lange, S. Wegner, L. Ruiz, MJ. Prez-Elas, S. Emery, J. Gatell, A. DArminio Monforte, C. Torti, M. Zazzi, and C. Lane. The development of artiﬁcial neural networks to predict virological response to combination HIV therapy. Antiviral Therapy, 12:15–24, 2007.</p>
<p>[11] RH. Lathrop and MJ. Pazzani. Combinatorial optimization in rapidly mutating drug-resistant viruses. Journal of Combinatorial Optimization, 3:301–320, 1999.</p>
<p>[12] TF. Liu and Shafer RW. Web resources for HIV type 1 genotypic-resistance test interpretation. Clinical Infectious Diseases, 42, 2006.</p>
<p>[13] S. Needleman and C. Wunsch. A general method applicable to the search for similarities in the amino acid sequence of two proteins. Journal of Molecular Biology, 48(3):443–453, 1970.</p>
<p>[14] DA. Ouattara. Mathematical analysis of the HIV-1 infection: parameter estimation, therapies effectiveness and therapeutical failures. In Engineering in Medicine and Biology Society, 2005.</p>
<p>[15] M. Prosperi, A. Altmann, M. Rosen-Zvi, E. Aharoni, G. Borgulya, F. Bazso, A. S¨ nnerborg, o E. Sch¨ lter, D. Struck, G. Ulivi, A. Vandamme, J. Vercauteren, and M. Zazzi. Investigation of u expert rule bases, logistic regression, and non-linear machine learning techniques for predicting response to antiretroviral treatment. Antiviral Therapy, 14:433–442, 2009.</p>
<p>[16] M. Rosen-Zvi, A. Altmann, M. Prosperi, E. Aharoni, H. Neuvirth, A. S¨ nnerborg, E. Sch¨ lter, o u D. Struck, Y. Peres, F. Incardona, R. Kaiser, M. Zazzi, and T. Lengauer. Selecting anti-HIV therapies based on a variety of genomic and clinical factors. Proceedings of the ISMB, 2008.</p>
<p>[17] P. J. Rousseeuw. Silhouettes: a graphical aid to the interpretation and validation of cluster analysis. Journal of Computational and Applied Mathematics, 20:53–65, 1987.</p>
<p>[18] UNAIDS/WHO. Report on the global aids epidemic: 2010. 2010.</p>
<p>[19] D. Wang, BA. Larder, A. Revell, R. Harrigan, and J. Montaner. A neural network model using clinical cohort data accurately predicts virological response and identiﬁes regimens with increased probability of success in treatment failures. Antiviral Therapy, 8:U99–U99, 2003.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
