<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>9 nips-2011-A More Powerful Two-Sample Test in High Dimensions using Random Projection</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2011" href="../home/nips2011_home.html">nips2011</a> <a title="nips-2011-9" href="../nips2011/nips-2011-A_More_Powerful_Two-Sample_Test_in_High_Dimensions_using_Random_Projection.html">nips2011-9</a> <a title="nips-2011-9-reference" href="#">nips2011-9-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>9 nips-2011-A More Powerful Two-Sample Test in High Dimensions using Random Projection</h1>
<br/><p>Source: <a title="nips-2011-9-pdf" href="http://papers.nips.cc/paper/4260-a-more-powerful-two-sample-test-in-high-dimensions-using-random-projection.pdf">pdf</a></p><p>Author: Miles Lopes, Laurent Jacob, Martin J. Wainwright</p><p>Abstract: We consider the hypothesis testing problem of detecting a shift between the means of two multivariate normal distributions in the high-dimensional setting, allowing for the data dimension p to exceed the sample size n. Our contribution is a new test statistic for the two-sample test of means that integrates a random projection with the classical Hotelling T 2 statistic. Working within a high-dimensional framework that allows (p, n) → ∞, we ﬁrst derive an asymptotic power function for our test, and then provide sufﬁcient conditions for it to achieve greater power than other state-of-the-art tests. Using ROC curves generated from simulated data, we demonstrate superior performance against competing tests in the parameter regimes anticipated by our theoretical results. Lastly, we illustrate an advantage of our procedure with comparisons on a high-dimensional gene expression dataset involving the discrimination of different types of cancer. 1</p><br/>
<h2>reference text</h2><p>[1] E. L. Lehmann and J. P. Romano. Testing statistical hypotheses. Springer Texts in Statistics. Springer, New York, third edition, 2005.</p>
<p>[2] Y. Lu, P. Liu, P. Xiao, and H. Deng. Hotelling’s T2 multivariate proﬁling for detecting differential expression in microarrays. Bioinformatics, 21(14):3105–3113, Jul 2005.</p>
<p>[3] J. J. Goeman and P. B¨ hlmann. Analyzing gene expression data in terms of gene sets: methodological u issues. Bioinformatics, 23(8):980–987, Apr 2007.</p>
<p>[4] D. V. D. Ville, T. Blue, and M. Unser. Integrated wavelet processing and spatial statistical testing of fMRI data. Neuroimage, 23(4):1472–1485, 2004.</p>
<p>[5] U. Ruttimann et al. Statistical analysis of functional MRI data in the wavelet domain. IEEE Transactions on Medical Imaging, 17(2):142–154, 1998.</p>
<p>[6] Z. Bai and H. Saranadasa. Effect of high dimension: by an example of a two sample problem. Statistica Sinica, 6:311,329, 1996.</p>
<p>[7] M. S. Srivastava and M. Du. A test for the mean vector with fewer observations than the dimension. Journal of Multivariate Analysis, 99:386–402, 2008.</p>
<p>[8] M. S. Srivastava. A test for the mean with fewer observations than the dimension under non-normality. Journal of Multivariate Analysis, 100:518–532, 2009.</p>
<p>[9] S. X. Chen and Y. L. Qin. A two-sample test for high-dimensional data with applications to gene-set testing. Annals of Statistics, 38(2):808–835, Feb 2010.</p>
<p>[10] S. Cl´ mencon, M. Depecker, and Vayatis N. AUC optimization and the two-sample problem. In Advances e ¸ in Neural Information Processing Systems (NIPS 2009), 2009.</p>
<p>[11] A. Gretton, K. M. Borgwardt, M. Rasch, B. Sch¨ lkop, and A.J. Smola. A kernel method for the twoo sample-problem. In B. Sch¨ lkopf, J. Platt, and T. Hoffman, editors, Advances in Neural Information o Processing Systems 19, pages 513–520. MIT Press, Cambridge, MA, 2007.</p>
<p>[12] Z. Harchaoui, F. Bach, and E. Moulines. Testing for homogeneity with kernel Fisher discriminant analysis. In John C. Platt, Daphne Koller, Yoram Singer, and Sam T. Roweis, editors, NIPS. MIT Press, 2007.</p>
<p>[13] M. E. Lopes, L. J. Jacob, and M. J. Wainwright. A more powerful two-sample test in high dimensions using random projection. Technical Report arXiv: 1108.2401, 2011.</p>
<p>[14] S. S. Vempala. The Random Projection Method. DIMACS Series in Discrete Mathematics and Theoretical Computer Science. American Mathematical Society, 2004.</p>
<p>[15] L. Jacob, P. Neuvial, and S. Dudoit. Gains in power from structured two-sample tests of means on graphs. Technical Report arXiv: q-bio/1009.5173v1, 2010.</p>
<p>[16] J. A. Cuesta-Albertos, E. Del Barrio, R. Fraiman, and C. Matr´ n. The random projection method in a goodness of ﬁt for functional data. Computational Statistics & Data Analysis, 51(10):4814–4831, 2007.</p>
<p>[17] R. J. Muirhead. Aspects of Multivariate Statistical Theory. John Wiley & Sons, inc., 1982.</p>
<p>[18] A. W. van der Vaart. Asymptotic Statistics. Cambridge, 2007.</p>
<p>[19] A. P. Dempster. A high dimensional two sample signiﬁcance test. Annals of Mathematical Statistics, 29(4):995–1010, 1958.</p>
<p>[20] A. P. Dempster. A signiﬁcance test for the separation of two highly multivariate small samples. Biometrics, 16(1):41–50, 1960.</p>
<p>[21] R. W. Tothill et al. Novel molecular subtypes of serous and endometrioid ovarian cancer linked to clinical outcome. Clin Cancer Res, 14(16):5198–5208, Aug 2008.</p>
<p>[22] J. Moreaux et al. A high-risk signature for patients with multiple myeloma established from the molecular classiﬁcation of human myeloma cell lines. Haematologica, 96(4):574–582, Apr 2011.</p>
<p>[23] R. N. Jorissen et al. Metastasis-associated gene expression changes predict poor outcomes in patients with dukes stage b and c colorectal cancer. Clin Cancer Res, 15(24):7642–7651, Dec 2009.</p>
<p>[24] A. Subramanian et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression proﬁles. Proc. Natl. Acad. Sci. USA, 102(43):15545–15550, Oct 2005.</p>
<p>[25] T. Beissbarth and T. P. Speed. Gostat: ﬁnd statistically overrepresented gene ontologies within a group of genes. Bioinformatics, 20(9):1464–1465, Jun 2004.  9</p>
<br/>
<br/><br/><br/></body>
</html>
