<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>6 nips-2011-A Global Structural EM Algorithm for a Model of Cancer Progression</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2011" href="../home/nips2011_home.html">nips2011</a> <a title="nips-2011-6" href="../nips2011/nips-2011-A_Global_Structural_EM_Algorithm_for_a_Model_of_Cancer_Progression.html">nips2011-6</a> <a title="nips-2011-6-reference" href="#">nips2011-6-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>6 nips-2011-A Global Structural EM Algorithm for a Model of Cancer Progression</h1>
<br/><p>Source: <a title="nips-2011-6-pdf" href="http://papers.nips.cc/paper/4397-a-global-structural-em-algorithm-for-a-model-of-cancer-progression.pdf">pdf</a></p><p>Author: Ali Tofigh, Erik Sj̦lund, Mattias H̦glund, Jens Lagergren</p><p>Abstract: Cancer has complex patterns of progression that include converging as well as diverging progressional pathways. Vogelstein’s path model of colon cancer was a pioneering contribution to cancer research. Since then, several attempts have been made at obtaining mathematical models of cancer progression, devising learning algorithms, and applying these to cross-sectional data. Beerenwinkel et al. provided, what they coined, EM-like algorithms for Oncogenetic Trees (OTs) and mixtures of such. Given the small size of current and future data sets, it is important to minimize the number of parameters of a model. For this reason, we too focus on tree-based models and introduce Hidden-variable Oncogenetic Trees (HOTs). In contrast to OTs, HOTs allow for errors in the data and thereby provide more realistic modeling. We also design global structural EM algorithms for learning HOTs and mixtures of HOTs (HOT-mixtures). The algorithms are global in the sense that, during the M-step, they ﬁnd a structure that yields a global maximum of the expected complete log-likelihood rather than merely one that improves it. The algorithm for single HOTs performs very well on reasonable-sized data sets, while that for HOT-mixtures requires data sets of sizes obtainable only with tomorrow’s more cost-efﬁcient technologies. 1</p><br/>
<h2>reference text</h2><p>[1] C. Chow and C. Liu. Approximating discrete probability distributions with dependence trees. IEEE Trans Inform Theor, 14(3):462–467, 1968.</p>
<p>[2] M. Meila and M.I. Jordan. Learning with mixtures of trees. J Mach Learn Res, 1(1):1–48, 2000.</p>
<p>[3] N. Friedman, M. Ninio, I. Pe’er, and T. Pupko. A structural em algorithm for phylogenetic inference. J Comput Biol, 9(2):331–353, 2002.</p>
<p>[4] N. Friedman. The bayesian structural em algorithm. In Proceedings of the Conference on Uncertainty in Artiﬁcial Intelligence, pages 129–138. Morgan Kaufmann, 1998.</p>
<p>[5] P Leray and O Francois. Bayesian network structural learning and incomplete data. Proceedings of the ¸ International and Interdisciplinary Conference on Adaptive Knowledge Representation and Reasoning (AKRR 2005), pages 33–40, 2005.</p>
<p>[6] R. Desper, F. Jiang, O.P. Kallioniemi, H. Moch, C.H. Papadimitriou, and A.A. Schaffer. Inferring tree models for oncogenesis from comparative genome hybridization data. J Comput Biol, 6(1):37–51, 1999.</p>
<p>[7] M. Hjelm, M. H¨ glund, and J. Lagergren. New probabilistic network models and algorithms for oncogeo nesis. J Comput Biol, 13(4):853–865, May 2006.</p>
<p>[8] M.D. Radmacher, R. Simon, R. Desper, R. Taetle, A.A. Schaffer, and M.A. Nelson. Graph models of oncogenesis with an application to melanoma. J Theor Biol, 212(4):535–48, Oct 2001.</p>
<p>[9] N. Beerenwinkel, J. Rahnenfuhrer, M. Daumer, D. Hoffmann, R. Kaiser, J. Selbig, and T. Lengauer. Learning multiple evolutionary pathways from cross-sectional data. J Comput Biol, 12(6):584–598, Jul 2005.</p>
<p>[10] J. Rahnenfuhrer, N. Beerenwinkel, W.A. Schulz, C. Hartmann, A. von Deimling, B. Wullich, and T. Lengauer. Estimating cancer survival and clinical outcome based on genetic tumor progression scores. Bioinformatics, 21(10):2438–2446, May 2005.</p>
<p>[11] N. Beerenwinkel, J. Rahnenfuhrer, R. Kaiser, D. Hoffmann, J. Selbig, and T. Lengauer. Mtreemix: a software package for learning and using mixture models of mutagenetic trees. Bioinformatics, 21(9):2106– 2107, 2005.</p>
<p>[12] N. Beerenwinkel, N. Eriksson, and B. Sturmfels. Conjunctive bayesian networks. Bernoulli, 13(4):893– 909, Jan 2007.</p>
<p>[13] N. Beerenwinkel, N. Eriksson, and B. Sturmfels. Evolution on distributive lattices. J Theor Biol, 242(2):409–20, Sep 2006.</p>
<p>[14] M. Gerstung, M. Baudis, H. Moch, and N. Beerenwinkel. Quantifying cancer progression with conjunctive bayesian networks. Bioinformatics, 25(21):2809–15, Nov 2009.</p>
<p>[15] R.E. Tarjan. Finding optimum branchings. Networks, 7(1):25–36, 1977.</p>
<p>[16] R.M. Karp. A simple derivation of edmond’s algorithm for optimum branching. Networks, 1(265-272):5, 1971.</p>
<p>[17] P. Camerini, L. Fratta, and F. Mafﬁoli. The k best spanning arborescences of a network. Networks, 10(2):91–110, 1980.</p>
<p>[18] F. Mitelman, B. Johansson, and F. Mertens (Eds.). Mitelman database of chromosome aberrations and gene fusions in cancer, 2010. http://cgap.nci.nih.gov/Chromosomes/Mitelman.</p>
<p>[19] R. Durbin, S.R. Eddy, A. Krogh, and G. Mitchison. Biological Sequence Analysis. Cambridge University Press, Cambridge, 1998.</p>
<p>[20] A.P. Dempster, N.M. Laird, and D.B. Rubin. Maximum likelihood from incomplete data via the em algorithm. J Roy Stat Soc B, 39(1):1–38, 1977.</p>
<p>[21] M. H¨ glund, D. Gisselsson, G.B. Hansen, T. S¨ ll, F. Mitelman, and M. Nilbert. Dissecting karyotypic o a patterns in colorectal tumors: Two distinct but overlapping pathways in the adenoma-carcinoma transition. Canc Res, 62:5939–5946, 2002.</p>
<p>[22] M. H¨ glund, D. Gisselsson, M. Soller, G.B. Hansen, P. Elfving, and F. Mitelman. Dissecting karyotypic o patterns in renal cell carcinoma: an analysis of the accumulated cytogenetic data. Canc Genet Cytogenet, 153(1):1–9, 2004.  9</p>
<br/>
<br/><br/><br/></body>
</html>
