<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>299 nips-2012-Scalable imputation of genetic data with a discrete fragmentation-coagulation process</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2012" href="../home/nips2012_home.html">nips2012</a> <a title="nips-2012-299" href="../nips2012/nips-2012-Scalable_imputation_of_genetic_data_with_a_discrete_fragmentation-coagulation_process.html">nips2012-299</a> <a title="nips-2012-299-reference" href="#">nips2012-299-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>299 nips-2012-Scalable imputation of genetic data with a discrete fragmentation-coagulation process</h1>
<br/><p>Source: <a title="nips-2012-299-pdf" href="http://papers.nips.cc/paper/4782-scalable-imputation-of-genetic-data-with-a-discrete-fragmentation-coagulation-process.pdf">pdf</a></p><p>Author: Lloyd Elliott, Yee W. Teh</p><p>Abstract: We present a Bayesian nonparametric model for genetic sequence data in which a set of genetic sequences is modelled using a Markov model of partitions. The partitions at consecutive locations in the genome are related by the splitting and merging of their clusters. Our model can be thought of as a discrete analogue of the continuous fragmentation-coagulation process [Teh et al 2011], preserving the important properties of projectivity, exchangeability and reversibility, while being more scalable. We apply this model to the problem of genotype imputation, showing improved computational efﬁciency while maintaining accuracies comparable to other state-of-the-art genotype imputation methods. 1</p><br/>
<h2>reference text</h2><p>[1] The 1000 Genomes Project Consortium. A map of human genome variation from population-scale sequencing. Nature, 467:1061–1073, 2010.</p>
<p>[2] R. R. Hudson. Properties of a neutral allele model with intragenic recombination. Theoretical Population Biology, 23(2):183 – 201, 1983.</p>
<p>[3] P. Scheet and M. Stephens. A fast and ﬂexible statistical model for large-scale population genotype data: Applications to inferring missing genotypes and haplotypic phase. The American Journal of Human Genetics, 78(4):629 – 644, 2006.</p>
<p>[4] J. Marchini, B. Howie, S. Myers, G. McVean, and P. Donnelly. A new multipoint method for genome-wide association studies by imputation of genotypes. Nature Genetics, 39(7):906–913, 2007.</p>
<p>[5] M. J. Daly, J. D. Rioux, S. F. Schaffner, T. J. Hudson, and R. S. Lander. High-resolution haplotype structure in the human genome. Nature Genetics, 29:229–232, 2001.</p>
<p>[6] J. Marchini, D. Cutler, N. Patterson, M. Stephens, E. Eskin, E. Halperin, S. Lin, Z.S. Qin, H.M. Munro, G.R. Abecasis, P. Donnelly, and the International HapMap Consortium. A comparison of phasing algorithms for trios and unrelated individuals. The American Journal of Human Genetics, 78(3):437 – 450, 2006.</p>
<p>[7] M. Stephens. Dealing with label switching in mixture models. Journal of the Royal Statistical Society: Series B (Statistical Methodology), 62(4):795–809, 2000.</p>
<p>[8] A. Jasra, C. C. Holmes, and D. A. Stephens. Markov chain Monte Carlo methods and the label switching problem in Bayesian mixture modeling. Statistical Science, 20(1):50–67, 2005.</p>
<p>[9] Y. W. Teh, C. Blundell, and L. T. Elliott. Modelling genetic variations using fragmentation-coagulation processes. In Advances in neural information processing systems, 2011.</p>
<p>[10] The International HapMap Consortium. The international HapMap project. Nature, 426:789–796, 2003.</p>
<p>[11] J. Pitman. Combinatorial stochastic processes. Springer-Verlag, 2006.</p>
<p>[12] J. Pitman. Coalescents with multiple collisions. Annals of Probability, 27:1870–1902, 1999.</p>
<p>[13] W. J. Ewens. The sampling theory of selectively neutral alleles. Theoretical Population Biology, 3:87– 112, 1972.</p>
<p>[14] R. R. Hudson. Generating samples under a Wright-Fisher neutral model of genetic variation. Bioinfomatics, 18:337–338, 2002.</p>
<p>[15] R. M. Neal. Slice sampling. Annals of Statistics, 31:705–767, 2003.</p>
<p>[16] V. Rao and Y. W. Teh. Fast MCMC sampling for Markov jump processes and continuous time Bayesian networks. In Proceedings of the International Conference on Uncertainty in Artiﬁcial Intelligence, 2011.</p>
<p>[17] B. L. Browning and S. R. Browning. A uniﬁed approach to genotype imputation and haplotype-phase inference for large data sets of trios and unrelated individuals. American Journal of Human Genetics, 84:210–223, 2009.</p>
<p>[18] V. Rao and Y. W. Teh. Spatial normalized gamma processes. In Advances in Neural Information Processing Systems, volume 22, pages 1554–1562, 2009.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
