<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>118 nips-2012-Entangled Monte Carlo</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2012" href="../home/nips2012_home.html">nips2012</a> <a title="nips-2012-118" href="../nips2012/nips-2012-Entangled_Monte_Carlo.html">nips2012-118</a> <a title="nips-2012-118-reference" href="#">nips2012-118-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>118 nips-2012-Entangled Monte Carlo</h1>
<br/><p>Source: <a title="nips-2012-118-pdf" href="http://papers.nips.cc/paper/4648-entangled-monte-carlo.pdf">pdf</a></p><p>Author: Seong-hwan Jun, Liangliang Wang, Alexandre Bouchard-côté</p><p>Abstract: We propose a novel method for scalable parallelization of SMC algorithms, Entangled Monte Carlo simulation (EMC). EMC avoids the transmission of particles between nodes, and instead reconstructs them from the particle genealogy. In particular, we show that we can reduce the communication to the particle weights for each machine while efﬁciently maintaining implicit global coherence of the parallel simulation. We explain methods to efﬁciently maintain a genealogy of particles from which any particle can be reconstructed. We demonstrate using examples from Bayesian phylogenetic that the computational gain from parallelization using EMC signiﬁcantly outweighs the cost of particle reconstruction. The timing experiments show that reconstruction of particles is indeed much more efﬁcient as compared to transmission of particles. 1</p><br/>
<h2>reference text</h2><p>[1] D. P. Anderson. BOINC: A System for Public-Resource Computing and Storage. In GRID ’04: Proceedings of the 5th IEEE/ACM International Workshop on Grid Computing, pages 4–10, Washington, DC, USA, 2004. IEEE Computer Society.</p>
<p>[2] Y. W. Teh, H. Daum´ III, and D. M. Roy. Bayesian agglomerative clustering with coalescents. e In Advances in Neural Information Processing Systems (NIPS), 2008.</p>
<p>[3] J. G. Propp and D. B. Wilson. Coupling from the past: a user’s guide. Microsurveys in Discrete Probability. DIMACS Series in Discrete Mathematics and Theoretical Computer Science, 41:181–192, 1998.</p>
<p>[4] P. Del Moral, A. Doucet, and A. Jasra. Sequential Monte Carlo samplers. Journal of The Royal Statistical Society Series B-statistical Methodology, 68(3):411–436, 2006.</p>
<p>[5] P. Del Moral, A. Doucet, and A. Jasra. Sequential Monte Carlo for Bayesian computation. Bayesian Statistics, 8:1–34, 2007.</p>
<p>[6] A. Lee, C. Yau, M. B. Giles, A. Doucet, and C. C. Holmes. On the utility of graphics cards to perform massively parallel simulation of advanced Monte Carlo methods. Journal of Computational and Graphical Statistics, 19(4):769–789, 2010.</p>
<p>[7] S. Singh and A. McCallum. Towards asynchronous distributed MCMC inference for large graphical models. In Neural Information Processing Systems (NIPS), Big Learning Workshop on Algorithms, Systems, and Tools for Learning at Scale, 2011.</p>
<p>[8] J. Gonzalez, Y. Low, A. Gretton, and C. Guestrin. Parallel Gibbs sampling: From colored ﬁelds to thin junction trees. In In Artiﬁcial Intelligence and Statistics (AISTATS), Ft. Lauderdale, FL, May 2011.</p>
<p>[9] S. Singh, A. Subramanya, F. Pereira, and A. McCallum. Large-scale cross-document coreference using distributed inference and hierarchical models. In Association for Computational Linguistics: Human Language Technologies (ACL HLT), 2011.</p>
<p>[10] R. H. Swendsen and J-S. Wang. Replica Monte Carlo simulation of spin-glasses. Phys. Rev. Lett., 57:2607–2609, Nov 1986.</p>
<p>[11] A. Bouchard-Cˆ t´ , S. Sankararaman, and M. I. Jordan. Phylogenetic inference via Sequential oe Monte Carlo. Systematic Biology, 2011.</p>
<p>[12] A. Doucet, N. de Freitas, and N. Gordon. Sequential Monte Carlo methods in practice. Springer, 2001.</p>
<p>[13] I. Stoica, R. Morris, D. Karger, M. F. Kaashoek, and H. Balakrishnan. Chord: A scalable peer-to-peer lookup service for internet applications. ACM SIGCOMM 2001, pages 149–160, 2001.</p>
<p>[14] J. F. C. Kingman. On the Genealogy of Large Populations. Journal of Applied Probability, 19:27–43, 1982.</p>
<p>[15] J. Felsenstein. Inferring phylogenies. Sinauer Associates, 2003.</p>
<p>[16] C. Semple and M. Steel. Phylogenetics. Oxford, 2003.</p>
<p>[17] J. P. Huelsenbeck and F. Ronquist. MRBAYES: Bayesian inference of phylogenetic trees. Bioinformatics, 17(8):754–755, August 2001.</p>
<p>[18] J.J. Cannone, S. Subramanian, M.N. Schnare, J.R. Collett, L.M. D’Souza, Y. Du, B. Feng, N. Lin, L.V. Madabusi, K.M. Muller, N. Pande, Z. Shang, N. Yu, and R.R. Gutell. The comparative RNA web (CRW) site: An online database of comparative sequence and structure information for ribosomal, intron, and other RNAs. BioMed Central Bioinformatics, 2002.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
