<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>53 nips-2012-Bayesian Pedigree Analysis using Measure Factorization</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2012" href="../home/nips2012_home.html">nips2012</a> <a title="nips-2012-53" href="../nips2012/nips-2012-Bayesian_Pedigree_Analysis_using_Measure_Factorization.html">nips2012-53</a> <a title="nips-2012-53-reference" href="#">nips2012-53-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>53 nips-2012-Bayesian Pedigree Analysis using Measure Factorization</h1>
<br/><p>Source: <a title="nips-2012-53-pdf" href="http://papers.nips.cc/paper/4602-bayesian-pedigree-analysis-using-measure-factorization.pdf">pdf</a></p><p>Author: Bonnie Kirkpatrick, Alexandre Bouchard-côté</p><p>Abstract: Pedigrees, or family trees, are directed graphs used to identify sites of the genome that are correlated with the presence or absence of a disease. With the advent of genotyping and sequencing technologies, there has been an explosion in the amount of data available, both in the number of individuals and in the number of sites. Some pedigrees number in the thousands of individuals. Meanwhile, analysis methods have remained limited to pedigrees of < 100 individuals which limits analyses to many small independent pedigrees. Disease models, such those used for the linkage analysis log-odds (LOD) estimator, have similarly been limited. This is because linkage analysis was originally designed with a different task in mind, that of ordering the sites in the genome, before there were technologies that could reveal the order. LODs are difﬁcult to interpret and nontrivial to extend to consider interactions among sites. These developments and difﬁculties call for the creation of modern methods of pedigree analysis. Drawing from recent advances in graphical model inference and transducer theory, we introduce a simple yet powerful formalism for expressing genetic disease models. We show that these disease models can be turned into accurate and computationally efﬁcient estimators. The technique we use for constructing the variational approximation has potential applications to inference in other large-scale graphical models. This method allows inference on larger pedigrees than previously analyzed in the literature, which improves disease site prediction. 1</p><br/>
<h2>reference text</h2><p>[1] G. Mendel. Experiments in plant-hybridisation. In English Translation and Commentary by R. A. Fisher, J.H. Bennett, ed. Oliver and Boyd, Edinburgh 1965, 1866.</p>
<p>[2] A. H. Sturtevant. The linear arrangement of six sex-linked factors in drosophila, as shown by their mode of association. Journal of Experimental Zoology, 14:43–59, 1913.  8</p>
<p>[3] GR Abecasis, SS Cherny, WO Cookson, et al. Merlin-rapid analysis of dense genetic maps using sparse gene ﬂow trees. Nature Genetics, 30:97–101, 2002.</p>
<p>[4] M Silberstein, A. Tzemach, N. Dovgolevsky, M. Fishelson, A. Schuster, and D. Geiger. On-line system for faster linkage analysis via parallel execution on thousands of personal computers. Americal Journal of Human Genetics, 78(6):922–935, 2006.</p>
<p>[5] D. Geiger, C. Meek, and Y. Wexler. Speeding up HMM algorithms for genetic linkage analysis via chain reductions of the state space. Bioinformatics, 25(12):i196, 2009.</p>
<p>[6] C. A. Albers, M. A. R. Leisink, and H. J. Kappen. The cluster variation method for efﬁcient linkage analysis on extended pedigrees. BMC Bioinformatics, 7(S-1), 2006.</p>
<p>[7] M. L. Metzker. Sequencing technologies–the next generation. Nat Rev Genet, 11(1):31–46, January 2010.</p>
<p>[8] M. Abney, C. Ober, and M. S. McPeek. Quantitative-trait homozygosity and association mapping and empirical genome wide signiﬁcance in large, complex pedigrees: Fasting serum-insulin level in the hutterites. American Journal of Human Genetics, 70(4):920 – 934, 2002.</p>
<p>[9] N.B. Sutter and et al. A Single IGF1 Allele Is a Major Determinant of Small Size in Dogs. Science, 316(5821):112–115, 2007.</p>
<p>[10] M. Mohri. Handbook of Weighted Automata, chapter 6. Monographs in Theoretical Computer Science. Springer, 2009.</p>
<p>[11] A. Bouchard-Cˆ t´ and M. I. Jordan. Variational Inference over Combinatorial Spaces. In Advances in oe Neural Information Processing Systems 23 (NIPS), 2010.</p>
<p>[12] J. S. Yedidia, W. T. Freeman, and Y. Weiss. Bethe free energy, Kikuchi approximations and belief propagation algorithms. In Advances in Neural Information Processing Systems (NIPS), 2001.</p>
<p>[13] E. M. Wijsman. Penetrance. John Wiley & Sons, Ltd, 2005.</p>
<p>[14] R.C. Elston and J. Stewart. A general model for the analysis of pedigree data. Human Heredity, 21:523– 542, 1971.</p>
<p>[15] E.S. Lander and P. Green. Construction of multilocus genetic linkage maps in humans. Proceedings of the National Academy of Science, 84(5):2363–2367, 1987.</p>
<p>[16] J. Marchini, P. Donnelly, and L. R. Cardon. Genome-wide strategies for detecting multiple loci that inﬂuence complex diseases. Nat. Genet., 37(4):413–417, 2005.</p>
<p>[17] Y. W. Teh, C. Blundell, and L. T. Elliott. Modelling genetic variations with fragmentation-coagulation processes. In Advances In Neural Information Processing Systems, 2011.</p>
<p>[18] A. Piccolboni and D. Gusﬁeld. On the complexity of fundamental computational problems in pedigree analysis. Journal of Computational Biology, 10(5):763–773, 2003.</p>
<p>[19] S. L. Lauritzen and N. A. Sheehan. Graphical models for genetic analysis. Statistical Science, 18(4):489– 514, 2003.</p>
<p>[20] A. Thomas, A. Gutin, V. Abkevich, and A. Bansal. Multilocus linkage analysis by blocked Gibbs sampling. Statistics and Computing, 10(3):259–269, July 2000.</p>
<p>[21] G. O. Roberts and S. K. Sahu. Updating schemes, correlation structure, blocking and parameterization for the Gibbs sampler. Journal of the Royal Statistical Society: Series B (Statistical Methodology), 59(2):291– 317, 1997.</p>
<p>[22] A. Bouchard-Cˆ t´ and M.I. Jordan. Optimization of structured mean ﬁeld objectives. In Proceedings of oe the Twenty-Fifth Conference Annual Conference on Uncertainty in Artiﬁcial Intelligence (UAI-09), pages 67–74, Corvallis, Oregon, 2009. AUAI Press.</p>
<p>[23] T. Minka and Y. Qi. Tree-structured approximations by expectation. In Advances in Neural Information Processing Systems (NIPS), 2003.</p>
<p>[24] M. J. Wainwright, T. S. Jaakkola, and A. S. Willsky. Tree-reweighted belief propagation algorithms and approximate ML estimation by pseudo-moment matching. In AISTATS, 2003.</p>
<p>[25] R. H. Swendsen and J.-S. Wang. Nonuniversal critical dynamics in Monte Carlo simulations. Phys. Rev. Lett., 58:86–88, Jan 1987.</p>
<p>[26] J. Wakeley. Coalescent Theory: An Introduction. Roberts & Company Publishers, 1 edition, June 2008.</p>
<p>[27] B. Kirkpatrick, E. Halperin, and R. M. Karp. Haplotype inference in complex pedigrees. Journal of Computational Biology, 17(3):269–280, 2010.</p>
<p>[28] C. A. Albers, T. Heskes, and H. J. Kappen. Haplotype inference in general pedigrees using the cluster variation method. Genetics, 177(2):1101–1116, October 2007.</p>
<p>[29] The International HapMap Consortium. The international HapMap project. Nature, 426:789–796, 2003.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
