<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>327 nips-2012-Structured Learning of Gaussian Graphical Models</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2012" href="../home/nips2012_home.html">nips2012</a> <a title="nips-2012-327" href="../nips2012/nips-2012-Structured_Learning_of_Gaussian_Graphical_Models.html">nips2012-327</a> <a title="nips-2012-327-reference" href="#">nips2012-327-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>327 nips-2012-Structured Learning of Gaussian Graphical Models</h1>
<br/><p>Source: <a title="nips-2012-327-pdf" href="http://papers.nips.cc/paper/4499-structured-learning-of-gaussian-graphical-models.pdf">pdf</a></p><p>Author: Karthik Mohan, Mike Chung, Seungyeop Han, Daniela Witten, Su-in Lee, Maryam Fazel</p><p>Abstract: We consider estimation of multiple high-dimensional Gaussian graphical models corresponding to a single set of nodes under several distinct conditions. We assume that most aspects of the networks are shared, but that there are some structured differences between them. Speciﬁcally, the network differences are generated from node perturbations: a few nodes are perturbed across networks, and most or all edges stemming from such nodes differ between networks. This corresponds to a simple model for the mechanism underlying many cancers, in which the gene regulatory network is disrupted due to the aberrant activity of a few speciﬁc genes. We propose to solve this problem using the perturbed-node joint graphical lasso, a convex optimization problem that is based upon the use of a row-column overlap norm penalty. We then solve the convex problem using an alternating directions method of multipliers algorithm. Our proposal is illustrated on synthetic data and on an application to brain cancer gene expression data. 1</p><br/>
<h2>reference text</h2><p>[1] K.V. Mardia, J. Kent, and J.M. Bibby. Multivariate Analysis. Academic Press, 1979.</p>
<p>[2] S.L. Lauritzen. Graphical Models. Oxford Science Publications, 1996.</p>
<p>[3] M. Yuan and Y. Lin. Model selection and estimation in the Gaussian graphical model. Biometrika, 94(10):19–35, 2007.</p>
<p>[4] J. Friedman, T. Hastie, and R. Tibshirani. Sparse inverse covariance estimation with the graphical lasso. Biostatistics, 9:432–441, 2007.</p>
<p>[5] O. Banerjee, L. E. El Ghaoui, and A. d’Aspremont. Model selection through sparse maximum likelihood estimation for multivariate Gaussian or binary data. JMLR, 9:485–516, 2008.</p>
<p>[6] D.M. Witten, J.H. Friedman, and N. Simon. New insights and faster computations for the graphical lasso. Journal of Computational and Graphical Statistics, 20(4):892–900, 2011.</p>
<p>[7] K. Scheinberg, S. Ma, and D. Goldfarb. Sparse inverse covariance selection via alternating linearization methods. Advances in Neural Information Processing Systems, 2010.</p>
<p>[8] P. Ravikumar, M.J. Wainwright, G. Raskutti, and B. Yu. Model selection in gaussian graphical models: high-dimensional consistency of l1-regularized MLE. Advances in NIPS, 2008.</p>
<p>[9] C.J. Hsieh, M. Sustik, I. Dhillon, and P. Ravikumar. Sparse inverse covariance estimation using quadratic approximation. Advances in Neural Information Processing Systems, 2011.</p>
<p>[10] R. Tibshirani. Regression shrinkage and selection via the lasso. Journal of the Royal Statistical Society, Series B, 58:267–288, 1996.</p>
<p>[11] A. D’Aspremont, O. Banerjee, and L. El Ghaoui. First-order methods for sparse covariance selection. SIAM Journal on Matrix Analysis and Applications, 30(1):56–66, 2008.</p>
<p>[12] J. Guo, E. Levina, G. Michailidis, and J. Zhu. Joint estimation of multiple graphical models. Biometrika, 98(1):1–15, 2011.</p>
<p>[13] P. Danaher, P. Wang, and D. Witten. The joint graphical lasso for inverse covariance estimation across multiple classes, 2012. http://arxiv.org/abs/1111.0324.</p>
<p>[14] R. Tibshirani, M. Saunders, S. Rosset, J. Zhu, and K. Knight. Sparsity and smoothness via the fused lasso. Journal of the Royal Statistical Society, Series B, 67:91–108, 2005.</p>
<p>[15] M. Yuan and Y. Lin. Model selection and estimation in regression with grouped variables. Journal of the Royal Statistical Society, Series B, 68:49–67, 2007.</p>
<p>[16] L. Jacob, G. Obozinski, and J.P. Vert. Group lasso with overlap and graph lasso. Proceedings of the 26th International Conference on Machine Learning, 2009.</p>
<p>[17] G. Obozinski, L. Jacob, and J.P. Vert. Group lasso with overlaps: the latent group lasso approach. 2011. http://arxiv.org/abs/1110.0413.</p>
<p>[18] M. Grant and S. Boyd. cvx version 1.21. ”http://cvxr.com/cvx”, October 2010.</p>
<p>[19] A. Argyriou, C.A. Micchelli, and M. Pontil. Efﬁcient ﬁrst order methods for linear composite regularizers. 2011. http://arxiv.org/pdf/1104.1436.</p>
<p>[20] X. Chen, Q. Lin, S. Kim, J.G. Carbonell, and E.P. Xing. Smoothing proximal gradient method for general structured sparse learning. Proceedings of the conference on Uncertainty in Artiﬁcial Intelligence, 2011.</p>
<p>[21] S. Mosci, S. Villa, A. Verri, and L. Rosasco. A primal-dual algorithm for group sparse regularization with overlapping groups. Neural Information Processing Systems, pages 2604 – 2612, 2010.</p>
<p>[22] S.P. Boyd, N. Parikh, E. Chu, B. Peleato, and J. Eckstein. Distributed optimization and statistical learning via the alternating direction method of multipliers. Foundations and Trends in ML, 3(1):1–122, 2010.</p>
<p>[23] M. Hong and Z. Luo. On the linear convergence of the alternating direction method of multipliers. 2012. Available at arxiv.org/abs/1208.3922.</p>
<p>[24] B. He, M. Tao, and X. Yuan. Alternating direction method with gaussian back substitution for separable convex programming. SIAM Journal of Optimization, pages 313 – 340, 2012.</p>
<p>[25] J. Duchi and Y. Singer. Efﬁcient online and batch learning using forward backward splitting. Journal of Machine Learning Research, pages 2899 – 2934, 2009.</p>
<p>[26] Verhaak et al. Integrated genomic analysis identiﬁes clinically relevant subtypes of glioblastoma characterized by abnormalities in PDGFRA, IDH1, EGFR, and NF1. Cancer Cell, 17(1):98–110, 2010.</p>
<p>[27] Grosso et al. Chemokine CXCL13 is overexpressed in the tumour tissue and in the peripheral blood of breast cancer patients. British Journal Cancer, 99(6):930–938, 2008.</p>
<p>[28] El-Haibi et al. CXCL13-CXCR5 interactions support prostate cancer cell migration and invasion in a PI3K p110-, SRC- and FAK-dependent fashion. The Journal of Immunology, 15(19):5968–73, 2009.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
