<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>108 nips-2006-Large Scale Hidden Semi-Markov SVMs</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2006" href="../home/nips2006_home.html">nips2006</a> <a title="nips-2006-108" href="../nips2006/nips-2006-Large_Scale_Hidden_Semi-Markov_SVMs.html">nips2006-108</a> <a title="nips-2006-108-reference" href="#">nips2006-108-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>108 nips-2006-Large Scale Hidden Semi-Markov SVMs</h1>
<br/><p>Source: <a title="nips-2006-108-pdf" href="http://papers.nips.cc/paper/2988-large-scale-hidden-semi-markov-svms.pdf">pdf</a></p><p>Author: Gunnar Rätsch, Sören Sonnenburg</p><p>Abstract: We describe Hidden Semi-Markov Support Vector Machines (SHM SVMs), an extension of HM SVMs to semi-Markov chains. This allows us to predict segmentations of sequences based on segment-based features measuring properties such as the length of the segment. We propose a novel technique to partition the problem into sub-problems. The independently obtained partial solutions can then be recombined in an efﬁcient way, which allows us to solve label sequence learning problems with several thousands of labeled sequences. We have tested our algorithm for predicting gene structures, an important problem in computational biology. Results on a well-known model organism illustrate the great potential of SHM SVMs in computational biology. 1</p><br/>
<h2>reference text</h2><p>[1] Y. Altun, T. Hofmann, and A. Smola. Gaussian process classiﬁcation for segmenting and annotating sequences. In Proc. ICML 2004, 2004.</p>
<p>[2] Y. Altun, D. McAllester, and M. Belkin. Maximum margin semi-supervised learning for structured variables. In Proc. NIPS 2005, 2006.</p>
<p>[3] Y. Altun, I. Tsochantaridis, and T. Hofmann. Hidden Markov support vector machines. In T. Fawcett, editor, Proc. 20th Int. Conf. Mach. Learn., pages 3–10, 2003.</p>
<p>[4] B. Brejova, D.G. Brown, M. Li, and T. Vinar. ExonHunter: a comprehensive approach to gene ﬁnding. Bioinformatics, 21(Suppl 1):i57–i65, 2005.</p>
<p>[5] C. Burge and S. Karlin. Prediction of complete gene structures in human genomic DNA. Journal of Molecular Biology, 268:78–94, 1997.</p>
<p>[6] X. Ge. Segmental Semi-Markov Models and Applications to Sequence Analysis. PhD thesis, University of California, Irvine, 2002.</p>
<p>[7] J. Janssen and N. Limnios. Semi-Markov Models and Applications. Kluwer Academic, 1999.</p>
<p>[8] I. Korf. Gene ﬁnding in novel genomes. BMC Bioinformatics, 5(59), 2004.</p>
<p>[9] D. Kulp, D. Haussler, M.G. Reese, and F.H. Eeckman. A generalized hidden markov model for the recognition of human genes in DNA. ISMB 1996, pages 134–141, 1996.</p>
<p>[10] B. Lewin. Genes VII. Oxford University Press, New York, 2000.</p>
<p>[11] L.R. Rabiner. A tutorial on hidden markov models and selected applications in speech recognition. Proceedings of the IEEE, 77(2):257–285, February 1989.</p>
<p>[12] G. R¨ tsch and S. Sonnenburg. Accurate splice site prediction for Caenorhabditis elegans. In B. Sch¨ lkopf, a o K. Tsuda, and J.-P. Vert, editors, Kernel Methods in Computational Biology. MIT Press, 2004.</p>
<p>[13] G. R¨ tsch, S. Sonnenburg, J. Srinivasan, H. Witte, K.-R. M¨ ller, R. Sommer, and B. Sch¨ lkopf. Improving a u o the C. elegans genome annotation using machine learning. PLoS Computational Biology, 2007. In press.</p>
<p>[14] S. Sarawagi and W.W. Cohen. Semi-markov conditional random ﬁelds for information extraction. In Proc. NIPS 2004, 2005.</p>
<p>[15] G.D. Stormo and D. Haussler. Optimally parsing a sequence into different classes based on multiple types of information. In Proc. ISMB 1994, pages 369–375, Menlo Park, CA, 1994. AAAI/MIT Press.</p>
<p>[16] R. Sutton, D. Precup, and S. Singh. Between mdps and semi-mdps: A framework for temporal abstraction in reinforcement learrning. Artiﬁcial Intelligence, 112:181–211, 1999.</p>
<p>[17] B. Taskar, C. Guestrin, and D. Koller. Max-margin markov networks. In Proc. NIPS 2003, 16, 2004.</p>
<p>[18] I. Tsochantaridis, T. Hofmann, T. Joachims, and Y. Altun. Large margin methods for structured output spaces. Journal for Machine Learning Research, 6, September 2005.</p>
<p>[19] V.N. Vapnik. The nature of statistical learning theory. Springer Verlag, New York, 1995.</p>
<p>[20] A. J. Viterbi. Error bounds for convolutional codes and an asymptotically optimal decoding algorithm. IEEE Trans. Informat. Theory, IT-13:260–269, Apr 1967.</p>
<p>[21] X.H. Zhang, K.A. Heller, I. Hefter, C.S. Leslie, and L.A. Chasin. Sequence information for the splicing of human pre-mRNA identiﬁed by SVM classiﬁcation. Genome Res, 13(12):2637–50, 2003.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
