<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>64 nips-2006-Data Integration for Classification Problems Employing Gaussian Process Priors</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2006" href="../home/nips2006_home.html">nips2006</a> <a title="nips-2006-64" href="../nips2006/nips-2006-Data_Integration_for_Classification_Problems_Employing_Gaussian_Process_Priors.html">nips2006-64</a> <a title="nips-2006-64-reference" href="#">nips2006-64-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>64 nips-2006-Data Integration for Classification Problems Employing Gaussian Process Priors</h1>
<br/><p>Source: <a title="nips-2006-64-pdf" href="http://papers.nips.cc/paper/3065-data-integration-for-classification-problems-employing-gaussian-process-priors.pdf">pdf</a></p><p>Author: Mark Girolami, Mingjun Zhong</p><p>Abstract: By adopting Gaussian process priors a fully Bayesian solution to the problem of integrating possibly heterogeneous data sets within a classiﬁcation setting is presented. Approximate inference schemes employing Variational & Expectation Propagation based methods are developed and rigorously assessed. We demonstrate our approach to integrating multiple data sets on a large scale protein fold prediction problem where we infer the optimal combinations of covariance functions and achieve state-of-the-art performance without resorting to any ad hoc parameter tuning and classiﬁer combination. 1</p><br/>
<h2>reference text</h2><p>[1] A. Ben-Hur and W.S. Noble. Kernel methods for predicting protein-protein interactions. Bioinformatics, 21, Suppl. 1:38–46, 2005.</p>
<p>[2] Chris Ding and Inna Dubchak. Multi-class protein fold recognition using support vector machines and neural networks. Bioinformatics, 17:349–358, 2001.</p>
<p>[3] Mark Girolami and Simon Rogers. Variational Bayesian multinomial probit regression with Gaussian process priors. Neural Computation, 18(8):1790–1817, 2006.</p>
<p>[4] M. Kuss and C.E. Rasmussen. Assessing approximate inference for binary Gaussian process classiﬁcation. Journal of Machine Learning Research, 6:1679–1704, 2005.</p>
<p>[5] G. R. G. Lanckriet, T. De Bie, N. Cristianini, M. I. Jordan, and W. S. Noble. A statistical framework for genomic data fusion. Bioinformatics, 20:2626–2635, 2004.</p>
<p>[6] Neil Lawrence, Matthias Seeger, and Ralf Herbrich. Fast sparse Gaussian process methods: The informative vector machine. In S. Thrun S. Becker and K. Obermayer, editors, Advances in Neural Information Processing Systems 15. MIT Press.</p>
<p>[7] Thomas Minka. A family of algorithms for approximate Bayesian inference. PhD thesis, MIT, 2001.</p>
<p>[8] R. Neal. Regression and classiﬁcation using Gaussian process priors. In A.P. Dawid, M. Bernardo, J.O. Berger, and A.F.M. Smith, editors, Bayesian Statistics 6, pages 475–501. Oxford University Press, 1998.</p>
<p>[9] Paul Pavlidis, Jason Weston, Jinsong Cai, and William Stafford Noble. Learning gene functional classiﬁcations from multiple data types. Journal of Computational Biology, 9(2):401– 411, 2002.</p>
<p>[10] J.C. Platt. Probabilities for support vector machines. In A. Smola, P. Bartlett, B. Schlkopf, and D. Schuurmans, editors, Advances in Large Margin Classiﬁers, pages 61–74. MIT Press, 1999.</p>
<p>[11] Carl Edward Rasmussen and Christopher K. I. Williams. Gaussian Processes for Machine Learning. MIT Press, 2006.</p>
<p>[12] M.W. Seeger, N.D. Lawrence, and R. Herbrich. Efﬁcient nonparametric Bayesian modelling with sparse Gaussian process approximations. Technical Report, ”http://www.kyb.tuebingen.mpg.de/bs/people/seeger/”, 2006.</p>
<p>[13] Hong-Bin Shen and Kuo-Chen Chou. Ensemble classiﬁer for protein fold pattern recognition. Bioinformatics, Advanced Access(doi:10.1093), 2006.</p>
<p>[14] Alexander Smola, Vishy Vishwanathan, and Eleazar Eskin. Laplace propagation. In Sebastian Thrun, Lawrence Saul, and Bernhard Sch¨ lkopf, editors, Advances in Neural Information o Processing Systems 16. MIT Press, Cambridge, MA, 2004.</p>
<p>[15] C.K.I. Williams and D. Barber. Bayesian classiﬁcation with Gaussian processes. IEEE Transactions on Pattern Analysis and Machine Intelligence, 20(12):1342–1352, 1998.</p>
<p>[16] Y. Yamanishi, J. P. Vert, and M. Kanehisa. Protein network inference from multiple genomic data: a supervised approach. Bioinformatics, 20, Suppl. 1:363–370, 2004.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
