<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>55 nips-2006-Computation of Similarity Measures for Sequential Data using Generalized Suffix Trees</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2006" href="../home/nips2006_home.html">nips2006</a> <a title="nips-2006-55" href="../nips2006/nips-2006-Computation_of_Similarity_Measures_for_Sequential_Data_using_Generalized_Suffix_Trees.html">nips2006-55</a> <a title="nips-2006-55-reference" href="#">nips2006-55-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>55 nips-2006-Computation of Similarity Measures for Sequential Data using Generalized Suffix Trees</h1>
<br/><p>Source: <a title="nips-2006-55-pdf" href="http://papers.nips.cc/paper/2999-computation-of-similarity-measures-for-sequential-data-using-generalized-suffix-trees.pdf">pdf</a></p><p>Author: Konrad Rieck, Pavel Laskov, Sören Sonnenburg</p><p>Abstract: We propose a generic algorithm for computation of similarity measures for sequential data. The algorithm uses generalized sufﬁx trees for efﬁcient calculation of various kernel, distance and non-metric similarity functions. Its worst-case run-time is linear in the length of sequences and independent of the underlying embedding language, which can cover words, k-grams or all contained subsequences. Experiments with network intrusion detection, DNA analysis and text processing applications demonstrate the utility of distances and similarity coefﬁcients for sequences as alternatives to classical kernel functions.</p><br/>
<h2>reference text</h2><p>[1] V.N. Vapnik. Statistical Learning Theory. Wiley, New York, 1998.</p>
<p>[2] B. Sch¨ lkopf and A.J. Smola. Learning with Kernels. MIT Press, Cambridge, MA, 2002. o</p>
<p>[3] J. Shawe-Taylor and N. Cristianini. Kernel methods for pattern analysis. Cambridge University Press, 2004.</p>
<p>[4] C. Watkins. Dynamic alignment kernels. In A.J. Smola, P.L. Bartlett, B. Sch¨ lkopf, and D. Schuurmans, o editors, Advances in Large Margin Classiﬁers, pages 39–50, Cambridge, MA, 2000. MIT Press.</p>
<p>[5] D. Haussler. Convolution kernels on discrete structures. Technical Report UCSC-CRL-99-10, UC Santa Cruz, July 1999.</p>
<p>[6] T. Joachims. Text categorization with support vector machines: Learning with many relevant features. Technical Report 23, LS VIII, University of Dortmund, 1997.</p>
<p>[7] E. Leopold and J. Kindermann. Text categorization with Support Vector Machines. how to represent texts in input space? Machine Learning, 46:423–444, 2002.</p>
<p>[8] H. Lodhi, C. Saunders, J. Shawe-Taylor, N. Cristianini, and C. Watkins. Text classiﬁcation using string kernels. Journal of Machine Learning Research, 2:419–444, 2002.</p>
<p>[9] A. Zien, G. R¨ tsch, S. Mika, B. Sch¨ lkopf, T. Lengauer, and K.-R. M¨ ller. Engineering Support Vector a o u Machine Kernels That Recognize Translation Initiation Sites. BioInformatics, 16(9):799–807, September 2000.</p>
<p>[10] C. Leslie, E. Eskin, and W.S. Noble. The spectrum kernel: A string kernel for SVM protein classiﬁcation. In Proc. Paciﬁc Symp. Biocomputing, pages 564–575, 2002.</p>
<p>[11] C. Leslie, E. Eskin, A. Cohen, J. Weston, and W.S. Noble. Mismatch string kernel for discriminative protein classiﬁcation. Bioinformatics, 1(1):1–10, 2003.</p>
<p>[12] J. Rousu and J. Shawe-Taylor. Efﬁcient computation of gapped substring kernels for large alphabets. Journal of Machine Leaning Research, 6:1323–1344, 2005.</p>
<p>[13] G. R¨ tsch, S. Sonnenburg, and B. Sch¨ lkopf. RASE: recognition of alternatively spliced exons in c. a o elegans. Bioinformatics, 21:i369–i377, June 2005.</p>
<p>[14] S. Sonnenburg, A. Zien, and G. R¨ tsch. ARTS: Accurate Recognition of Transcription Starts in Human. a Bioinformatics, 22(14):e472–e480, 2006.</p>
<p>[15] H. Drucker, D. Wu, and V.N. Vapnik. Support vector machines for spam categorization. IEEE Transactions on Neural Networks, 10(5):1048–1054, 1999.</p>
<p>[16] E. Eskin, A. Arnold, M. Prerau, L. Portnoy, and S. Stolfo. Applications of Data Mining in Computer Security, chapter A geometric framework for unsupervised anomaly detection: detecting intrusions in unlabeled data. Kluwer, 2002.</p>
<p>[17] K. Rieck and P. Laskov. Detecting unknown network attacks using language models. In Proc. DIMVA, pages 74–90, July 2006.</p>
<p>[18] T. Graepel, R. Herbrich, P. Bollmann-Sdorra, and K. Obermayer. Classiﬁcation on pairwise proximity data. In M.S. Kearns, S.A. Solla, and D.A. Cohn, editors, Advances in Neural Information Processing Systems, volume 11, pages 438–444. MIT Press, 1999.</p>
<p>[19] V. Roth, J Laub, M. Kawanabe, and J.M. Buhmann. Optimal cluster preserving embedding of non-metric proximity data. IEEE Trans. PAMI, 25:1540–1551, December 2003.</p>
<p>[20] J. Laub and K.-R. M¨ ller. Feature discovery in non-metric pairwise data. Journal of Machine Learning, u 5(Jul):801–818, July 2004.</p>
<p>[21] C. Ong, X. Mary, S. Canu, and A.J. Smola. Learning with non-positive kernels. In Proc. ICML, pages 639–646, 2004.</p>
<p>[22] G. Salton. Mathematics and information retrieval. Journal of Documentation, 35(1):1–29, 1979.</p>
<p>[23] M. Damashek. Gauging similarity with n-grams: Language-independent categorization of text. Science, 267(5199):843–848, 1995.</p>
<p>[24] S.V.N. Vishwanathan and A.J. Smola. Kernels and Bioinformatics, chapter Fast Kernels for String and Tree Matching, pages 113–130. MIT Press, 2004.</p>
<p>[25] K. Rieck, P. Laskov, and K.-R. M¨ ller. Efﬁcient algorithms for similarity measures over sequential data: u A look beyond kernels. In Proc. DAGM, pages 374–383, September 2006.</p>
<p>[26] R.R. Sokal and P.H. Sneath. Principles of numerical taxonomy. Freeman, San Francisco, CA, USA, 1963.</p>
<p>[27] M.R. Anderberg. Cluster Analysis for Applications. Academic Press, Inc., New York, NY, USA, 1973.</p>
<p>[28] E. Fredkin. Trie memory. Communications of ACM, 3(9):490–499, 1960.</p>
<p>[29] D. Knuth. The art of computer programming, volume 3. Addison-Wesley, 1973.</p>
<p>[30] P. Weiner. Linear pattern matching algorithms. In Proc. 14th Annual Symposium on Switching and Automata Theory, pages 1–11, 1973.</p>
<p>[31] D. Gusﬁeld. Algorithms on strings, trees, and sequences. Cambridge University Press, 1997.</p>
<p>[32] E. Ukkonen. Online construction of sufﬁx trees. Algorithmica, 14(3):249–260, 1995.</p>
<p>[33] C.H. Teo and S.V.N. Vishwanathan. Fast and space efﬁcient string kernels using sufﬁx arrays. In Proceedings, 23rd ICMP, pages 939–936. ACM Press, 2006.</p>
<p>[34] M.I. Abouelhoda, S. Kurtz, and E. Ohlebusch. Replacing sufﬁx trees with enhanced sufﬁx arrays. Journal of Discrete Algorithms, 2(1):53–86, 2002.</p>
<p>[35] R. Lippmann, J.W. Haines, D.J. Fried, J. Korba, and K. Das. The 1999 DARPA off-line intrusion detection evaluation. Computer Networks, 34(4):579–595, 2000.</p>
<p>[36] D.D. Lewis. Reuters-21578 text categorization test collection. AT&T; Labs Research, 1997.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
