<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>135 nips-2006-Modelling transcriptional regulation using Gaussian Processes</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2006" href="../home/nips2006_home.html">nips2006</a> <a title="nips-2006-135" href="../nips2006/nips-2006-Modelling_transcriptional_regulation_using_Gaussian_Processes.html">nips2006-135</a> <a title="nips-2006-135-reference" href="#">nips2006-135-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>135 nips-2006-Modelling transcriptional regulation using Gaussian Processes</h1>
<br/><p>Source: <a title="nips-2006-135-pdf" href="http://papers.nips.cc/paper/3119-modelling-transcriptional-regulation-using-gaussian-processes.pdf">pdf</a></p><p>Author: Neil D. Lawrence, Guido Sanguinetti, Magnus Rattray</p><p>Abstract: Modelling the dynamics of transcriptional processes in the cell requires the knowledge of a number of key biological quantities. While some of them are relatively easy to measure, such as mRNA decay rates and mRNA abundance levels, it is still very hard to measure the active concentration levels of the transcription factor proteins that drive the process and the sensitivity of target genes to these concentrations. In this paper we show how these quantities for a given transcription factor can be inferred from gene expression levels of a set of known target genes. We treat the protein concentration as a latent function with a Gaussian process prior, and include the sensitivities, mRNA decay rates and baseline expression levels as hyperparameters. We apply this procedure to a human leukemia dataset, focusing on the tumour repressor p53 and obtaining results in good accordance with recent biological studies.</p><br/>
<h2>reference text</h2><p>[1] M. Barenco, D. Tomescu, D. Brewer, R. Callard, J. Stark, and M. Hubank. Ranked prediction of p53 targets using hidden variable dynamic modeling. Genome Biology, 7(3):R25, 2006.</p>
<p>[2] T. Graepel. Solving noisy linear operator equations by Gaussian processes: Application to ordinary and partial differential equations. In T. Fawcett and N. Mishra, editors, Proceedings of the International Conference in Machine Learning, volume 20, pages 234–241. AAAI Press, 2003.</p>
<p>[3] J. C. Liao, R. Boscolo, Y.-L. Yang, L. M. Tran, C. Sabatti, and V. P. Roychowdhury. Network component analysis: Reconstruction of regulatory signals in biological systems. Proceedings of the National Academy of Sciences USA, 100(26):15522–15527, 2003.</p>
<p>[4] X. Liu, M. Milo, N. D. Lawrence, and M. Rattray. A tractable probabilistic model for affymetrix probelevel analysis across multiple chips. Bioinformatics, 21(18):3637–3644, 2005.</p>
<p>[5] N. A. Monk. Unravelling nature’s networks. Biochemical Society Transactions, 31:1457–1461, 2003.</p>
<p>[6] R. Murray-Smith and B. A. Pearlmutter. Transformations of Gaussian process priors. In J. Winkler, N. D. Lawrence, and M. Niranjan, editors, Deterministic and Statistical Methods in Machine Learning, volume 3635 of Lecture Notes in Artiﬁcial Intelligence, pages 110–123, Berlin, 2005. Springer-Verlag.</p>
<p>[7] R. M. Neal. Bayesian Learning for Neural Networks. Springer, 1996. Lecture Notes in Statistics 118.</p>
<p>[8] C. E. Rasmussen and C. K. Williams. Gaussian Processes for Machine Learning. MIT press, 2005.</p>
<p>[9] S. Rogers, R. Khanin, and M. Girolami. Model based identiﬁcation of transcription factor activity from microarray data. In Probabilistic Modeling and Machine Learning in Structural and Systems Biology, Tuusula, Finland, 17-18th June 2006.</p>
<p>[10] C. Sabatti and G. M. James. Bayesian sparse hidden components analysis for transcription regulation networks. Bioinformatics, 22(6):739–746, 2006.</p>
<p>[11] G. Sanguinetti, M. Rattray, and N. D. Lawrence. A probabilistic dynamical model for quantitative inference of the regulatory mechanism of transcription. Bioinformatics, 22(14):1753–1759, 2006.</p>
<p>[12] C. K. I. Williams. Computation with inﬁnite neural networks. Neural Computation, 10(5):1203–1216, 1998.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
