<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>142 nips-2006-Mutagenetic tree Fisher kernel improves prediction of HIV drug resistance from viral genotype</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2006" href="../home/nips2006_home.html">nips2006</a> <a title="nips-2006-142" href="../nips2006/nips-2006-Mutagenetic_tree_Fisher_kernel_improves_prediction_of_HIV_drug_resistance_from_viral_genotype.html">nips2006-142</a> <a title="nips-2006-142-reference" href="#">nips2006-142-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>142 nips-2006-Mutagenetic tree Fisher kernel improves prediction of HIV drug resistance from viral genotype</h1>
<br/><p>Source: <a title="nips-2006-142-pdf" href="http://papers.nips.cc/paper/2978-mutagenetic-tree-fisher-kernel-improves-prediction-of-hiv-drug-resistance-from-viral-genotype.pdf">pdf</a></p><p>Author: Tobias Sing, Niko Beerenwinkel</p><p>Abstract: Starting with the work of Jaakkola and Haussler, a variety of approaches have been proposed for coupling domain-speciﬁc generative models with statistical learning methods. The link is established by a kernel function which provides a similarity measure based inherently on the underlying model. In computational biology, the full promise of this framework has rarely ever been exploited, as most kernels are derived from very generic models, such as sequence proﬁles or hidden Markov models. Here, we introduce the MTreeMix kernel, which is based on a generative model tailored to the underlying biological mechanism. Speciﬁcally, the kernel quantiﬁes the similarity of evolutionary escape from antiviral drug pressure between two viral sequence samples. We compare this novel kernel to a standard, evolution-agnostic amino acid encoding in the prediction of HIV drug resistance from genotype, using support vector regression. The results show signiﬁcant improvements in predictive performance across 17 anti-HIV drugs. Thus, in our study, the generative-discriminative paradigm is key to bridging the gap between population genetic modeling and clinical decision making. 1</p><br/>
<h2>reference text</h2><p>[1] B. Sch¨ lkopf, K. Tsuda, and J.-P. Vert, editors. Kernel methods in computational biology. MIT Press, o Cambridge, MA, 2004.</p>
<p>[2] T. Jaakkola and D. Haussler. Exploiting generative models in discriminative classiﬁers. In M. J. Kearns, S. A. Solla, and D. A. Cohn, editors, Advances in Neural Information Processing Systems 11, pages 487–493. MIT Press, Cambridge, MA, 1999. ¨</p>
<p>[3] K. Tsuda, M. Kawanabe, G. R¨ tsch, S. Sonnenburg, and K. Muller. A new discriminative kernel from a probabilistic models. In T.G. Dietterich, S. Becker, and Z. Ghahramani, editors, Advances in Neural Information Processing Systems 14, pages 977–984. MIT Press, Cambridge, MA, 2002.</p>
<p>[4] S. Amari and H. Nagaoka. Methods of Information Geometry. American Mathematical Society, Oxford University Press, 2000.</p>
<p>[5] K. Tsuda, T. Kin, and K. Asai. Marginalized kernels for biological sequences. Bioinformatics, 18 Suppl 1:S268–S275, 2002.</p>
<p>[6] J. Lafferty and G. Lebanon. Information diffusion kernels. In S. Becker, S. Thrun, and K. Obermayer, editors, Advances in Neural Information Processing Systems 15, pages 375–382. MIT Press, Cambridge, MA, 2003.</p>
<p>[7] T. Jebara, R. Kondor, and A. Howard. Probability product kernels. Journal of Machine Learning Research, 5:819–844, July 2004. ¨</p>
<p>[8] N. Beerenwinkel, T. Sing, T. Lengauer, J. Rahnenfuhrer, K. Roomp, I. Savenkov, R. Fischer, D. Hoffmann, J. Selbig, K. Korn, H. Walter, T. Berg, P. Braun, G. F¨ tkenheuer, M. Oette, J. Rockstroh, B. Kupfer, a R. Kaiser, and M. D¨ umer. Computational methods for the design of effective therapies against drug a resistant HIV strains. Bioinformatics, 21(21):3943–3950, Sep 2005.</p>
<p>[9] F. Clavel and A. J. Hance. HIV drug resistance. N Engl J Med, 350(10):1023–1035, Mar 2004.</p>
<p>[10] R. W. Shafer and J. M. Schapiro. Drug resistance and antiretroviral drug development. J Antimicrob Chemother, 55(6):817–820, Jun 2005. ¨</p>
<p>[11] N. Beerenwinkel, J. Rahnenfuhrer, M. D¨ umer, D. Hoffmann, R. Kaiser, J. Selbig, and T. Lengauer. a Learning multiple evolutionary pathways from cross-sectional data. J Comput Biol, 12(6):584–598, 2005.</p>
<p>[12] J. C. Schmit, L. Ruiz, L. Stuyver, K. Van Laethem, I. Vanderlinden, T. Puig, R. Rossau, J. Desmyter, E. De Clercq, B. Clotet, and A. M. Vandamme. Comparison of the LiPA HIV-1 RT test, selective PCR and direct solid phase sequencing for the detection of HIV-1 drug resistance mutations. J Virol Methods, 73(1):77–82, Jul 1998.</p>
<p>[13] M. Rabinowitz, L. Myers, M. Banjevic, A. Chan, J. Sweetkind-Singer, J. Haberer, K. McCann, and R. Wolkowicz. Accurate prediction of HIV-1 drug response from the reverse transcriptase and protease amino acid sequences using sparse models created by convex optimization. Bioinformatics, 22(5):541– 549, Mar 2006. ¨</p>
<p>[14] H. Walter, B. Schmidt, K. Korn, A. M. Vandamme, T. Harrer, and K. Uberla. Rapid, phenotypic HIV-1 drug sensitivity assay for protease and reverse transcriptase inhibitors. J. Clin. Virol., 13:71–80, 1999.</p>
<p>[15] V. A. Johnson, F. Brun-Vezinet, B. Clotet, B. Conway, D. R. Kuritzkes, D. Pillay, J. M. Schapiro, A. Telenti, and D. D. Richman. Update of the drug resistance mutations in HIV-1: Fall 2005. Topics in HIV Medicine, 13(4):125–131, 2005.</p>
<p>[16] J. Rahnenf¨ hrer, N. Beerenwinkel, W. A. Schulz, C. Hartmann, A. von Deimling, B. Wullich, and u T. Lengauer. Estimating cancer survival and clinical outcome based on genetic tumor progression scores. Bioinformatics, 21(10):2438–2446, May 2005.</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
