<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>246 nips-2009-Time-Varying Dynamic Bayesian Networks</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2009" href="../home/nips2009_home.html">nips2009</a> <a title="nips-2009-246" href="../nips2009/nips-2009-Time-Varying_Dynamic_Bayesian_Networks.html">nips2009-246</a> <a title="nips-2009-246-reference" href="#">nips2009-246-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>246 nips-2009-Time-Varying Dynamic Bayesian Networks</h1>
<br/><p>Source: <a title="nips-2009-246-pdf" href="http://papers.nips.cc/paper/3716-time-varying-dynamic-bayesian-networks.pdf">pdf</a></p><p>Author: Le Song, Mladen Kolar, Eric P. Xing</p><p>Abstract: Directed graphical models such as Bayesian networks are a favored formalism for modeling the dependency structures in complex multivariate systems such as those encountered in biology and neural science. When a system is undergoing dynamic transformation, temporally rewiring networks are needed for capturing the dynamic causal inﬂuences between covariates. In this paper, we propose time-varying dynamic Bayesian networks (TV-DBN) for modeling the structurally varying directed dependency structures underlying non-stationary biological/neural time series. This is a challenging problem due the non-stationarity and sample scarcity of time series data. We present a kernel reweighted 1 -regularized auto-regressive procedure for this problem which enjoys nice properties such as computational efﬁciency and provable asymptotic consistency. To our knowledge, this is the ﬁrst practical and statistically sound method for structure learning of TVDBNs. We applied TV-DBNs to time series measurements during yeast cell cycle and brain response to visual stimuli. In both cases, TV-DBNs reveal interesting dynamics underlying the respective biological systems. 1</p><br/>
<h2>reference text</h2><p>[1] A. L. Barabasi and Z. N. Oltvai. Network biology: Understanding the cell’s functional organization. Nature Reviews Genetics, 5(2):101–113, 2004.</p>
<p>[2] Francisco Varela, Jean-Philippe Lachaux, Eugenio Rodriguez, and Jacques Martinerie. The brainweb: Phase synchronization and large-scale integration. Nature Reviews Neuroscience, 2:229–239, 2001.</p>
<p>[3] N. Luscombe, M. Babu, H. Yu, M. Snyder, S. Teichmann, and M. Gerstein. Genomic analysis of regulatory network dynamics reveals large topological changes. Nature, 431:308–312, 2004.</p>
<p>[4] Eugenio Rodriguez, Nathalie George, Jean-Philippe Lachaux, Jacques Martinerie, Bernard Renault, and Francisco J. Varela1. Perception’s shadow: long-distance synchronization of human brain activity. Nature, 397(6718):430–433, 1999.</p>
<p>[5] M. Talih and N. Hengartner. Structural learning with time-varying components: Tracking the crosssection of ﬁnancial time series. J. Royal Stat. Soc. B, 67(3):321C341, 2005.</p>
<p>[6] S. Hanneke and E. P. Xing. Discrete temporal models of social networks. In Workshop on Statistical Network Analysis, ICML06, 2006.</p>
<p>[7] F. Guo, S. Hanneke, W. Fu, and E. P. Xing. Recovering temporally rewiring networks: A model-based approach. In International Conference in Machine Learning, 2007.</p>
<p>[8] X. Xuan and K. Murphy. Modeling changing dependency structure in multivariate time series. In International Conference in Machine Learning, 2007.</p>
<p>[9] J. Robinson and A. Hartemink. Non-stationary dynamic bayesian networks. In Neural Information Processing Systems, 2008.</p>
<p>[10] Amr Ahmed and Eric P. Xing. Tesla: Recovering time-varying networks of dependencies in social and biological studies. Proceeding of the National Academy of Sciences, in press, 2009.</p>
<p>[11] S. Zhou, J. Lafferty, and L. Wasserman. Time varying undirected graphs. In Computational Learning Theory, 2008.</p>
<p>[12] L. Song, M. Kolar, and E. Xing. Keller: Estimating time-evolving interactions between genes. In Bioinformatics (ISMB), 2009.</p>
<p>[13] N. Friedman, M. Linial, I. Nachman, and D. Peter. Using bayesian networks to analyze expression data. Journal of Computational Biology, 7:601–620, 2000.</p>
<p>[14] N. Dobingeon, J. Tourneret, and M. Davy. Joint segmentation of piecewise constant autoregressive processes by using a hierarchical model and a bayesian sampling approach. IEEE Transactions on Signal Processing, 55(4):1251–1263, 2007.</p>
<p>[15] K. Kanazawa, D. Koller, and S. Russell. Stochastic simulation algorithms for dynamic probabilistic networks. Uncertainty in AI, 1995.</p>
<p>[16] L. Getoor, N. Friedman, D. Koller, and B. Taskar. Learning probabilistic models with link uncertainty. Journal of Machine Learning Research, 2002.</p>
<p>[17] R. Dahlhaus. Fitting time series models to nonstationary processes. Ann. Statist, (25):1–37, 1997.</p>
<p>[18] C. Andrieu, M. Davy, and A. Doucet. Efﬁcient particle ﬁltering for jump markov systems: Application to time-varying autoregressions. IEEE Transactions on Signal Processing, 51(7):1762–1770, 2003.</p>
<p>[19] E. H. Davidson. Genomic Regulatory Systems. Academic Press, 2001.</p>
<p>[20] Florentina Bunea. Honest variable selection in linear and logistic regression models via 1 and 1 + 2 penalization. Electronic Journal of Statistics, 2:1153, 2008.</p>
<p>[21] W. Fu. Penalized regressions: the bridge versus the lasso. Journal of Computational and Graphical Statistics, 7(3):397–416, 1998.</p>
<p>[22] M. Schmidt, A. Niculescu-Mizil, and K Murphy. Learning graphical model structure using l1regularization paths. In AAAI, 2007.</p>
<p>[23] Tata Pramila, Wei Wu, Shawna Miles, William Noble, and Linda Breeden. The forkhead transcription factor hcm1 regulates chromosome segregation genes and ﬁlls the s-phase gap in the transcriptional circuitry of the cell cycle. Gene and Development, 20:2266–2278, 2006.</p>
<p>[24] Jun Zhu, Bin Zhang, Erin Smith, Becky Drees, Rachel Brem, Leonid Kruglyak, Roger Bumgarner, and Eric E Schadt. Integrating large-scale functional genomic data to dissect the complexity of yeast regulatory networks. Nature Genetics, 40:854–861, 2008.</p>
<p>[25] T. Nichols and A. Holmes. Nonparametric permutation tests for functional neuroimaging: a primer with examples. Human Brain Mapping, 15:1–25, 2001.</p>
<p>[26] G. Dornhege, B. Blankertz, G. Curio, and K.R. M¨ ller. Boosting bit rates in non-invasive eeg single-trial u classiﬁcations by feature combination and multi-class paradigms. IEEE Trans. Biomed. Eng., 51:993– 1002, 2004.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
