<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>168 nips-2009-Non-stationary continuous dynamic Bayesian networks</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2009" href="../home/nips2009_home.html">nips2009</a> <a title="nips-2009-168" href="../nips2009/nips-2009-Non-stationary_continuous_dynamic_Bayesian_networks.html">nips2009-168</a> <a title="nips-2009-168-reference" href="#">nips2009-168-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>168 nips-2009-Non-stationary continuous dynamic Bayesian networks</h1>
<br/><p>Source: <a title="nips-2009-168-pdf" href="http://papers.nips.cc/paper/3687-non-stationary-continuous-dynamic-bayesian-networks.pdf">pdf</a></p><p>Author: Marco Grzegorczyk, Dirk Husmeier</p><p>Abstract: Dynamic Bayesian networks have been applied widely to reconstruct the structure of regulatory processes from time series data. The standard approach is based on the assumption of a homogeneous Markov chain, which is not valid in many realworld scenarios. Recent research efforts addressing this shortcoming have considered undirected graphs, directed graphs for discretized data, or over-ﬂexible models that lack any information sharing among time series segments. In the present article, we propose a non-stationary dynamic Bayesian network for continuous data, in which parameters are allowed to vary among segments, and in which a common network structure provides essential information sharing across segments. Our model is based on a Bayesian multiple change-point process, where the number and location of the change-points is sampled from the posterior distribution. 1</p><br/>
<h2>reference text</h2><p>[1] N. Friedman, M. Linial, I. Nachman, and D. Pe’er. Using Bayesian networks to analyze expression data. Journal of Computational Biology, 7:601–620, 2000.</p>
<p>[2] K. Sachs, O. Perez, D. Pe’er, D. A. Lauffenburger, and G. P. Nolan. Protein-signaling networks derived from multiparameter single-cell data. Science, 308:523–529, 2005.</p>
<p>[3] V. A. Smith, J. Yu, T. V. Smulders, A. J. Hartemink, and E. D. Jarvi. Computational inference of neural information ﬂow networks. PLoS Computational Biology, 2:1436–1449, 2006.</p>
<p>[4] M. Talih and N. Hengartner. Structural learning with time-varying components: Tracking the crosssection of ﬁnancial time series. Journal of the Royal Statistical Society B, 67(3):321–341, 2005.</p>
<p>[5] X. Xuan and K. Murphy. Modeling changing dependency structure in multivariate time series. In Zoubin Ghahramani, editor, Proceedings of the 24th Annual International Conference on Machine Learning (ICML 2007), pages 1055–1062. Omnipress, 2007.</p>
<p>[6] J. W. Robinson and A. J. Hartemink. Non-stationary dynamic Bayesian networks. In D. Koller, D. Schuurmans, Y. Bengio, and L. Bottou, editors, Advances in Neural Information Processing Systems 21, pages 1369–1376. Morgan Kaufmann Publishers, 2009.</p>
<p>[7] S. L` bre. Analyse de processus stochastiques pour la g´ nomique : etude du mod` le MTD et inf´ rence de e e e e ´ r´ seaux bay´ siens dynamiques. PhD thesis, Universit´ d‘Evry-Val-d‘Essonne, 2008. e e e</p>
<p>[8] D. Heckerman, D. Geiger, and D. M. Chickering. Learning Bayesian networks: The combination of knowledge and statistical data. Machine Learning, 20:245–274, 1995.</p>
<p>[9] C. Andrieu and A. Doucet. Joint Bayesian model selection and estimation of noisy sinusoids via reversible jump MCMC. IEEE Transactions on Signal Processing, 47(10):2667–2676, 1999.</p>
<p>[10] D. Geiger and D. Heckerman. Learning Gaussian networks. In Proceedings of the Tenth Conference on Uncertainty in Artiﬁcial Intelligence, pages 235–243, San Francisco, CA., 1994. Morgan Kaufmann.</p>
<p>[11] M. Grzegorczyk, D. Husmeier, K. Edwards, P. Ghazal, and A. Millar. Modelling non-stationary gene regulatory processes with a non-homogeneous Bayesian network and the allocation sampler. Bioinformatics, 24(18):2071–2078, 2008.</p>
<p>[12] Y. Ko, C. Zhai, and S.L. Rodriguez-Zas. Inference of gene pathways using Gaussian mixture models. In BIBM International Conference on Bioinformatics and Biomedicine, pages 362–367. Fremont, CA, 2007.</p>
<p>[13] A. Nobile and A.T. Fearnside. Bayesian ﬁnite mixtures with an unknown number of components: The allocation sampler. Statistics and Computing, 17(2):147–162, 2007.</p>
<p>[14] G. Schwarz. Estimating the dimension of a model. Annals of Statistics, 6:461–464, 1978.</p>
<p>[15] P. Green. Reversible jump Markov chain Monte Carlo computation and Bayesian model determination. Biometrika, 82:711–732, 1995.</p>
<p>[16] N. Friedman and D. Koller. Being Bayesian about network structure. Machine Learning, 50:95–126, 2003.</p>
<p>[17] P. Giudici and R. Castelo. Improving Markov chain Monte Carlo model search for data mining. Machine Learning, 50:127–158, 2003.</p>
<p>[18] D. Madigan and J. York. Bayesian graphical models for discrete data. International Statistical Review, 63:215–232, 1995.</p>
<p>[19] P. Fearnhead. Exact and efﬁcient Bayesian inference for multiple changepoint problems. Statistics and Computing, 16:203–213, 2006.</p>
<p>[20] S. S. Shen-Orr, R. Milo, S. Mangan, and U. Alon. Network motifs in the transcriptional regulation network of Escherichia coli. Nature Genetics, 31:64–68, 2002.</p>
<p>[21] M. K. Dougherty, J. Muller, D. A. Ritt, M. Zhou, X. Z. Zhou, T. D. Copeland, T. P. Conrads, T. D. Veenstra, K. P. Lu, and D. K. Morrison. Regulation of Raf-1 by direct feedback phosphorylation. Molecular Cell, 17:215–224, 2005.</p>
<p>[22] A. J. Hartemink. Principled Computational Methods for the Validation and Discovery of Genetic Regulatory Networks. PhD thesis, MIT, 2001.</p>
<p>[23] A. P. Dempster, N. M. Laird, and D. B. Rubin. Maximum likelihood from incomplete data via the EM algorithm. Journal of the Royal Statistical Society, B39(1):1–38, 1977.</p>
<p>[24] I. T. Nabney. NETLAB: Algorithms for Pattern Recognition. Springer Verlag, New York, 2004.</p>
<p>[25] A. Gelman and D. B. Rubin. Inference from iterative simulation using multiple sequences. Statistical Science, 7:457–472, 1992.</p>
<p>[26] W.K. Lim, K. Wang, C. Lefebvre, and A. Califano. Comparative analysis of microarray normalization procedures: effects on reverse engineering gene networks. Bioinformatics, 23(13):i282–i288, 2007.</p>
<p>[27] K. D. Edwards, P. E. Anderson, A. Hall, N. S. Salathia, J. C.W. Locke, J. R. Lynn, M. Straume, J. Q. Smith, and A. J. Millar. Flowering locus C mediates natural variation in the high-temperature response of the Arabidopsis circadian clock. The Plant Cell, 18:639–650, 2006.</p>
<p>[28] T.C. Mockler, T.P. Michael, H.D. Priest, R. Shen, C.M. Sullivan, S.A. Givan, C. McEntee, S.A. Kay, and J. Chory. The diurnal project: Diurnal and circadian expression proﬁling, model-based pattern matching and promoter analysis. Cold Spring Harbor Symposia on Quantitative Biology, 72:353–363, 2007.</p>
<p>[29] C. R. McClung. Plant circadian rhythms. Plant Cell, 18:792–803, 2006.</p>
<p>[30] J.C.W. Locke, M.M. Southern, L. Kozma-Bognar, V. Hibberd, P.E. Brown, M.S. Turner, and A.J. Millar. Extension of a genetic network model by iterative experimentation and mathematical analysis. Molecular Systems Biology, 1:(online), 2005.</p>
<p>[31] S. Imoto, S. Kim, T. Goto, , S. Aburatani, K. Tashiro, Satoru Kuhara, and Satoru Miyano. Bayesian networks and nonparametric heteroscedastic regression for nonlinear modeling of genetic networks. Journal of Bioinformatics and Computational Biology, 1(2):231–252, 2003.  9</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
