<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>5 jmlr-2006-A Robust Procedure For Gaussian Graphical Model Search From Microarray Data WithpLarger Thann</title>
</head>

<body>
<p><a title="jmlr" href="../jmlr_home.html">jmlr</a> <a title="jmlr-2006" href="../home/jmlr2006_home.html">jmlr2006</a> <a title="jmlr-2006-5" href="../jmlr2006/jmlr-2006-A_Robust_Procedure_For_Gaussian_Graphical_Model_Search_From_Microarray_Data_WithpLarger_Thann.html">jmlr2006-5</a> <a title="jmlr-2006-5-reference" href="#">jmlr2006-5-reference</a> knowledge-graph by maker-knowledge-mining</p><script async src="//pagead2.googlesyndication.com/pagead/js/adsbygoogle.js"></script>
<!-- maker adsense -->
<ins class="adsbygoogle"
     style="display:inline-block;width:728px;height:90px"
     data-ad-client="ca-pub-5027806277543591"
     data-ad-slot="4192012269"></ins>
<script>
(adsbygoogle = window.adsbygoogle || []).push({});
</script>
<h1>5 jmlr-2006-A Robust Procedure For Gaussian Graphical Model Search From Microarray Data WithpLarger Thann</h1>
<br/><p>Source: <a title="jmlr-2006-5-pdf" href="http://jmlr.org/papers/volume7/castelo06a/castelo06a.pdf">pdf</a></p><p>Author: Robert Castelo, Alberto Roverato</p><p>Abstract: Learning of large-scale networks of interactions from microarray data is an important and challenging problem in bioinformatics. A widely used approach is to assume that the available data constitute a random sample from a multivariate distribution belonging to a Gaussian graphical model. As a consequence, the prime objects of inference are full-order partial correlations which are partial correlations between two variables given the remaining ones. In the context of microarray data the number of variables exceed the sample size and this precludes the application of traditional structure learning procedures because a sampling version of full-order partial correlations does not exist. In this paper we consider limited-order partial correlations, these are partial correlations computed on marginal distributions of manageable size, and provide a set of rules that allow one to assess the usefulness of these quantities to derive the independence structure of the underlying Gaussian graphical model. Furthermore, we introduce a novel structure learning procedure based on a quantity, obtained from limited-order partial correlations, that we call the non-rejection rate. The applicability and usefulness of the procedure are demonstrated by both simulated and real data. Keywords: Gaussian distribution, gene network, graphical model, microarray data, non-rejection rate, partial correlation, small-sample inference</p><br/>
<h2>reference text</h2><p>A.J. Butte, P. Tamayo, D. Slonim, T.R. Golub and I.S. Kohane. Discovering functional relationships between RNA expression and chemotherapeutic susceptibility using relevance networks. Proceedings of the National Academy of Sciences, 97(22): 12182-12186, 2000. R.G. Cowell, A.P. Dawid, S.L. Lauritzen and D.J. Spiegelhalter. Probabilistic networks and expert systems. Springer-Verlag, New York, 1999. D.R. Cox and N. Wermuth. Linear dependencies represented by chain graphs (with discussion). Statist. Sci., 8: 204–283, 1993. D.R. Cox and N. Wermuth. Multivariate dependencies: Models, analysis and interpretation. Chapman and Hall, London, 1996. A. de la Fuente, N. Bing, I. Hoeschele and P. Mendes. Discovery of meaningful associations in genomic data using partial correlation coefﬁcients. Bioinformatics, 20: 3565-3574, 2004. A.P. Dempster. Elements of continuous multivariate analysis. Addison-Wesley, Reading, Massachusetts, 1969. A.P. Dempster. Covariance selection. Biometrics, 28: 157–75, 1972. R. Diestel. (2005). Graph theory. Springer-Verlag, Heidelberg, 2005. A. Dobra, C. Hans, B. Jones, J.R. Nevins and M. West. Sparse graphical models for exploring gene expression data. J. Mult. Anal. 90: 196-212, 2004. M. Drton and M.D. Perlman. Model selection for Gaussian concentration graphs. Biometrika, 91(3): 591–602, 2004. 2648  G AUSSIAN G RAPHICAL M ODEL S EARCH F ROM M ICROARRAY DATA W ITH p L ARGER T HAN n  M. Drton and T. Richardson. Iterative conditional ﬁtting for estimation of a covariance matrix with zeros. Technical Report no. 469, Department of Statistics, University of Washington, 2004. R.L. Dykstra. Establishing the positive deﬁniteness of the sample covariance matrix. Ann. Math. Statist., 41(6): 2153–2154, 1970. D.E. Edwards. Introduction to graphical modelling. Springer-Verlag, New York, 2000. N. Friedman. Inferring cellular network using probabilistic graphical models. Science, 33: 799–805, 2004. B. Jones, A. Dobra, C. Carvalho, C. Hans, C. Carter and M. West. Experiments in stochastic computation for high-dimensional graphical models. Statistical Science, 20(4): 388–400, 2005. B. Jones and M. West. Covariance decomposition in undirected Gaussian graphical models. Biometrika, 92(4): 779-786, 2005. S.L. Lauritzen. Graphical models. Oxford University Press, Oxford, 1996. E.L. Lehmann. Testing statistical hypotheses, 2nd edition. Wiley, New York, 1986. P.M. Magwene and J. Kim. Estimating genomic coexpression networks using ﬁrst-order conditional independence. Genome Biology, 5: R100, 2004. A.A. Margolin, I. Nemenman, K. Basso, C. Wiggins, G. Stolovitzky, R.D. Favera and A. Califano. ARACNE: an algorithm for the reconstruction of gene regulatory networks ina mammalian cellular context. BMC Bioinformatics, 7(Suppl 1):S7, 2006. R. Milo, S. Shen-Orr, S. Itzkovitz, N. Kashtan, D. Chklovskii and U. Alon. Network motifs: simple building blocks of complex networks. Science, 298: 824–827, 2002. J. Pearl. Probabilistic reasoning in intelligent systems. Morgan Kaufmann, San Mateo, 1988. A.L. Rosenberg and L.S. Heath. Graph separators, with applications. Kluwer Academic Publishers, New York, 2001. A. Roverato. Hyper inverse Wishart distribution for non-decomposable graphs and its application to Bayesian inference for Gaussian graphical models. Scand. J. Statist., 29: 391–411, 2002. J. Sch¨ fer and K. Strimmer. An empirical Bayes approach to inferring large-scale gene association a networks. Bioinformatics, 21(6): 754-764, 2005a. J. Sch¨ fer and K. Strimmer. Learning large-scale graphical Gaussian models from genomic data. a In: J.F. Mendes. (Ed.). Proceeding of Science of Complex Networks: from Biology to the Internet and WWW (CNET 2004), Aveiro, PT, (Publisher: The American Institute of Physics), 2005b. J. Sch¨ fer and K. Strimmer. A shrinkage approach to large-scale covariance matrix estimation and a implications for functional genomics. Statistical Applications in Genetics and Molecular Biology, 4(1): article 32, 2005c. R. Steuer, J. Kurths, O. Fiehn and W. Weckwerth. Interpreting correlations in metabolomic networks. Bioch. Soc. Trans., 31: 1476-1478, 2003a. 2649  C ASTELO AND ROVERATO  R. Steuer, J. Kurths, O. Fiehn and W. Weckwerth. Observing and interpreting correlations in metabolomic networks. Bioinformatics, 19: 1019-1026, 2003b. M. West, C. Blanchette, H. Dressman, E. Huang, S. Ishida, R. Spang, H. Zuzan, J.A. Olson, J.R. Marks and J.R. Nevings. Predicting the clinical status of human breast cancer by using gene expression proﬁles. Proceedings of the National Academy of Sciences, 98(20): 11462–11467, 2001. A. Wille and P. B¨ hlmann. Low-order conditional independence graphs for inferring genetic netu works. Statistical Applications in Genetics and Molecular Biology, 5(1): article 1, 2006. A. Wille, P. Zimmermann, E. Vranov´ , A. F¨ rholz, O. Laule, S. Bleuler, L. Hennig, A. Preli´ , P. von a u c ¨ Rohr, L. Thiele, E. Zitzler, W. Gruissem and P. Buhlmann. Sparse graphical Gaussian modeling of the isoprenoid gene network in Arabidopsis thaliana. Genome Biology, 5:R92, 2004. E. Wit and J. McClure. Statistics for microarrays. Design, analysis and inference. Wiley, Chichester, 2004. J. Whittaker. Graphical models in applied multivariate statistics. Wiley, Chichester, 1990. F. Wong, C.K. Carter and R. Kohn. Efﬁcient estimation of covariance selection models. Biometrika, 90: 809–830, 2003. R. Yang, and J.O. Berger. Estimation of a covariance matrix using the reference priors. Ann. Statist., 3: 1195–1211, 1994. E. Yeger-Lotem, S. Sattath, N. Kashtan, S. Itzkovitz, R. Milo, R.Y. Pinter, U. Alon and H. Margalit. Network motifs in integrated cellular networks of transcription–regulation and protein–protein interaction. Proc. Natl. Acad. Sci., 101(16): 5934–5939, 2004.  2650</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
