<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>167 nips-2007-Regulator Discovery from Gene Expression Time Series of Malaria Parasites: a Hierachical Approach</title>
</head>

<body>
<p><a title="nips" href="../nips_home.html">nips</a> <a title="nips-2007" href="../home/nips2007_home.html">nips2007</a> <a title="nips-2007-167" href="../nips2007/nips-2007-Regulator_Discovery_from_Gene_Expression_Time_Series_of_Malaria_Parasites%3A_a_Hierachical_Approach.html">nips2007-167</a> <a title="nips-2007-167-reference" href="#">nips2007-167-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>167 nips-2007-Regulator Discovery from Gene Expression Time Series of Malaria Parasites: a Hierachical Approach</h1>
<br/><p>Source: <a title="nips-2007-167-pdf" href="http://papers.nips.cc/paper/3362-regulator-discovery-from-gene-expression-time-series-of-malaria-parasites-a-hierachical-approach.pdf">pdf</a></p><p>Author: José M. Hernández-lobato, Tjeerd Dijkstra, Tom Heskes</p><p>Abstract: We introduce a hierarchical Bayesian model for the discovery of putative regulators from gene expression data only. The hierarchy incorporates the knowledge that there are just a few regulators that by themselves only regulate a handful of genes. This is implemented through a so-called spike-and-slab prior, a mixture of Gaussians with different widths, with mixing weights from a hierarchical Bernoulli model. For efﬁcient inference we implemented expectation propagation. Running the model on a malaria parasite data set, we found four genes with signiﬁcant homology to transcription factors in an amoebe, one RNA regulator and three genes of unknown function (out of the top ten genes considered).</p><br/>
<h2>reference text</h2><p>[1] T.S. Gardner and J.J. Faith. Reverse-engineering transcription control networks. Physics of Life Reviews, 2:65–88, 2005.</p>
<p>[2] R. Coulson, N. Hall, and C. Ouzounis. Comparative genomics of transcriptional control in the human malaria parasite Plasmodium falciparum. Genome Res., 14:1548–1554, 2004.</p>
<p>[3] S. Balaji, M.M. Babu, L.M. Iyer, and L. Aravind. Discovery of the principal speciﬁc transcription factors of apicomplexa and their implication for the evolution of the ap2-integrase dna binding domains. Nucleic Acids Research, 33(13):3994–4006, 2005.</p>
<p>[4] T. Sakata and E.A. Winzeler. Genomics, systems biology and drug development for infectuous diseases. Molecular BioSystems, 3:841–848, 2007.</p>
<p>[5] P.T. Spellman, G. Sherlock, V.R. Iyer, K. Anders, M.B. Eisen, P.O. Brown, and D. Botstein. Comprehensive identiﬁcation of cell cycle-regulated genes of the yeast Saccharomyces cerevisiae by microarray hybridization. Molecular Biology of the Cell, 9(12):3273–3297, 1998.</p>
<p>[6] M. LLinas, Z. Bozdech, E. D. Wong, A.T. Adai, and J. L. DeRisi. Comparative whole genome transcriptome analysis of three Plasmodium falciparum strains. Nucleic Acids Research, 34(4):1166–1173, 2006.</p>
<p>[7] M. Beal. Variational Algorithms for Approximate Bayesian Inference. PhD thesis, UCL, 2003.</p>
<p>[8] C. Sabatti and G.M. James. Bayesian sparse hidden components analysis for transcription regulation networks. Bioinformatics, 22(6):739–746, 2006.</p>
<p>[9] S.T. Jensen, G. Chen, and C.J. Stoeckert. Bayesian variable selection and data integration for biological regulatory networks. The Annals of Applied Statistics, 1:612–633, 2007.</p>
<p>[10] E.I. George and R.E. McCulloch. Approaches for Bayesian variable selection. Statistica Sinica, 7:339–374, 1997.</p>
<p>[11] E.I. George and R.E. McCulloch. Variable selection via Gibbs sampling. Journal of the American Statistical Association, 88(423):881–889, 1993.</p>
<p>[12] T. Minka. A family of algorithms for approximate Bayesian inference. PhD thesis, MIT, 2001.</p>
<p>[13] T. Heskes and O. Zoeter. Expectation propagation for approximate inference in dynamic Bayesian networks. In UAI-2002, pages 216–223, 2002.</p>
<p>[14] O. Troyanskaya, M. Cantor, P. Brown, T. Hastie, R. Tibshirani, and D. Botstein. Missing value estimation methods for dna microarrays. Bioinformatics, 17(6):520–525, 2001.</p>
<p>[15] J. Lucas, C. Carvalho, Q. Wang, A. Bild, J. Nevins, and M. West. Sparse statistical modelling in gene expression genomics. In K.A. Do, P. M¨ ller, and M. Vannucci, editors, Bayesian u inference for gene expression and proteomics. Springer, 2006.</p>
<p>[16] M.Y. Park, T. Hastie, and R. Tibshirani. Averaged gene expressions for regression. Biostatistics, 8:212–227, 2007.  8</p>
<br/>
<br/><br/><br/>

<script>
(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

ga('create', 'UA-48522588-1', 'makerhacker.github.io');
ga('send', 'pageview');
</script>

</body>
</html>
