<!DOCTYPE html>
<html>
<head>
<meta charset=utf-8>
<title>125 iccv-2013-Drosophila Embryo Stage Annotation Using Label Propagation</title>
</head>

<body>
<p><a title="iccv" href="../iccv_home.html">iccv</a> <a title="iccv-2013" href="../home/iccv2013_home.html">iccv2013</a> <a title="iccv-2013-125" href="../iccv2013/iccv-2013-Drosophila_Embryo_Stage_Annotation_Using_Label_Propagation.html">iccv2013-125</a> <a title="iccv-2013-125-reference" href="#">iccv2013-125-reference</a> knowledge-graph by maker-knowledge-mining</p><h1>125 iccv-2013-Drosophila Embryo Stage Annotation Using Label Propagation</h1>
<br/><p>Source: <a title="iccv-2013-125-pdf" href="http://www.cv-foundation.org/openaccess/content_iccv_2013/papers/Kazmar_Drosophila_Embryo_Stage_2013_ICCV_paper.pdf">pdf</a></p><p>Author: Tomáš Kazmar, Evgeny Z. Kvon, Alexander Stark, Christoph H. Lampert</p><p>Abstract: In this work we propose a system for automatic classification of Drosophila embryos into developmental stages. While the system is designed to solve an actual problem in biological research, we believe that the principle underlying it is interesting not only for biologists, but also for researchers in computer vision. The main idea is to combine two orthogonal sources of information: one is a classifier trained on strongly invariant features, which makes it applicable to images of very different conditions, but also leads to rather noisy predictions. The other is a label propagation step based on a more powerful similarity measure that however is only consistent within specific subsets of the data at a time. In our biological setup, the information sources are the shape and the staining patterns of embryo images. We show experimentally that while neither of the methods can be used by itself to achieve satisfactory results, their combination achieves prediction quality comparable to human per- formance.</p><br/>
<h2>reference text</h2><p>[1] M. Belkin, P. Niyogi, and V. Sindhwani. Manifold regularization: a geometric framework for learning from labeled and unlabeled examples. Journal of Machine Learning Research (JMLR), 7:2399–2434, 2006.</p>
<p>[2] X. Cai, H. Wang, H. Huang, and C. Ding. Joint stage recognition and anatomical annotation of Drosophila gene expression patterns. Bioinformatics, 28(12): 116–124, 2012.</p>
<p>[3] J. A. Campos-Ortega and V. Hartenstein. The embryonic development of Drosophila melanogaster. Springer, 1997.</p>
<p>[4] C. Dance, J. Willamowski, L. Fan, C. Bray, and G. Csurka. Visual categorization with bags of keypoints. In ECCV International Workshop on Statistical Learning in Computer Vision, 2004.</p>
<p>[5] H. Freeman. On the encoding of arbitrary geometric configurations. IRE Transactions on Electronic Computers, EC10(2):260–268, 1961.</p>
<p>[6] E. Frise, A. S. Hammonds, and S. E. Celniker. Systematic image-driven analysis of the spatial Drosophila embryonic expression landscape. Molecular Systems Biology, 6:345, 2010.</p>
<p>[7] S. Ji, L. Yuan, Y.-X. Li, Z.-H. Zhou, S. Kumar, and J. Ye. Drosophila gene expression pattern annotation using sparse features and term-term interactions. In International Conference on Knowledge Discovery and Data Mining (KDD), 2009.</p>
<p>[8] S. Kumar, K. Jayaraman, S. Panchanathan, R. Gurunathan, A. Marti-Subirana, and S. J. Newfeld. BEST: a novel computational approach for comparing gene expression patterns from early stages of Drosophila melanogaster development.</p>
<p>[9]</p>
<p>[10]</p>
<p>[11]</p>
<p>[12]</p>
<p>[13]</p>
<p>[14]</p>
<p>[15]  Genetics, 162(4):2037–2047, 2002. E. Z. Kvon, T. Kazmar, G. Stampfel, J. O. Y ´a n˜ez-Cuna, M. Pagani, K. Schernhuber, B. J. Dickson, and A. Stark. Genome-scale functional characterization of developmental enhancers in Drosophila. (submitted). C. H. Lampert. Kernel methods in computer vision. Foundations and Trend in Computer Graphics and Vision, 4: 193– 285, 2009. Y.-X. Li, S. Ji, S. Kumar, J. Ye, and Z.-H. Zhou. Drosophila gene expression pattern annotation through multi-instance multi-label learning. In International Joint Conferences on Artificial Intelligence (IJCAI), 2009. H. Peng and E. W. Myers. Comparing in situ mRNA expression patterns of Drosophila embryos. In International Conference on Resaerch in Computational Molecular Biology (RECOMB), 2004. B. D. Pfeiffer, A. Jenett, A. S. Hammonds, T.-T. B. Ngo, S. Misra, C. Murphy, A. Scully, J. W. Carlson, K. H. Wan, T. R. Laverty, C. Mungall, R. Svirskas, J. T. Kadonaga, C. Q. Doe, M. B. Eisen, S. E. Celniker, and G. M. Rubin. Tools for neuroanatomy and neurogenetics in Drosophila. Proceedings of the National Academy of Sciences, 105(28):9715– 9720, 2008. K. Puniyani, C. Faloutsos, and E. P. Xing. SPEX2: automated concise extraction of spatial gene expression patterns from fly embryo ISH images. Bioinformatics, 26(12): 147– 156, June 2010. K. Puniyani and E. P. Xing. Inferring gene interaction networks from ISH images via kernelized graphical models. In European Conference on Computer Vision (ECCV). Springer, 2012.</p>
<p>[16] P. Tomancak, A. Beaton, R. Weiszmann, E. Kwan, S. Shu, S. E. Lewis, S. Richards, M. Ashburner, V. Hartenstein, S. E. Celniker, and G. M. Rubin. Systematic determination of patterns of gene expression during Drosophila embryogenesis. Genome Biology, 3(12):81–88, 2002.</p>
<p>[17] P. Tomancak, B. P. Berman, A. Beaton, R. Weiszmann, E. Kwan, V. Hartenstein, S. E. Celniker, and G. M. Rubin. Global analysis of patterns of gene expression during Drosophila embryogenesis. Genome Biology, 8(7):R145, 2007.</p>
<p>[18] J. Ye, J. Chen, Q. Li, and S. Kumar. Classification of Drosophila embryonic developmental stage range based on gene expression pattern images. In Computational System Bioinformatics Conference, 2006.</p>
<p>[19] J. Zhou and H. Peng. Automatic recognition and annotation of gene expression patterns of fly embryos. Bioinformatics, 23(5), 2007.</p>
<p>[20] X. Zhu, Z. Ghahramani, and J. D. Lafferty. Semi-supervised learning using Gaussian fields and harmonic functions. In International Conference on Machine Learning (ICML), 2003. 11009966</p>
<br/>
<br/><br/><br/></body>
</html>
